<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01+RDFa 1.0//EN" "http://www.w3.org/MarkUp/DTD/html401-rdfa-1.dtd">
<!-- saved from url=(0262)file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html -->
<html xmlns="http://www.w3.org/1999/xhtml" xmlns:foaf="http://xmlns.com/foaf/0.1/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dcterms="http://purl.org/dc/terms/" class=" webkit chrome win js webkit chrome linux js"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><script type="text/javascript" async="" defer="" src="file:///piwik/piwik.js"></script><script id="twitter-wjs" src="./Network_Diffusion_Brain_files/widgets.js"></script><script type="text/javascript" async="" defer="" src="./Network_Diffusion_Brain_files/piwik(1).js"></script><script type="text/javascript" src="./Network_Diffusion_Brain_files/jquery-1.11.1.min-ver-8101D596B2B8FA35FE3A634EA342D7C3.js"></script>
<link rel="stylesheet" type="text/css" href="./Network_Diffusion_Brain_files/orcid-min.css">
<script type="text/javascript" src="./Network_Diffusion_Brain_files/AccoutSigninPanel-ver-6FF2BBBDD25CB424618ABD4C179050EC.js"></script>
<script type="text/javascript" src="./Network_Diffusion_Brain_files/wicket-event-jquery.min-ver-F070EDFAD7E5A1BD9662F385470094FF.js"></script>
<script type="text/javascript" src="./Network_Diffusion_Brain_files/wicket-ajax-jquery.min-ver-1D4EE56166B3A1106E05B678CC2AB1EF.js"></script>
<script type="text/javascript" src="./Network_Diffusion_Brain_files/modal.min-ver-E43C9229F7812A6259B89C7E5B5028FB.js"></script>
<link rel="stylesheet" type="text/css" href="./Network_Diffusion_Brain_files/modal.min-ver-E99D4201F0F6C5D3081AD42ACB1F22C2.css">
<script type="text/javascript" id="wicket-ajax-base-url">
/*<![CDATA[*/
Wicket.Ajax.baseUrl="articles/pmc3951650";
/*]]>*/
</script>
<script type="text/javascript" src="./Network_Diffusion_Brain_files/TemplatePage-ver-CF32DB200AE9B4EE2568FE9CD0FB51C4.js"></script>


<meta name="DC.Date" content="2014-04-01">

<meta name="DCTERMS.isPartOf" content="NeuroImage [2014, 90], pp335-347">

<meta name="DC.Identifier" content="info:pmid/24384152">

<meta name="DC.Identifier" content="info:doi/10.1016/j.neuroimage.2013.12.039">

<meta name="DCTERMS.hasFormat" content="http://dx.doi.org/10.1016/j.neuroimage.2013.12.039">

<meta name="DC.Title" content="Network diffusion accurately models the relationship between structural and functional brain connectivity networks.">

<meta name="DC.Type" content="Journal Article">

<meta name="DC.Type" content="Research Support, U.S. Gov&#39;t, Non-P.H.S.">

<meta name="DC.Type" content="Research Support, N.I.H., Extramural">

<meta name="DC.Creator" content="Abdelnour F">

<meta name="DC.Creator" content="Voss HU">

<meta name="DC.Creator" content="Raj A">

<meta name="DC.Contributor" content="Department of Radiology, Weill Cornell Medical College, New York, NY, USA. Electronic address: faa2016@med.cornell.edu.">

<meta name="DC.Language" content="en">

<meta name="DC.Subject" content="info:mesh/Humans">

<meta name="DC.Subject" content="info:mesh/Brain">

<meta name="DC.Subject" content="info:mesh/Nerve Net">

<meta name="DC.Subject" content="info:mesh/Magnetic Resonance Imaging">

<meta name="DC.Subject" content="info:mesh/Brain Mapping">

<meta name="DC.Subject" content="info:mesh/Linear Models">

<meta name="DC.Subject" content="info:mesh/Image Processing, Computer-Assisted">

<meta name="DC.Subject" content="info:mesh/Diffusion Magnetic Resonance Imaging">

<meta name="DC.Subject" content="info:mesh/Models, Neurological">


<link rel="schema.DC" href="http://purl.org/dc/elements/1.1/">
<link rel="schema.DCTERMS" href="http://purl.org/dc/terms/">

<meta name="citation_journal_title" content="NeuroImage">

<meta name="citation_journal_abbrev" content="Neuroimage">

<meta name="citation_publication_date" content="2014/04">

<meta name="citation_issn" content="1053-8119">

<meta name="citation_volume" content="90">

<meta name="citation_firstpage" content="335">

<meta name="citation_lastpage" content="347">

<meta name="citation_title" content="Network diffusion accurately models the relationship between structural and functional brain connectivity networks.">

<meta name="citation_doi" content="10.1016/j.neuroimage.2013.12.039">

<meta name="citation_pmcid" content="PMC3951650">

<meta name="citation_author" content="Abdelnour F">

<meta name="citation_author" content="Voss HU">

<meta name="citation_author" content="Raj A">

<meta name="citation_keywords" content="Humans; Brain; Nerve Net; Magnetic Resonance Imaging; Brain Mapping; Linear Models; Image Processing, Computer-Assisted; Diffusion Magnetic Resonance Imaging; Models, Neurological">

<meta name="citation_language" content="eng">


<style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Hover_Arrow {position: absolute; width: 15px; height: 11px; cursor: pointer}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; color: #666666}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_Menu_Close {position: absolute; width: 31px; height: 31px; top: -15px; left: -15px}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; border: 0; padding: 0; margin: 0}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0 !important; padding: 0 !important; margin: 0 !important; vertical-align: 0 !important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap ! important}
.MathJax img {display: inline ! important; float: none ! important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block; overflow: hidden; width: 1px; height: 60ex}
.MathJax .MathJax_EmBox {display: block; overflow: hidden; width: 1px; height: 60em}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff') format('woff'), url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff') format('woff'), url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff') format('woff'), url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff') format('woff'), url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff') format('woff'), url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff') format('woff'), url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff') format('woff'), url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff') format('woff'), url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff') format('woff'), url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf') format('opentype')}
</style><style type="text/css">@font-face {font-family: MathJax_Script; src: url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/woff/MathJax_Script-Regular.woff') format('woff'), url('http://cdn.mathjax.org/mathjax/2.3-latest/fonts/HTML-CSS/TeX/otf/MathJax_Script-Regular.otf') format('opentype')}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style></head><body align="center" height="100%"><div id="WzTtDiV" style="visibility: hidden; position: absolute; overflow: hidden; padding: 0px; width: 0px; left: 0px; top: 0px;"></div><div id="MathJax_Message" style="display: none;"></div><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div><div id="WzTtDiV" style="visibility: hidden; position: absolute; overflow: hidden; padding: 0px; width: 0px; left: 0px; top: 0px;"></div><div style="display:none">

</div>



<span style="display:none" property="dc:abstract" datatype="xsd:string" itemprop="description">The relationship between anatomic connectivity of large-scale brain networks and their functional connectivity is of immense importance and an area of active research. Previous attempts have required complex simulations which model the dynamics of each cortical region, and explore the coupling between regions as derived by anatomic connections. While much insight is gained from these non-linear simulations, they can be computationally taxing tools for predicting functional from anatomic connectivities. Little attention has been paid to linear models. Here we show that a properly designed linear model appears to be superior to previous non-linear approaches in capturing the brain's long-range second order correlation structure that governs the relationship between anatomic and functional connectivities. We derive a linear network of brain dynamics based on graph diffusion, whereby the diffusing quantity undergoes a random walk on a graph. We test our model using subjects who underwent diffusion MRI and resting state fMRI. The network diffusion model applied to the structural networks largely predicts the correlation structures derived from their fMRI data, to a greater extent than other approaches. The utility of the proposed approach is that it can routinely be used to infer functional correlation from anatomic connectivity. And since it is linear, anatomic connectivity can also be inferred from functional data. The success of our model confirms the linearity of ensemble average signals in the brain, and implies that their long-range correlation structure may percolate within the brain via purely mechanistic processes enacted on its structural connectivity pathways.</span>

<meta name="bioentities" content="panel">


<meta name="DCTERMS.hasFormat" content="http://europepmc.org/articles/pmc3951650">

<meta name="citation_fulltext_html_url" content="http://europepmc.org/articles/pmc3951650">

<meta name="DC.Identifier" content="info:pmcid/pmc3951650">

<meta name="DCTERMS.hasFormat" content="http://europepmc.org/abstract/MED/24384152">

<meta name="citation_abstract_html_url" content="http://europepmc.org/abstract/MED/24384152">

<meta name="DCTERMS.hasFormat" content="http://europepmc.org/articles/pmc3951650?pdf=render">

<meta name="citation_pdf_url" content="http://europepmc.org/articles/pmc3951650?pdf=render">


<title>Network diffusion accurately models the relationship between structural and functional brain... - Europe PMC Article - Europe PubMed Central</title>
<link rel="SHORTCUT ICON" href="http://europepmc.org/images/favicon.ico" type="image/x-icon">
<link rel="alternate" type="application/rss+xml" title="Europe PMC: Popular articles" href="http://europepmc.org/Funders/RSS/MostViewedArticles.xml">
<link rel="alternate" type="application/rss+xml" title="Europe PMC: Recent articles" href="http://europepmc.org/Funders/RSS/AllFunders.xml">
<link rel="alternate" type="application/rss+xml" title="Europe PMC: Latest journals" href="http://europepmc.org/Funders/rss/NewJournals.xml">
<link rel="alternate" type="application/rss+xml" title="Europe PMC: Content Holdings" href="http://europepmc.org/contentrss">

<meta name="author" content="Europe PubMed Central (Europe PMC)">
<meta name="owner" content="Europe PubMed Central (Europe PMC)">
<meta name="keywords" content="Europe PMC, Europe PubMed Central, UK PhD Theses/Thesis, UK Clinical Guidelines, UK Research Reports, Biological Patents, life sciences, bioinformatics, biomedical research, health research, journal articles, citation search, medical journals">
<meta name="description" content="FULL TEXT Abstract: The relationship between anatomic connectivity of large-scale brain networks and their functional connectivity is of immense importance and an...">

<link rel="search" type="application/opensearchdescription+xml" title="Search Europe PMC" href="http://europepmc.org/tools/opensearch_epmc.xml">
<link rel="stylesheet" href="./Network_Diffusion_Brain_files/jquery_ui.css" type="text/css" media="print, projection, screen">
<link rel="stylesheet" href="./Network_Diffusion_Brain_files/jquery_menu.css" type="text/css" media="screen, projection">
<!--[if lte IE 7]>
<link rel="stylesheet" type="text/css" href="/template/jquery_menu_ie.css" media="screen" />
<![endif]-->

<script src="./Network_Diffusion_Brain_files/yahoo-min.js" type="text/javascript"></script>
<script src="./Network_Diffusion_Brain_files/dom-min.js" type="text/javascript"></script>
<script src="./Network_Diffusion_Brain_files/enrichment.js" type="text/javascript"></script>
<link rel="stylesheet" href="./Network_Diffusion_Brain_files/enrichment.css" type="text/css" media="print, projection, screen">


<script type="text/javascript" language="javascript" src="./Network_Diffusion_Brain_files/hoverIntent.js"></script>
<script type="text/javascript" language="javascript" src="./Network_Diffusion_Brain_files/jquery.js"></script>
<script src="./Network_Diffusion_Brain_files/urchin.js" type="text/javascript"></script>
<script type="text/javascript">
/*<![CDATA[*/

urchinTracker();

/*]]>*/
</script>
<link rel="stylesheet" type="text/css" href="./Network_Diffusion_Brain_files/epmc.ncbi.css">
<script src="./Network_Diffusion_Brain_files/epmc.js" type="text/javascript"></script>
<link href="./Network_Diffusion_Brain_files/epmc.css" rel="stylesheet" type="text/css">
<script type="text/javascript">
/*<![CDATA[*/
ie7 = false;ie8 = false;
/*]]>*/
</script>
<!--[if !IE]> -->
<link rel="stylesheet" type="text/css" href="./Network_Diffusion_Brain_files/non-ie.css">
<!--[if IE]>
<link rel="stylesheet" type="text/css" href="/template/ie.css" />
<![endif]-->
<!--[if IE 7]>
<link href="/template/ie7.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript">
/*<![CDATA[*/
ie7 = true;
/*]]>*/
</script>
<![endif]-->
<!--[if IE 8]>
<link href="/template/ie8.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript">
/*<![CDATA[*/
ie8 = true;
/*]]>*/
</script>
<![endif]-->
<!--[if IE 9]>
<link href="/template/ie9.css" rel="stylesheet" type="text/css"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<![endif]-->
<!--[if lt IE 9]>
<script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
<![endif]-->

<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="http://europepmc.org/articles/PMC3951650/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="NeuroImage"><meta name="citation_title" content="Network diffusion accurately models the relationship between structural and functional brain connectivity networks"><meta name="citation_date" content="15 April 2014"><meta name="citation_volume" content="90"><meta name="citation_firstpage" content="335"><meta name="citation_doi" content="10.1016/j.neuroimage.2013.12.039"><meta name="citation_abstract_html_url" content="http://europepmc.org/articles/PMC3951650/?report=abstract"><meta name="citation_pmid" content="24384152"><meta name="DC.Title" content="Network diffusion accurately models the relationship between structural and functional brain connectivity networks"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Farras Abdelnour"><meta name="DC.Contributor" content="Henning U. Voss"><meta name="DC.Contributor" content="Ashish Raj"><meta name="DC.Date" content="2014 April 15"><meta name="DC.Identifier" content="10.1016/j.neuroimage.2013.12.039"><meta name="DC.Language" content="en"><link rel="stylesheet" href="./Network_Diffusion_Brain_files/ncbi.min.css" type="text/css"><link rel="stylesheet" href="./Network_Diffusion_Brain_files/pmc.min.css" type="text/css"><link rel="stylesheet" href="./Network_Diffusion_Brain_files/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript">if (true || typeof jQuery === "undefined") document.write('[script type="text/javascript" src="/corehtml/pmc/jig/1.11/js/jig.min.js"][/script]'.replace(/\[/g,String.fromCharCode(60)).replace(/\]/g,String.fromCharCode(62)));</script><script type="text/javascript" src="file:///corehtml/pmc/jig/1.11/js/jig.min.js"></script><script type="text/javascript" src="./Network_Diffusion_Brain_files/jig.min(1).js"></script><link rel="stylesheet" type="text/css" href="file:///corehtml/pmc/jig/1.11/css/jig.core.theme.min.css"><script type="text/javascript" src="file:///corehtml/pmc/jig/1.11/js/jquery.ui.widgets.min.js"></script><link type="text/css" rel="stylesheet" href="file:///corehtml/pmc/jig/1.11/css/jquery.ui.widgets.min.css"><link rel="stylesheet" type="text/css" href="./Network_Diffusion_Brain_files/jig.core.theme.min(1).css"><script type="text/javascript" src="./Network_Diffusion_Brain_files/jquery.ui.widgets.min(1).js"></script><link type="text/css" rel="stylesheet" href="./Network_Diffusion_Brain_files/jquery.ui.widgets.min(2).css"><script type="text/javascript"></script><script type="text/javascript" src="./Network_Diffusion_Brain_files/common.min.js"></script><script type="text/javascript" src="./Network_Diffusion_Brain_files/NcbiTagServer.min.js"></script><script type="text/javascript" src="./Network_Diffusion_Brain_files/MathJax.js"></script><script type="text/javascript" src="./Network_Diffusion_Brain_files/large-obj-scrollbars.min.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

<span style="display:none" id="piwikIdSite">1</span>

<script type="text/javascript">
/*<![CDATA[*/

$(function()
{
$("#Highlight_head").click(function(event) {
event.preventDefault();
$("#Highlight_body").slideToggle();
});
$("#Highlight_body a").click(function(event) {
event.preventDefault();
$("#Highlight_body").slideUp();
});
var locationHost = $.cookie('location_host');
if (typeof locationHost === 'undefined' || locationHost == null || locationHost == "") {
$.cookie("location_host", window.location.host);
}
});

/*]]>*/
</script>
<script type="text/javascript">
/*<![CDATA[*/

Wicket.Event.subscribe('/ajax/call/complete', function(jqEvent, attributes, jqXHR, errorThrown, textStatus) {
if (jqXHR != null && jqXHR.responseText != null &&
(jqXHR.responseText.match("ShowCitationDetails]]></redirect></ajax-response>")
|| jqXHR.responseText.match("articles]]></redirect></ajax-response>"))) {
var ua = window.navigator.userAgent;
var msie = ua.indexOf("MSIE ");
var trident = ua.indexOf("Trident/");
if (msie > 0 || trident > 0) {
window.document.execCommand("Stop");
} else {
window.stop();
}
}
});

/*]]>*/
</script>

<script type="text/javascript" src="./Network_Diffusion_Brain_files/Biojs.js"></script>
<script type="text/javascript" src="./Network_Diffusion_Brain_files/Biojs.Annotator.js"></script>
<script type="text/javascript" src="./Network_Diffusion_Brain_files/jquery-ui.min-ver-80CBC031ED1AE0BD82C578DD2D6A4EA0.js"></script>
<script type="text/javascript" src="./Network_Diffusion_Brain_files/biojs_functions-min.js"></script>
<link rel="stylesheet" type="text/css" href="./Network_Diffusion_Brain_files/jquery-ui-min.css">
<link rel="stylesheet" type="text/css" href="./Network_Diffusion_Brain_files/labs_links-min.css">
<link rel="stylesheet" type="text/css" href="./Network_Diffusion_Brain_files/annotation_highlight-min.css">
<script type="text/javascript">
/*<![CDATA[*/
Wicket.Event.add(window, "domready", function(event) { 
Wicket.Ajax.ajax({"u":"./pmc3951650?1-1.IBehaviorListener.0-search-recentsearches","e":"click","c":"id11"});;
Wicket.Ajax.ajax({"u":"./pmc3951650?1-1.IBehaviorListener.0-search-clipboard","e":"click","c":"id12"});;
Wicket.Ajax.ajax({"coh":[function(attrs, jqXHR, textStatus){;onClickPanelLink();registerExportPanel();}],"u":"./pmc3951650?1-1.IBehaviorListener.0-search-exportPanelLink","e":"click","c":"exportPanelLink","i":"exportPanelLink--ajax-indicator"});;
Wicket.Ajax.ajax({"u":"./pmc3951650?1-1.IBehaviorListener.0-clipboard_icon-addclipboard","e":"click","c":"id13"});;
Wicket.Ajax.ajax({"u":"./pmc3951650?1-1.IBehaviorListener.0-anchor_related_labLinks","e":"click","c":"relatedlabLinks_link","pre":[function(attrs){return false;}]});;
Wicket.Ajax.ajax({"u":"./pmc3951650?1-1.IBehaviorListener.0-anchor_related_citations","e":"click","c":"relatedcitation_link","pre":[function(attrs){if (jQuery('#references_loading_label').length ==0){

 return false;

}else{

 return true;

}}]});;
Wicket.Ajax.ajax({"u":"./pmc3951650?1-1.IBehaviorListener.0-anchor_related_bioentities","e":"click","c":"relatedentity_link","pre":[function(attrs){if (jQuery('#loading_bioentities_label').length ==0){

 return false;

}else{

 return true;

}}]});;
Wicket.Ajax.ajax({"u":"./pmc3951650?1-1.IBehaviorListener.0-anchor_related_articles","e":"click","c":"relatedarts_link","pre":[function(attrs){if (jQuery('#rel_articles_loading_label').length ==0){

 return false;

}else{

 return true;

}}]});;
Wicket.Ajax.ajax({"u":"./pmc3951650?1-1.IBehaviorListener.0-anchor_related_labLinks","pre":[function(attrs){if (jQuery('#fragment-related-labLinks').css('display') =='block'){

 return true;

}else{

 return false;

}}],"sh":[function(attrs, jqXHR, data, textStatus){if (jQuery('#citationDetails').length>0){

 jQuery('#citationDetails').removeClass('ui-tabs-active');

}

if (jQuery('#fulltext').length>0){

 jQuery('#fulltext').removeClass('ui-tabs-active');

}

}]});;
Wicket.Ajax.ajax({"u":"./pmc3951650?1-1.IBehaviorListener.0-anchor_related_articles","pre":[function(attrs){if (jQuery('#fragment-related-pubmedarts').css('display') =='block'){

 return true;

}else{

 return false;

}}],"sh":[function(attrs, jqXHR, data, textStatus){if (jQuery('#citationDetails').length>0){

 jQuery('#citationDetails').removeClass('ui-tabs-active');

}

if (jQuery('#fulltext').length>0){

 jQuery('#fulltext').removeClass('ui-tabs-active');

}

}]});;
Wicket.Ajax.ajax({"u":"./pmc3951650?1-1.IBehaviorListener.0-anchor_related_citations","pre":[function(attrs){if (jQuery('#fragment-related-citations').css('display') =='block'){

 return true;

}else{

 return false;

}}],"sh":[function(attrs, jqXHR, data, textStatus){if (jQuery('#citationDetails').length>0){

 jQuery('#citationDetails').removeClass('ui-tabs-active');

}

if (jQuery('#fulltext').length>0){

 jQuery('#fulltext').removeClass('ui-tabs-active');

}

}]});;
Wicket.Ajax.ajax({"u":"./pmc3951650?1-1.IBehaviorListener.0-anchor_related_bioentities","pre":[function(attrs){if (jQuery('#fragment-related-bioentities').css('display') =='block'){

 return true;

}else{

 return false;

}}],"sh":[function(attrs, jqXHR, data, textStatus){if (jQuery('#citationDetails').length>0){

 jQuery('#citationDetails').removeClass('ui-tabs-active');

}

if (jQuery('#fulltext').length>0){

 jQuery('#fulltext').removeClass('ui-tabs-active');

}

}]});;
;});
/*]]>*/
</script>


<noscript>
&amp;lt;div align="center"&amp;gt;
&amp;lt;p&amp;gt;
&amp;lt;b&amp;gt;Europe PubMed Central (Europe PMC)&amp;lt;/b&amp;gt; requires Javascript to function effectively.
&amp;lt;/p&amp;gt;
&amp;lt;p&amp;gt;
Either your web browser doesn't support Javascript or it is currently turned off.
In the latter case, please turn on Javascript support in your web browser and reload this page.
&amp;lt;/p&amp;gt;
&amp;lt;/div&amp;gt;
</noscript>
<div class="page_proper">
<header>
<div id="epmc_pageHeader" class="not4Frame">
<div>


<a href="http://europepmc.org/feedback?url=http://europepmc.org/articles/pmc3951650" class="ukpmc_toolbar feedbackLink" onmouseover="TipAllowed(&#39;Leave your feedback&#39;)" onmouseout="UnTipAllowed()" onclick="var w = window.open(href, &#39;ukpmc_feedback&#39;, &#39;scrollbars=no,location=no,menuBar=no,resizable=no,status=no,toolbar=no,width=550,height=650&#39;); try {if (w.blur) w.focus();}catch(ignore){}; return false;"><img alt="Leave your feedback" src="./Network_Diffusion_Brain_files/feedback_icon.gif" border="0">Feedback</a>


<div style="background-color:#5A595E">
<div style="margin-left:auto;margin-right:auto;width:960px;height:30px;">
<span style="float:right; margin-right:20px; padding-top:4px;">
<link type="text/css" rel="stylesheet" href="./Network_Diffusion_Brain_files/style.css">
<span style="margin: 0 5px 0 42px;">

<span id="id14">
<span>
<a id="orcid_signin_text" href="http://europepmc.org/accounts/login">
Sign in
</a>
</span>
</span>
</span>
<iframe id="iframeId" name="aframe" frameborder="0" seamless=""></iframe>
<span id="orcidurlId"></span>
</span>
</div>
</div>
<nav style="margin-left:auto;margin-right:auto;width:960px;">
<ul class="dropdown" id="epmc_mainNav">
<li>
<a>About</a>
<ul class="sub_menu">
<li>
<a href="http://europepmc.org/About">
<span>Overview</span>

</a>
</li><li>
<a href="http://europepmc.org/Funders">
<span>Funders</span>

</a>
</li><li>
<a href="http://europepmc.org/Governance">
<span>Governance</span>

</a>
</li><li>
<a href="http://europepmc.org/Joining">
<span>Joining Europe PMC</span>

</a>
</li><li>
<a href="http://europepmc.org/ContactUs">
<span>Contact us</span>

</a>
</li><li>
<a href="http://blog.europepmc.org/" target="_blank">
<span>Blog</span>

</a>
</li>
</ul>
</li>
<li>
<a href="http://europepmc.org/journalList" accesskey="J">Journal List</a>
</li>
<li>
<a href="http://europepmc.org/GrantLookup/" accesskey="G">Grant Lookup</a>
</li>
<li>
<a>Resources</a>
<ul class="sub_menu">
<li>
<a href="http://europepmc.org/ResourcesOverview" accesskey="R">Resources Overview</a>
</li>
<li>
<a id="webservices">Web Services »  » </a>
<ul class="grand_sub_menu">
<li>
<a href="http://europepmc.org/SoapWebServices">
SOAP (articles)
</a>
</li>
<li>
<a href="http://europepmc.org/RestfulWebService">
RESTful (articles)
</a>
</li>
<li>
<a href="http://plus.europepmc.org/GristAPI/" target="_blank">
RESTful (grants)
</a>
</li>
</ul>
</li>
<li>
<a href="http://europepmc.org/RssFeeds" accesskey="R">RSS Feeds</a>
</li>
<li>
<a href="http://europepmc.org/FtpSite" accesskey="F">OA subset and FTP site</a>
</li>
<li>
<a href="http://europepmc.org/OaiService" accesskey="I">OAI Service</a>
</li>
<li>
<a href="http://europepmc.org/LabsLink" accesskey="F">External Links Service</a>
</li>

<li>
<a href="http://europepmc.org/orcid/import" accesskey="O">ORCID Article Claiming</a>
</li>

</ul>
</li>
<li>
<a href="http://europepmc.org/Help" accesskey="H">Help</a>
</li>
<li class="plus">
<a href="http://plus.europepmc.org/" target="epmcplus" onmouseover="TipAllowed(&#39;Submit your manuscripts, manage your grants and publish your research impact using &lt;b&gt;Europe PMC plus&lt;/b&gt;&#39;)" onmouseout="UnTipAllowed()">
<span>Europe PMC plus</span>
</a>
</li>
<li class="labs last">
<a href="http://labs.europepmc.org/" target="epmclabs" onmouseover="TipAllowed(&#39;Prototypes of semantic applications&#39;)" onmouseout="UnTipAllowed()">
<span>Europe PMC labs</span>
</a>
</li>
</ul>
</nav>
</div>
</div>
</header>
<div id="epmc_pageBody">



<div>

<link rel="stylesheet" type="text/css" href="./Network_Diffusion_Brain_files/TemplatePage-ver-900FDE4D18F036E68A784894A5065243.css">

<link type="text/css" rel="stylesheet" href="./Network_Diffusion_Brain_files/style.css">
<script type="text/javascript" src="./Network_Diffusion_Brain_files/recaptcha_ajax.js"></script>
<div id="epmc_searchBanner">
<div class="holder">
<div class="logo">
<a href="http://europepmc.org/" target="_top" accesskey="U">
<h1>Europe PubMed Central</h1>
</a>
</div>
<div class="search">
<form action="http://europepmc.org/articles/pmc3951650?1-1.IFormSubmitListener-search-searchForm" method="post" id="searchForm11" name="searchForm11" accept-charset="UTF-8"><div style="width:0px;height:0px;position:absolute;left:-100px;top:-100px;overflow:hidden"><input type="hidden" name="searchForm11_hf_0" id="searchForm11_hf_0"></div>
<input type="text" id="textfield" name="searchString" maxlength="1500" size="50" placeholder="search for..." value="">
<input value="Search" type="submit">
</form>
<a href="http://europepmc.org/advancesearch" class="advancedSearch" title="Advanced Search">Advanced Search</a>
<div class="epmc_searchTools">

<div id="save_search_modal" style="display:none">
<div id="id15" style="display:none"></div>
</div>
<div id="save_search_modal" style="display:none">
<div id="id16" style="display:none"></div>
</div>

<a href="http://europepmc.org/articles/pmc3951650?1-1.ILinkListener-search-recentsearches" class="ukpmc_toolbar" onmouseover="TipAllowed(&#39;View your Recent Activity summary.&#39;, DELAY, 1000)" onmouseout="UnTipAllowed()" id="id11">
<img alt="" src="./Network_Diffusion_Brain_files/recentactivityICON_small.png" border="0">
Recent Activity
</a>
<div id="id17" style="display:none">
<div id="id18" style="display:none"></div>
</div>
&nbsp;|&nbsp;
<a style="display:none" href="http://europepmc.org/articles/pmc3951650#" class="ukpmc_toolbar" onmouseover="TipAllowed(&#39;View your Clipboard summary&#39;, DELAY, 1000)" onmouseout="UnTipAllowed()">
<img alt="View summary of your Clipboard" src="./Network_Diffusion_Brain_files/clipboardICON_small.png" border="0">
[Export]
</a>
<a style="display:none" href="http://europepmc.org/articles/pmc3951650?1-1.ILinkListener-search-clipboard" class="ukpmc_toolbar" onmouseover="TipAllowed(&#39;View your Clipboard summary.&lt;br/&gt;The Clipboard is a temporary store for items of interest to you.&lt;br/&gt;The items in your Clipboard may be cleared after 24 hours.&#39;, DELAY, 1000)" onmouseout="UnTipAllowed()" id="id12">
<img alt="View summary of your Clipboard" src="./Network_Diffusion_Brain_files/clipboardICON_small.png" border="0">
Clipboard
</a>
<div id="id19" style="display:none">
<div id="id1a" style="display:none"></div>
</div>
<a id="exportPanelLink" href="javascript:;" class="ukpmc_toolbar" onmouseover="TipAllowed(&#39;Export citations&#39;, DELAY, 1000)" onmouseout="UnTipAllowed()">
<img alt="View summary of your Clipboard" src="./Network_Diffusion_Brain_files/clipboardICON_small.png" border="0">
Export
</a><span style="display:none;" class="wicket-ajax-indicator" id="exportPanelLink--ajax-indicator"><img src="./Network_Diffusion_Brain_files/indicator-ver-03CE3DCC84AF110E9DA8699A841E5200.gif" alt=""></span>
<div id="exportPanel" style="display: none;"></div>

</div>
</div>
</div>
</div>
<div id="id1b">

</div>
<script type="text/javascript">
/*<![CDATA[*/

jQuery('#textfield').on('input', function() {
if (jQuery(this).val().length>=1500){
alert('The maximum allowed query length is 1500 characters');
}
});
function onClickPanelLink() {
var speed = 500;
if ($('#exportPanel').css('display') == 'none') {
$("#exportPanel").slideDown(speed);
exportDisplaySection();
// $(".results_pagination_range").fadeOut(speed - 100)
$('#exportPanel').focus();
} else {
$("#exportPanel").slideUp(speed);
// $(".results_pagination_range").fadeIn(speed + 150);
}
}

/*]]>*/
</script>
</div>
<div class="epmc_pageHolder articleContentPage">
<div class="epmc_wideLeft">
<div id="container-1" itemscope="" itemtype="http://schema.org/ScholarlyArticle">

<div class="epmc_citationName">
<div class="clipboard_icon">
<span>
<a href="http://europepmc.org/articles/pmc3951650?1-1.ILinkListener-clipboard_icon-addclipboard" id="id13">
<img style="vertical-align:top;" border="0" id="id1c" onmouseover="TipAllowed(&#39;Add Citation to Export list&#39;)" src="./Network_Diffusion_Brain_files/small_clipboard_off-ver-0CF126D6BE0C2618F52BA585360BFB49.gif"></a>
</span>
</div>
<div class="title">
<span class="abs_citation_title"><span itemprop="name">Network diffusion accurately models the relationship between structural and functional brain connectivity networks.</span></span>
</div>
<div class="abs_nonlink_metadata">(PMID:24384152 PMCID:pmc3951650)</div>
<div property="dc:identifier" class="abs_nonlink_metadata" style="display:none">PMID:24384152 PMCID:pmc3951650</div>
</div>

<div id="tabs_abstract" class="ui-tabs ui-widget ui-widget-content ui-corner-all">
<ul class="ui-tabs-nav ui-helper-reset ui-helper-clearfix ui-widget-header ui-corner-all">
<li id="fulltext" class="ui-state-default ui-corner-top ui-tabs-selected ui-state-active">
<a href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#ui-tabs-1">
<span>Full Text

<img src="./Network_Diffusion_Brain_files/small_logo-ver-AA65016C2A6B5C01BBEF323E29CF92AD.gif" border="0" alt="This article is available in Europe PMC" height="14" width="14" onmouseover="TipAllowed(&#39;The full text of this article is available in Europe PMC&#39;, DELAY, 1000)" onmouseout="UnTipAllowed()">

</span>
</a>
</li>
<li id="relatedCitations" class="ui-tabs-enabled ui-state-default ui-corner-top"><a href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#ui-tabs-2" id="relatedcitation_link"><span>Citations <img border="0" src="./Network_Diffusion_Brain_files/qmark.png" onmouseout="UnTipAllowed()" onmouseover="TipAllowed(&#39;List of citations in Europe PMC which this record is cited by and&lt;br/&gt; those which it cites&#39;)"></span></a></li>
<li id="relatedarts" class="ui-tabs-enabled ui-state-default ui-corner-top"><a href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#ui-tabs-3" id="relatedentity_link"><span>BioEntities <img border="0" src="./Network_Diffusion_Brain_files/qmark.png" onmouseout="UnTipAllowed()" onmouseover="TipAllowed(&#39;Biological entities annotated by the EMBL EBI and NCBI teams&#39;)"></span></a></li>
<li id="relatedpubmedarts" class="ui-tabs-enabled ui-state-default ui-corner-top"><a href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#ui-tabs-4" id="relatedarts_link"><span>Related Articles <img border="0" src="./Network_Diffusion_Brain_files/qmark.png" onmouseout="UnTipAllowed()" onmouseover="TipAllowed(&#39;Articles closely related to the current article derived by NCBI &lt;br/&gt;by comparing subject headings and words in the title and abstract&#39;)"></span></a></li>
<li id="labLinksarts" class="ui-state-default ui-corner-top ui-tabs-disabled"><a href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#ui-tabs-5" id="relatedlabLinks_link"><span>External Links <img border="0" src="./Network_Diffusion_Brain_files/qmark.png" onmouseout="UnTipAllowed()" onmouseover="TipAllowed(&#39;External Links references added by Providers&#39;)"></span></a></li>
</ul><div id="ui-tabs-1" class="ui-tabs-panel ui-widget-content ui-corner-bottom"></div><div id="ui-tabs-2" class="ui-tabs-panel ui-widget-content ui-corner-bottom ui-tabs-hide"></div><div id="ui-tabs-3" class="ui-tabs-panel ui-widget-content ui-corner-bottom ui-tabs-hide"></div><div id="ui-tabs-4" class="ui-tabs-panel ui-widget-content ui-corner-bottom ui-tabs-hide"></div><div id="ui-tabs-5" class="ui-tabs-panel ui-widget-content ui-corner-bottom ui-tabs-hide"></div>
<div class="ui-tabs-panel ncbi_container ui-widget-content ui-corner-bottom" id="fragment-fulltext">
<table width="100%">
<tbody><tr>
<td align="left" class="pmc-wm pmc-wm-column"></td>
<td valign="top">
<div align="left">
<small>
<div><img src="./Network_Diffusion_Brain_files/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="http://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="http://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="http://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="http://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"></map></div> 
</small>
</div>
<div id="article_body">
<div class="hide-overflow article lit-style content pmc-wm slang-en page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: [&#39;h2&#39;], headingExclude: &#39;:hidden&#39;"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="small"><div class="inline_block eight_col va_top"><div>Neuroimage. Author manuscript; available in PMC 2015 April 15.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&retmode=ref&cmd=prlinks&id=24384152" target="pmc_ext" ref="reftype=publisher&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access&amp;rendering-type=normal"><span class="cit">Neuroimage. 2014 April 15; 90: 335–347. </span></a></div><span class="fm-vol-iss-date">Published online 2013 December 30. </span> <span class="doi">doi:&nbsp; <a href="http://dx.doi.org/10.1016%2Fj.neuroimage.2013.12.039" target="pmc_ext" ref="reftype=other&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=normal">10.1016/j.neuroimage.2013.12.039</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3951650</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS552872</span></div></div></div></div></div><h1 class="content-title">Network diffusion accurately models the relationship between structural and functional brain connectivity networks</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="http://europepmc.org/search/?scope=fulltext&page=1&query=AUTH:%22Abdelnour%20F%22">Farras Abdelnour</a>,<sup>a</sup> <a href="http://europepmc.org/search/?scope=fulltext&page=1&query=AUTH:%22Voss%20HU%22">Henning U. Voss</a>,<sup>a</sup> and  <a href="http://europepmc.org/search/?scope=fulltext&page=1&query=AUTH:%22Raj%20A%22">Ashish Raj</a><sup>a</sup></div></div><div class="fm-panel small half_rhythm"><div class="fm-authors-info fm-panel hide half_rhythm" id="id757780_ai" style="display:none"><div class="fm-affl"><sup>a</sup>Department of Radiology, Weill Cornell Medical College, New York, NY, USA</div><div><span class="contrib-email" id="__contribid760778">Farras Abdelnour: <a href="mailto:dev@null" data-email="ude.llenroc.dem@6102aaf" class="oemail">ude.llenroc.dem@6102aaf</a></span> </div></div><div class="togglers"><a href="http://europepmc.org/articles/pmc3951650#" class="pmctoggle" rid="id757780_ai">Author information <span>►</span></a> <a href="http://europepmc.org/articles/pmc3951650#" class="pmctoggle" rid="id757780_cpl">Copyright and License information <span>►</span></a></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="fm-cpl-info fm-panel hide half_rhythm" id="id757780_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="http://europepmc.org/pmc/about/copyright.html">Copyright notice</a>  and <a href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a></div><div><a href="http://europepmc.org/articles/pmc3951650#FN1">Publisher's Disclaimer</a></div></div></div><div class="links-box fm-panel whole_rhythm"><div class="small"><div>The publisher's final edited version of this article is available  at <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&retmode=ref&cmd=prlinks&id=24384152" target="pmc_ext" ref="reftype=publisher&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access&amp;rendering-type=normal">Neuroimage</a></div><div>See other articles in PMC that <a href="http://europepmc.org/articles/PMC3951650/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="__abstractid760818" lang="en" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://europepmc.org/articles/pmc3951650#ui-ncbiinpagenav-2" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#ui-ncbiinpagenav3" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__abstractid760818title">Abstract</h2><!--article-meta--><div><p id="P2" class="p p-first-last">The relationship between anatomic connectivity of large-scale brain networks and their functional connectivity is of immense importance and an area of active research. Previous attempts have required complex simulations which model the dynamics of each cortical region, and explore the coupling between regions as derived by anatomic connections. While much insight is gained from these non-linear simulations, they can be computationally taxing tools for predicting functional from anatomic connectivities. Little attention has been paid to linear models. Here we show that a properly designed linear model appears to be superior to previous non-linear approaches in capturing the brain’s long-range second order correlation structure that governs the relationship between anatomic and functional connectivities. We derive a linear network of brain dynamics based on graph diffusion, whereby the diffusing quantity undergoes a random walk on a graph. We test our model using subjects who underwent diffusion MRI and resting state fMRI. The network diffusion model applied to the structural networks largely predicts the correlation structures derived from their fMRI data, to a greater extent than other approaches. The utility of the proposed approach is that it can routinely be used to infer functional correlation from anatomic connectivity. And since it is linear, anatomic connectivity can also be inferred from functional data. The success of our model confirms the linearity of ensemble average signals in the brain, and implies that their long-range correlation structure may percolate within the brain via purely mechanistic processes enacted on its structural connectivity pathways.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">networks, brain connectivity, functional connectivity, structural connectivity</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://europepmc.org/articles/pmc3951650#ui-ncbiinpagenav-2" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#ui-ncbiinpagenav3" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">1. Introduction</h2><p id="P3" class="p p-first">Whole brain connectivity networks or “connectomes” come in two flavors: structural networks extracted from tractography algorithms applied to diffusion MRI (dMRI) [<a href="http://europepmc.org/articles/pmc3951650#R32" rid="R32" class=" bibr popnode">32</a>, <a href="http://europepmc.org/articles/pmc3951650#R38" rid="R38" class=" bibr popnode">38</a>]; and (resting-state) functional networks, inferred from the strength of long-range second order temporal correlation structure of activation signals in various brain regions [<a href="http://europepmc.org/articles/pmc3951650#R14" rid="R14" class=" bibr popnode">14</a>]. Subsequent analysis using ICA [<a href="http://europepmc.org/articles/pmc3951650#R16" rid="R16" class=" bibr popnode">16</a>] or graph clustering techniques [<a href="http://europepmc.org/articles/pmc3951650#R56" rid="R56" class=" bibr popnode">56</a>], indicate the presence of distinct sub-networks, prominently the default mode and salience networks [<a href="http://europepmc.org/articles/pmc3951650#R33" rid="R33" class=" bibr popnode">33</a>]. Diffusion tensor imaging (DTI) has been extensively used as an estimate of structural connectivity [<a href="http://europepmc.org/articles/pmc3951650#R12" rid="R12" class=" bibr popnode">12</a>, <a href="http://europepmc.org/articles/pmc3951650#R13" rid="R13" class=" bibr popnode">13</a>, <a href="http://europepmc.org/articles/pmc3951650#R64" rid="R64" class=" bibr popnode">64</a>]. Probabilistic tractography methods for estimating structural connectivity from DTI have been adopted in the literature, e.g. [<a href="http://europepmc.org/articles/pmc3951650#R38" rid="R38" class=" bibr popnode">38</a>, <a href="http://europepmc.org/articles/pmc3951650#R39" rid="R39" class=" bibr popnode">39</a>]. Both forms of connectivity have experienced great interest from the neuroscience community, as shown in [<a href="http://europepmc.org/articles/pmc3951650#R1" rid="R1" class=" bibr popnode">1</a>, <a href="http://europepmc.org/articles/pmc3951650#R8" rid="R8" class=" bibr popnode">8</a>, <a href="http://europepmc.org/articles/pmc3951650#R34" rid="R34" class=" bibr popnode">34</a>, <a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>, <a href="http://europepmc.org/articles/pmc3951650#R43" rid="R43" class=" bibr popnode">43</a>].</p><p id="P4">A major goal of connectome research is to discover whether, and how, the structural and functional networks of the brain are related - an active area with tremendous interest and wide ramifications in neuroscience and computational biology [<a href="http://europepmc.org/articles/pmc3951650#R15" rid="R15" class=" bibr popnode">15</a>, <a href="http://europepmc.org/articles/pmc3951650#R21" rid="R21" class=" bibr popnode">21</a>, <a href="http://europepmc.org/articles/pmc3951650#R25" rid="R25" class=" bibr popnode">25</a>, <a href="http://europepmc.org/articles/pmc3951650#R30" rid="R30" class=" bibr popnode">30</a>, <a href="http://europepmc.org/articles/pmc3951650#R34" rid="R34" class=" bibr popnode">34</a>, <a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>, <a href="http://europepmc.org/articles/pmc3951650#R35" rid="R35" class=" bibr popnode">35</a>, <a href="http://europepmc.org/articles/pmc3951650#R46" rid="R46" class=" bibr popnode">46</a>]. Previous investigations have relied on non-linear models of cortical activity which were extended to model whole-brain behavior via coupling between regions based on structural connectivity [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>]. Other studies place non-linear oscillators at each cortical location and likewise couple them using anatomic connectivity strength [<a href="http://europepmc.org/articles/pmc3951650#R15" rid="R15" class=" bibr popnode">15</a>, <a href="http://europepmc.org/articles/pmc3951650#R21" rid="R21" class=" bibr popnode">21</a>, <a href="http://europepmc.org/articles/pmc3951650#R25" rid="R25" class=" bibr popnode">25</a>, <a href="http://europepmc.org/articles/pmc3951650#R30" rid="R30" class=" bibr popnode">30</a>]. Since these powerful <em>generative simulation models</em> are only revealed through large scale, fine-grained finite difference stochastic simulations over thousands of time samples, they present a practical challenge for the task of inferring functional connectivity from anatomic. The field has not actively considered linear graph-theoretic dynamic models for this purpose, with a few exceptions described below. Although complex brain dynamics preclude completely linear responses, <em>ensemble-averaged</em> behavior of large connected but individually non-linear neural populations can be quite linear [<a href="http://europepmc.org/articles/pmc3951650#R59" rid="R59" class=" bibr popnode">59</a>].</p><p id="P5">In this paper we (re)introduce a class of linear models capturing the correlation structure of whole brain dynamics at low frequency BOLD levels [<a href="http://europepmc.org/articles/pmc3951650#R29" rid="R29" class=" bibr popnode">29</a>, <a href="http://europepmc.org/articles/pmc3951650#R34" rid="R34" class=" bibr popnode">34</a>, <a href="http://europepmc.org/articles/pmc3951650#R35" rid="R35" class=" bibr popnode">35</a>]. We argue that while local brain dynamics are not linear or stationary [<a href="http://europepmc.org/articles/pmc3951650#R8" rid="R8" class=" bibr popnode">8</a>, <a href="http://europepmc.org/articles/pmc3951650#R42" rid="R42" class=" bibr popnode">42</a>, <a href="http://europepmc.org/articles/pmc3951650#R37" rid="R37" class=" bibr popnode">37</a>], the emergent behavior of <em>long-range steady state 2nd order correlations</em> should be insensitive to detailed local dynamics, and dependent only on the topology of structural networks. Thus, our hypothesis is that linear macroscopic models are sufficient to infer the long-range correlation structure of brain activity, without requiring detailed non-linear simulation models. Specifically, we present a simple, low-dimensional <em>network diffusion model</em> producing accurate description of the structure-function relationship. Network diffusion models random walks on a graph, covering phenomena from image noise removal [<a href="http://europepmc.org/articles/pmc3951650#R67" rid="R67" class=" bibr popnode">67</a>] to Markov random fields [<a href="http://europepmc.org/articles/pmc3951650#R57" rid="R57" class=" bibr popnode">57</a>]. Interestingly, network diffusion successfully captured the progression of misfolded proteins within brain networks, and recapitulated patterns of dementias like Alzheimer’s disease [<a href="http://europepmc.org/articles/pmc3951650#R53" rid="R53" class=" bibr popnode">53</a>]. We hypothesize that resting-state functional relationships between brain regions can be captured by a similar diffusion process applied to the structural network. While the proposed model is linear, similar to [<a href="http://europepmc.org/articles/pmc3951650#R29" rid="R29" class=" bibr popnode">29</a>], we impose constraints modeled after the interaction of the various cortical regions by taking the Laplacian of the connectivity matrix. We test the proposed model using dMRI and fMRI brain scans of healthy subjects, and demonstrate higher structure-function correspondence than other competing methods including neural mass models [<a href="http://europepmc.org/articles/pmc3951650#R23" rid="R23" class=" bibr popnode">23</a>, <a href="http://europepmc.org/articles/pmc3951650#R48" rid="R48" class=" bibr popnode">48</a>, <a href="http://europepmc.org/articles/pmc3951650#R11" rid="R11" class=" bibr popnode">11</a>]. Our work could provide impetus for similar parsimonious approaches in modeling other complex biophysical phenomena.</p><p id="P6" class="p p-last">Our key idea is that functional signals at the spatial and temporal resolution of BOLD signals in brain regions are an ensemble average of millions of neurons, and are therefore governed mainly by the number of neurons firing at any time rather than by the complex behavior of individual neuronal activity. The non-linearities associated in neurons’ individual firing patterns are largely obliterated in the ensemble signal. Thus, the signal correlation between two large connected regions ought to be governed dominantly by linear processes. We show that the simplest linear and purely mechanistic process enacted on the network can reproduce the functional relationship between brain regions. Since functional relationships appear to be enacted on a physical substrate the brains structural connectivity our work implies that the former is a derivative property of brain structure rather than an independent property.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://europepmc.org/articles/pmc3951650#ui-ncbiinpagenav-2" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#ui-ncbiinpagenav3" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">2. Theory</h2><div id="S3" class="sec sec-first"><h3>2.1. Network notation</h3><p id="P7" class="p p-first-last">In a brain network each node represents a gray matter region located on either the neocortex or in deep brain subcortical areas. We define a network 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig1.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig1.jpg"> = (
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig2.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig2.jpg"><em>,</em>
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig3.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig3.jpg">) with a set of <em>N</em> nodes 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig2.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig2.jpg"> = {<em>v<sub>i</sub></em> | <em>i</em> ∈ 1<em>, …,N</em>} and a set of edges given by an ordered node pair 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig3.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig3.jpg"> = {(<em>i, j</em>) | <em>i</em> ∈ 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig2.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig2.jpg"><em>, j</em> ∈ 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig2.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig2.jpg">} [<a href="http://europepmc.org/articles/pmc3951650#R2" rid="R2" class=" bibr popnode">2</a>]. Between any two nodes <em>i</em> and <em>j</em> there is a fiber pathway whose connectivity weight <em>c<sub>i,j</sub></em> ∈ [0<em>,</em>∞) can be measured from dMRI tractography. The structural connectivity matrix <strong>C</strong> = {<em>c<sub>i,j</sub></em> |(<em>i, j</em>) ∈ 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig3.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig3.jpg">} is obtained via anatomical connection probability (ACP), where the matrix elements are obtained as a function of weighted fiber densities between nodes [<a href="http://europepmc.org/articles/pmc3951650#R39" rid="R39" class=" bibr popnode">39</a>]. Although some individual neurons are known to be directional, dMRI does not allow measurement of directionality. Major fiber bundles resolvable by dMRI, especially cortico-cortical pathways are generally bidirectional, having roughly equal number of connections in either direction [<a href="http://europepmc.org/articles/pmc3951650#R3" rid="R3" class=" bibr popnode">3</a>]. We define the <em>connectivity strength</em> or the <em>weighted degree</em> of a node <em>i</em> in this graph as the sum of all connection weights: 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig8.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig8.jpg">.</p></div><div id="S4" class="sec"><h3>2.2. Linear network models</h3><div id="P8"><p>A previous implementation of a linear model for achieving the structure-function correspondence by [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>] is used in this paper as a comparison, following [<a href="http://europepmc.org/articles/pmc3951650#R29" rid="R29" class=" bibr popnode">29</a>] where an <em>i.i.d.</em> Gaussian noise source <em>ξ</em>(<em>n</em>) drives a discretized multivariate autoregressive linear system given as: 
</p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-1-Frame"><nobr><span class="math" id="M1" style="width: 11.288em; display: inline-block;"><span style="display: inline-block; position: relative; width: 7.815em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(1.276em 1000.003em 2.607em -0.46em); top: -2.196em; left: 0.003em;"><span class="mrow" id="MathJax-Span-2"><span class="mi" id="MathJax-Span-3" style="font-family: MathJax_Main; font-weight: bold;">u</span><span class="mo" id="MathJax-Span-4" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-5" style="font-family: MathJax_Math; font-style: italic;">n</span><span class="mo" id="MathJax-Span-6" style="font-family: MathJax_Main; padding-left: 0.234em;">+</span><span class="mn" id="MathJax-Span-7" style="font-family: MathJax_Main; padding-left: 0.234em;">1</span><span class="mo" id="MathJax-Span-8" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-9" style="font-family: MathJax_Main; padding-left: 0.292em;">=</span><span class="mi" id="MathJax-Span-10" style="font-family: MathJax_Main; font-weight: bold; padding-left: 0.292em;">A</span><span class="mspace" id="MathJax-Span-11" style="height: 0.003em; vertical-align: 0.003em; width: 0.177em; display: inline-block; overflow: hidden;"></span><span class="mi" id="MathJax-Span-12" style="font-family: MathJax_Main; font-weight: bold;">u</span><span class="mo" id="MathJax-Span-13" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-14" style="font-family: MathJax_Math; font-style: italic;">n</span><span class="mo" id="MathJax-Span-15" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-16" style="font-family: MathJax_Main; padding-left: 0.234em;">+</span><span class="mi" id="MathJax-Span-17" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.234em;">ξ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mo" id="MathJax-Span-18" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-19" style="font-family: MathJax_Math; font-style: italic;">n</span><span class="mo" id="MathJax-Span-20" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-21" style="font-family: MathJax_Main;">.</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.588em; vertical-align: -0.413em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" display="block" overflow="scroll"><mi mathvariant="bold">u</mi><mo stretchy="false">(</mo><mi>n</mi><mo>+</mo><mn>1</mn><mo stretchy="false">)</mo><mo>=</mo><mi mathvariant="bold">A</mi><mspace width="0.16667em"></mspace><mi mathvariant="bold">u</mi><mo stretchy="false">(</mo><mi>n</mi><mo stretchy="false">)</mo><mo>+</mo><mi>ξ</mi><mo stretchy="false">(</mo><mi>n</mi><mo stretchy="false">)</mo><mo>.</mo></math></script></div><div class="l">(1)</div></div></div><div id="P9"><p>Here vector <strong>u</strong>(<em>n</em>) is the activation signal at time point <em>n</em> of all network nodes corresponding to regions of the brain. The matrix <strong>A</strong> serves to relate the mixing between signals at different nodes, as per <strong>A</strong> = (1−<em>α</em>)<strong>I</strong>+<strong>C</strong>, where <em>α</em> is some leak parameter from the activity of each node, and <strong>C</strong> is the anatomic connectivity matrix described earlier. Since a single “mixing” parameter <em>α</em> cannot access many interesting regimes in the space of linear models, here we modify <strong>A</strong> via two parameters: 
</p><div class="disp-formula" id="FD2"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-2-Frame"><nobr><span class="math" id="M2" style="width: 8.452em; display: inline-block;"><span style="display: inline-block; position: relative; width: 5.848em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(1.276em 1000.003em 2.607em -0.46em); top: -2.196em; left: 0.003em;"><span class="mrow" id="MathJax-Span-23"><span class="mi" id="MathJax-Span-24" style="font-family: MathJax_Main; font-weight: bold;">A</span><span class="mo" id="MathJax-Span-25" style="font-family: MathJax_Main; padding-left: 0.292em;">=</span><span class="mo" id="MathJax-Span-26" style="font-family: MathJax_Main; padding-left: 0.292em;">(</span><span class="mn" id="MathJax-Span-27" style="font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-28" style="font-family: MathJax_Main; padding-left: 0.234em;">−</span><span class="mi" id="MathJax-Span-29" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.234em;">α</span><span class="mo" id="MathJax-Span-30" style="font-family: MathJax_Main;">)</span><span class="mi" id="MathJax-Span-31" style="font-family: MathJax_Main; font-weight: bold;">I</span><span class="mo" id="MathJax-Span-32" style="font-family: MathJax_Main; padding-left: 0.234em;">+</span><span class="mi" id="MathJax-Span-33" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.234em;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mi" id="MathJax-Span-34" style="font-family: MathJax_Main; font-weight: bold;">C</span><span class="mo" id="MathJax-Span-35" style="font-family: MathJax_Main;">.</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.588em; vertical-align: -0.413em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-2"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" display="block" overflow="scroll"><mi mathvariant="bold">A</mi><mo>=</mo><mo stretchy="false">(</mo><mn>1</mn><mo>-</mo><mi>α</mi><mo stretchy="false">)</mo><mi mathvariant="bold">I</mi><mo>+</mo><mi>β</mi><mi mathvariant="bold">C</mi><mo>.</mo></math></script></div></div></div><p id="P10" class="p p-last">By allowing two degrees of freedom instead of one, we obtain a broader range of linear models than the one proposed by Galán. In order for the simulation to be stable, the matrix <strong>A</strong> is normalized to have unit norm, or ||<strong>A</strong>|| = 1. Following [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>], resting state functional connectivity was obtained via stochastic discrete-time simulation over a range of <em>α</em> ∈ [−3, 3] and <em>β</em> ∈ [0, 6] in steps of 0.1 for both parameters. At each point (<em>α, β</em>), the <img src="./Network_Diffusion_Brain_files/x2113.gif" border="0" alt="[ell]" title=""><sub>1</sub> error with respect to true functional connectivity was computed. We chose the (<em>α, β</em>) pair that gives the smallest error for final computation of functional connectivity.</p></div><div id="S5" class="sec"><h3>2.3. Non-linear neural mass models (NMMs)</h3><p id="P11" class="p p-first">NMMs model neural activity in localized populations (minicolumns) in terms of second order state-space differential equations, where the post-synaptic potential (PSP) of neuronal populations is the hidden state, and the activation signal, whether measured via EEG, MEG or BOLD, is the output variables. The model gives rise to systems of coupled second order non-linear differential equations, whose coupling coefficients are determined by the amount of connectivity between them, which is not known <em>a priori</em>. Since no closed-form solution exists for these equations, the model is a <em>simulated generative model</em>, whose behavior is accessed via large-scale simulations over thousands of time points, starting from stochastic endogenous and exogenous signals representing mean firing rates.</p><p id="P12" class="p p-last">An NMM defined in terms of voltages and conductances was utilized [<a href="http://europepmc.org/articles/pmc3951650#R11" rid="R11" class=" bibr popnode">11</a>], and applied to networks ranging from 66 to 1000 nodes. In a more complex recent model, a set of coupled NMMs were instantiated at each node of a connected brain network, with inter-regional couplings determined by anatomic connectivity [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>]. In the proposed model, inter-regional coupling is modulated by a single coupling parameter <em>c</em>, whose chosen value greatly affects the behavior of this highly non-linear coupled system. Here we implement this approach using original computer code used in [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>]. Values of <em>c</em> were varied over a range <em>c</em> = {0.02, 0.07, 0.12, 0.17, 0.22, 0.27, 0.32} for each subject and the value yielding the highest match with empirical functional connectivity was chosen.</p></div><div id="S6" class="sec"><h3>2.4. Proposed network diffusion model</h3><p id="P13" class="p p-first">We now introduce from first principles a physically realistic linear dynamic network model of functional connectivity relying on its emergent linearity, and obtain a closed-form solution which obviates the need for generating simulated signals. Consider first an isolated cortical region <em>R</em>1. We assume that the average activation signal over all neurons in this region, denoted by <em>x</em><sub>1</sub>(<em>t</em>), is proportional to the number of firing neurons per voxel (rather than to the actual action potentials thereof). Although the internal dynamics of this isolated neural population is complex and likely chaotic, in keeping with our emphasis on simple linear models, we allow the simplest possible dynamic behavior of a damped system, given by d<em>x</em><sub>1</sub>(<em>t</em>)<em>/</em>d<em>t</em> = −<em>βx</em><sub>1</sub>(<em>t</em>). This behavior is consistent with a highly damped system whose impulse response to transient signals dies away as an exponential decay, whose rate is controlled by the decay rate <em>β</em>. This behavior could arise by a number of mechanisms; for instance the refractory period after neural discharge which effectively acts as a damping function on the neural activation signal.</p><p id="P14">Now we expand the model to cover an isolated <em>pair</em> of cortical regions <em>R</em>1 and <em>R</em>2 connected by a single fiber population, whose connectivity weight is given by <em>c</em><sub>1</sub><em><sub>,</sub></em><sub>2</sub>. The number of firing neurons in <em>R</em>2 is <em>V</em><sub>2</sub><em>x</em><sub>2</sub>, where <em>V</em><sub>2</sub> is the number of voxels in <em>R</em>2. Of these, the number of axonal projections from <em>R</em>2 to <em>R</em>1 is proportional to 
<span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-3-Frame" role="textbox" aria-readonly="true"><nobr><span class="math" id="M3" style="width: 3.591em; display: inline-block;"><span style="display: inline-block; position: relative; width: 2.491em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(0.929em 1000.003em 2.491em -0.46em); top: -1.791em; left: 0.003em;"><span class="mrow" id="MathJax-Span-37"><span class="msub" id="MathJax-Span-38"><span style="display: inline-block; position: relative; width: 1.045em; height: 0px;"><span style="position: absolute; clip: rect(1.45em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-39" style="font-family: MathJax_Math; font-style: italic;">c</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.35em;"><span class="mrow" id="MathJax-Span-40"><span class="mn" id="MathJax-Span-41" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-42" style="font-size: 70.7%; font-family: MathJax_Main;">,</span><span class="mn" id="MathJax-Span-43" style="font-size: 70.7%; font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mstyle" id="MathJax-Span-44" style="padding-left: 0.177em;"><span class="mrow" id="MathJax-Span-45"><span class="mfrac" id="MathJax-Span-46" style="padding-left: 0.119em; padding-right: 0.119em;"><span style="display: inline-block; position: relative; width: 0.524em; height: 0px;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.028em -0.46em); top: -2.196em; left: 50%; margin-left: -0.055em;"><span class="mn" id="MathJax-Span-47" style="font-size: 50%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 1.855em;"></span></span><span style="position: absolute; clip: rect(3.475em 1000.003em 4.343em -0.518em); top: -3.643em; left: 50%; margin-left: -0.171em;"><span class="msub" id="MathJax-Span-48"><span style="display: inline-block; position: relative; width: 0.408em; height: 0px;"><span style="position: absolute; clip: rect(1.276em 1000.003em 1.97em -0.518em); top: -1.791em; left: 0.003em;"><span class="mi" id="MathJax-Span-49" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">δ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 1.797em;"></span></span><span style="position: absolute; top: -1.675em; left: 0.177em;"><span class="mn" id="MathJax-Span-50" style="font-size: 50%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 1.855em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.996em;"></span></span><span style="position: absolute; clip: rect(0.871em 1000.003em 1.276em -0.518em); top: -1.27em; left: 0.003em;"><span style="border-left-width: 0.524em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.25px; vertical-align: 0.003em;"></span><span style="display: inline-block; width: 0px; height: 1.102em;"></span></span></span></span></span></span><span class="msub" id="MathJax-Span-51" style="padding-left: 0.177em;"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-52" style="font-family: MathJax_Math; font-style: italic;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mn" id="MathJax-Span-53" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="msub" id="MathJax-Span-54"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px;"><span style="position: absolute; clip: rect(1.45em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-55" style="font-family: MathJax_Math; font-style: italic;">x</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mn" id="MathJax-Span-56" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 1.797em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.921em; vertical-align: -0.829em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-3"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" overflow="scroll"><msub><mi>c</mi><mrow><mn>1</mn><mo>,</mo><mn>2</mn></mrow></msub><mstyle scriptlevel="1"><mfrac><mn>1</mn><msub><mi>δ</mi><mn>2</mn></msub></mfrac></mstyle><msub><mi>V</mi><mn>2</mn></msub><msub><mi>x</mi><mn>2</mn></msub></math></script>, where we divide the connectivity by the degree of <em>R</em>2, <em>δ</em><sub>2</sub>, to get a ratio. The proportion of neurons in <em>R</em>1 which experience a firing afferent from <em>R</em>2, assuming uniform mixing of afferents, is then given by 
<span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-4-Frame" role="textbox" aria-readonly="true"><nobr><span class="math" id="M4" style="width: 4.517em; display: inline-block;"><span style="display: inline-block; position: relative; width: 3.128em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(0.929em 1000.003em 2.491em -0.402em); top: -1.791em; left: 0.003em;"><span class="mrow" id="MathJax-Span-58"><span class="mstyle" id="MathJax-Span-59"><span class="mrow" id="MathJax-Span-60"><span class="mfrac" id="MathJax-Span-61" style="padding-left: 0.119em; padding-right: 0.119em;"><span style="display: inline-block; position: relative; width: 0.524em; height: 0px;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.028em -0.46em); top: -2.196em; left: 50%; margin-left: -0.055em;"><span class="mn" id="MathJax-Span-62" style="font-size: 50%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 1.855em;"></span></span><span style="position: absolute; clip: rect(3.475em 1000.003em 4.343em -0.518em); top: -3.643em; left: 50%; margin-left: -0.229em;"><span class="msub" id="MathJax-Span-63"><span style="display: inline-block; position: relative; width: 0.408em; height: 0px;"><span style="position: absolute; clip: rect(1.276em 1000.003em 1.97em -0.518em); top: -1.791em; left: 0.003em;"><span class="mi" id="MathJax-Span-64" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.119em;"></span></span><span style="display: inline-block; width: 0px; height: 1.797em;"></span></span><span style="position: absolute; top: -1.675em; left: 0.177em;"><span class="mn" id="MathJax-Span-65" style="font-size: 50%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 1.855em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.996em;"></span></span><span style="position: absolute; clip: rect(0.871em 1000.003em 1.276em -0.518em); top: -1.27em; left: 0.003em;"><span style="border-left-width: 0.524em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.25px; vertical-align: 0.003em;"></span><span style="display: inline-block; width: 0px; height: 1.102em;"></span></span></span></span></span></span><span class="msub" id="MathJax-Span-66" style="padding-left: 0.177em;"><span style="display: inline-block; position: relative; width: 1.045em; height: 0px;"><span style="position: absolute; clip: rect(1.45em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-67" style="font-family: MathJax_Math; font-style: italic;">c</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.35em;"><span class="mrow" id="MathJax-Span-68"><span class="mn" id="MathJax-Span-69" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-70" style="font-size: 70.7%; font-family: MathJax_Main;">,</span><span class="mn" id="MathJax-Span-71" style="font-size: 70.7%; font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mstyle" id="MathJax-Span-72" style="padding-left: 0.177em;"><span class="mrow" id="MathJax-Span-73"><span class="mfrac" id="MathJax-Span-74" style="padding-left: 0.119em; padding-right: 0.119em;"><span style="display: inline-block; position: relative; width: 0.524em; height: 0px;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.028em -0.46em); top: -2.196em; left: 50%; margin-left: -0.055em;"><span class="mn" id="MathJax-Span-75" style="font-size: 50%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 1.855em;"></span></span><span style="position: absolute; clip: rect(3.475em 1000.003em 4.343em -0.518em); top: -3.643em; left: 50%; margin-left: -0.171em;"><span class="msub" id="MathJax-Span-76"><span style="display: inline-block; position: relative; width: 0.408em; height: 0px;"><span style="position: absolute; clip: rect(1.276em 1000.003em 1.97em -0.518em); top: -1.791em; left: 0.003em;"><span class="mi" id="MathJax-Span-77" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">δ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 1.797em;"></span></span><span style="position: absolute; top: -1.675em; left: 0.177em;"><span class="mn" id="MathJax-Span-78" style="font-size: 50%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 1.855em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.996em;"></span></span><span style="position: absolute; clip: rect(0.871em 1000.003em 1.276em -0.518em); top: -1.27em; left: 0.003em;"><span style="border-left-width: 0.524em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.25px; vertical-align: 0.003em;"></span><span style="display: inline-block; width: 0px; height: 1.102em;"></span></span></span></span></span></span><span class="msub" id="MathJax-Span-79" style="padding-left: 0.177em;"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-80" style="font-family: MathJax_Math; font-style: italic;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mn" id="MathJax-Span-81" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="msub" id="MathJax-Span-82"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px;"><span style="position: absolute; clip: rect(1.45em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-83" style="font-family: MathJax_Math; font-style: italic;">x</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mn" id="MathJax-Span-84" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 1.797em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.921em; vertical-align: -0.829em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-4"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M4" overflow="scroll"><mstyle scriptlevel="1"><mfrac><mn>1</mn><msub><mi>V</mi><mn>1</mn></msub></mfrac></mstyle><msub><mi>c</mi><mrow><mn>1</mn><mo>,</mo><mn>2</mn></mrow></msub><mstyle scriptlevel="1"><mfrac><mn>1</mn><msub><mi>δ</mi><mn>2</mn></msub></mfrac></mstyle><msub><mi>V</mi><mn>2</mn></msub><msub><mi>x</mi><mn>2</mn></msub></math></script>. If there are no other afferents into <em>R</em>1, the number of neurons which undergo activation secondary to depolarization due to the enervating active neurons from <em>R</em>2, followed by super-threshold post-synaptic integration, will in general depend in a highly complicated and non-linear fashion, well-characterized by the neural mass equations. However, under the emergent linear assumption, the net change in the number of firing neurons in <em>R</em>1 may be considered a linear proportion of the number of active enervating neurons from <em>R</em>2. After accounting for the internal first order dynamics of <em>R</em>1, this gives</p><div class="disp-formula" id="FD3"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-5-Frame"><nobr><span class="math" id="M5" style="width: 14.76em; display: inline-block;"><span style="display: inline-block; position: relative; width: 10.246em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(1.276em 1000.003em 4.054em -0.402em); top: -2.948em; left: 0.003em;"><span class="mrow" id="MathJax-Span-86"><span class="mfrac" id="MathJax-Span-87" style="padding-left: 0.119em; padding-right: 0.119em;"><span style="display: inline-block; position: relative; width: 1.855em; height: 0px;"><span style="position: absolute; clip: rect(1.276em 1000.003em 2.607em -0.46em); top: -2.948em; left: 50%; margin-left: -0.865em;"><span class="mrow" id="MathJax-Span-88"><span class="mi" id="MathJax-Span-89" style="font-family: MathJax_Main;">d</span><span class="msub" id="MathJax-Span-90"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px;"><span style="position: absolute; clip: rect(1.45em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-91" style="font-family: MathJax_Math; font-style: italic;">x</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mn" id="MathJax-Span-92" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-93" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-94" style="font-family: MathJax_Math; font-style: italic;">t</span><span class="mo" id="MathJax-Span-95" style="font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; clip: rect(1.334em 1000.003em 2.376em -0.46em); top: -1.502em; left: 50%; margin-left: -0.344em;"><span class="mrow" id="MathJax-Span-96"><span class="mi" id="MathJax-Span-97" style="font-family: MathJax_Main;">d</span><span class="mi" id="MathJax-Span-98" style="font-family: MathJax_Math; font-style: italic;">t</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; clip: rect(0.871em 1000.003em 1.276em -0.518em); top: -1.328em; left: 0.003em;"><span style="border-left-width: 1.855em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.25px; vertical-align: 0.003em;"></span><span style="display: inline-block; width: 0px; height: 1.102em;"></span></span></span></span><span class="mo" id="MathJax-Span-99" style="font-family: MathJax_Main; padding-left: 0.292em;">=</span><span class="mi" id="MathJax-Span-100" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.292em;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mspace" id="MathJax-Span-101" style="height: 0.003em; vertical-align: 0.003em; width: 0.177em; display: inline-block; overflow: hidden;"></span><span class="mrow" id="MathJax-Span-102"><span class="mo" id="MathJax-Span-103" style="vertical-align: 0.003em;"><span style="font-family: MathJax_Size3;">(</span></span><span class="mrow" id="MathJax-Span-104"><span class="mfrac" id="MathJax-Span-105" style="padding-left: 0.119em; padding-right: 0.119em;"><span style="display: inline-block; position: relative; width: 0.871em; height: 0px;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.376em -0.46em); top: -2.891em; left: 50%; margin-left: -0.171em;"><span class="mn" id="MathJax-Span-106" style="font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; clip: rect(3.128em 1000.003em 4.343em -0.46em); top: -3.296em; left: 50%; margin-left: -0.402em;"><span class="msub" id="MathJax-Span-107"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-108" style="font-family: MathJax_Math; font-style: italic;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mn" id="MathJax-Span-109" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.996em;"></span></span><span style="position: absolute; clip: rect(0.871em 1000.003em 1.276em -0.518em); top: -1.328em; left: 0.003em;"><span style="border-left-width: 0.871em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.25px; vertical-align: 0.003em;"></span><span style="display: inline-block; width: 0px; height: 1.102em;"></span></span></span></span><span class="msub" id="MathJax-Span-110" style="padding-left: 0.177em;"><span style="display: inline-block; position: relative; width: 1.045em; height: 0px;"><span style="position: absolute; clip: rect(1.45em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-111" style="font-family: MathJax_Math; font-style: italic;">c</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.35em;"><span class="mrow" id="MathJax-Span-112"><span class="mn" id="MathJax-Span-113" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-114" style="font-size: 70.7%; font-family: MathJax_Main;">,</span><span class="mn" id="MathJax-Span-115" style="font-size: 70.7%; font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mfrac" id="MathJax-Span-116" style="padding-left: 0.292em; padding-right: 0.119em;"><span style="display: inline-block; position: relative; width: 0.813em; height: 0px;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.376em -0.46em); top: -2.891em; left: 50%; margin-left: -0.171em;"><span class="mn" id="MathJax-Span-117" style="font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; clip: rect(3.128em 1000.003em 4.343em -0.46em); top: -3.296em; left: 50%; margin-left: -0.344em;"><span class="msub" id="MathJax-Span-118"><span style="display: inline-block; position: relative; width: 0.697em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-119" style="font-family: MathJax_Math; font-style: italic;">δ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.35em;"><span class="mn" id="MathJax-Span-120" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.996em;"></span></span><span style="position: absolute; clip: rect(0.871em 1000.003em 1.276em -0.518em); top: -1.328em; left: 0.003em;"><span style="border-left-width: 0.813em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.25px; vertical-align: 0.003em;"></span><span style="display: inline-block; width: 0px; height: 1.102em;"></span></span></span></span><span class="msub" id="MathJax-Span-121" style="padding-left: 0.177em;"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-122" style="font-family: MathJax_Math; font-style: italic;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mn" id="MathJax-Span-123" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="msub" id="MathJax-Span-124"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px;"><span style="position: absolute; clip: rect(1.45em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-125" style="font-family: MathJax_Math; font-style: italic;">x</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mn" id="MathJax-Span-126" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-127" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-128" style="font-family: MathJax_Math; font-style: italic;">t</span><span class="mo" id="MathJax-Span-129" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-130" style="font-family: MathJax_Main; padding-left: 0.234em;">−</span><span class="msub" id="MathJax-Span-131" style="padding-left: 0.234em;"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px;"><span style="position: absolute; clip: rect(1.45em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-132" style="font-family: MathJax_Math; font-style: italic;">x</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mn" id="MathJax-Span-133" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-134" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-135" style="font-family: MathJax_Math; font-style: italic;">t</span><span class="mo" id="MathJax-Span-136" style="font-family: MathJax_Main;">)</span></span><span class="mo" id="MathJax-Span-137" style="vertical-align: 0.003em;"><span style="font-family: MathJax_Size3;">)</span></span></span></span><span style="display: inline-block; width: 0px; height: 2.954em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 3.671em; vertical-align: -1.412em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-5"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M5" display="block" overflow="scroll"><mfrac><mrow><mi mathvariant="normal">d</mi><msub><mi>x</mi><mn>1</mn></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow><mrow><mi mathvariant="normal">d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mi>β</mi><mspace width="0.16667em"></mspace><mrow><mo>(</mo><mrow><mfrac><mn>1</mn><msub><mi>V</mi><mn>1</mn></msub></mfrac><msub><mi>c</mi><mrow><mn>1</mn><mo>,</mo><mn>2</mn></mrow></msub><mfrac><mn>1</mn><msub><mi>δ</mi><mn>2</mn></msub></mfrac><msub><mi>V</mi><mn>2</mn></msub><msub><mi>x</mi><mn>2</mn></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>-</mo><msub><mi>x</mi><mn>1</mn></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow><mo>)</mo></mrow></math></script></div><div class="l">(2)</div></div><p id="P15">There is, of course, no reason why the rate constant <em>β</em> should be identical for both the internal and external signals contributing to the dynamics of <em>R</em>1, but in the interest of simplicity and in the absence of evidence to the contrary we have assumed identical rates. This then is the first order dynamics of an isolated pair of neuronal populations. For multiple afferents into <em>R</em>1, we modify this to</p><div class="disp-formula" id="FD4"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-6-Frame"><nobr><span class="math" id="M6" style="width: 15.917em; display: inline-block;"><span style="display: inline-block; position: relative; width: 11.056em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(1.276em 1000.003em 4.806em -0.402em); top: -3.296em; left: 0.003em;"><span class="mrow" id="MathJax-Span-139"><span class="mfrac" id="MathJax-Span-140" style="padding-left: 0.119em; padding-right: 0.119em;"><span style="display: inline-block; position: relative; width: 1.739em; height: 0px;"><span style="position: absolute; clip: rect(1.276em 1000.003em 2.607em -0.46em); top: -2.948em; left: 50%; margin-left: -0.807em;"><span class="mrow" id="MathJax-Span-141"><span class="mi" id="MathJax-Span-142" style="font-family: MathJax_Main;">d</span><span class="msub" id="MathJax-Span-143"><span style="display: inline-block; position: relative; width: 0.639em; height: 0px;"><span style="position: absolute; clip: rect(1.45em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-144" style="font-family: MathJax_Math; font-style: italic;">x</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mi" id="MathJax-Span-145" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-146" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-147" style="font-family: MathJax_Math; font-style: italic;">t</span><span class="mo" id="MathJax-Span-148" style="font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; clip: rect(1.334em 1000.003em 2.376em -0.46em); top: -1.502em; left: 50%; margin-left: -0.344em;"><span class="mrow" id="MathJax-Span-149"><span class="mi" id="MathJax-Span-150" style="font-family: MathJax_Main;">d</span><span class="mi" id="MathJax-Span-151" style="font-family: MathJax_Math; font-style: italic;">t</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; clip: rect(0.871em 1000.003em 1.276em -0.518em); top: -1.328em; left: 0.003em;"><span style="border-left-width: 1.739em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.25px; vertical-align: 0.003em;"></span><span style="display: inline-block; width: 0px; height: 1.102em;"></span></span></span></span><span class="mo" id="MathJax-Span-152" style="font-family: MathJax_Main; padding-left: 0.292em;">=</span><span class="mi" id="MathJax-Span-153" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.292em;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mspace" id="MathJax-Span-154" style="height: 0.003em; vertical-align: 0.003em; width: 0.177em; display: inline-block; overflow: hidden;"></span><span class="mrow" id="MathJax-Span-155"><span class="mo" id="MathJax-Span-156" style="vertical-align: 1.855em;"><span style="display: inline-block; position: relative; width: 0.871em; height: 0px;"><span style="position: absolute; font-family: MathJax_Size4; top: -2.833em; left: 0.003em;">⎛<span style="display: inline-block; width: 0px; height: 3.996em;"></span></span><span style="position: absolute; font-family: MathJax_Size4; top: -1.502em; left: 0.003em;">⎝<span style="display: inline-block; width: 0px; height: 3.996em;"></span></span></span></span><span class="mrow" id="MathJax-Span-157"><span class="mfrac" id="MathJax-Span-158" style="padding-left: 0.119em; padding-right: 0.119em;"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.376em -0.46em); top: -2.891em; left: 50%; margin-left: -0.171em;"><span class="mn" id="MathJax-Span-159" style="font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; clip: rect(3.128em 1000.003em 4.343em -0.46em); top: -3.296em; left: 50%; margin-left: -0.344em;"><span class="msub" id="MathJax-Span-160"><span style="display: inline-block; position: relative; width: 0.639em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-161" style="font-family: MathJax_Math; font-style: italic;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mi" id="MathJax-Span-162" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.996em;"></span></span><span style="position: absolute; clip: rect(0.871em 1000.003em 1.276em -0.518em); top: -1.328em; left: 0.003em;"><span style="border-left-width: 0.755em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.25px; vertical-align: 0.003em;"></span><span style="display: inline-block; width: 0px; height: 1.102em;"></span></span></span></span><span class="munder" id="MathJax-Span-163" style="padding-left: 0.177em;"><span style="display: inline-block; position: relative; width: 1.045em; height: 0px;"><span style="position: absolute; clip: rect(1.739em 1000.003em 3.475em -0.46em); top: -2.833em; left: 0.003em;"><span class="mo" id="MathJax-Span-164" style="font-family: MathJax_Size2; vertical-align: 0.003em;">∑</span><span style="display: inline-block; width: 0px; height: 2.839em;"></span></span><span style="position: absolute; clip: rect(1.276em 1000.003em 2.318em -0.518em); top: -0.807em; left: 0.408em;"><span class="mi" id="MathJax-Span-165" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mrow" id="MathJax-Span-166" style="padding-left: 0.177em;"><span class="msub" id="MathJax-Span-167"><span style="display: inline-block; position: relative; width: 0.929em; height: 0px;"><span style="position: absolute; clip: rect(1.45em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-168" style="font-family: MathJax_Math; font-style: italic;">c</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.35em;"><span class="mrow" id="MathJax-Span-169"><span class="mi" id="MathJax-Span-170" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mo" id="MathJax-Span-171" style="font-size: 70.7%; font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-172" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mfrac" id="MathJax-Span-173" style="padding-left: 0.292em; padding-right: 0.119em;"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.376em -0.46em); top: -2.891em; left: 50%; margin-left: -0.171em;"><span class="mn" id="MathJax-Span-174" style="font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; clip: rect(3.128em 1000.003em 4.459em -0.46em); top: -3.296em; left: 50%; margin-left: -0.344em;"><span class="msub" id="MathJax-Span-175"><span style="display: inline-block; position: relative; width: 0.639em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-176" style="font-family: MathJax_Math; font-style: italic;">δ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.35em;"><span class="mi" id="MathJax-Span-177" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.996em;"></span></span><span style="position: absolute; clip: rect(0.871em 1000.003em 1.276em -0.518em); top: -1.328em; left: 0.003em;"><span style="border-left-width: 0.755em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.25px; vertical-align: 0.003em;"></span><span style="display: inline-block; width: 0px; height: 1.102em;"></span></span></span></span><span class="msub" id="MathJax-Span-178" style="padding-left: 0.177em;"><span style="display: inline-block; position: relative; width: 0.697em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-179" style="font-family: MathJax_Math; font-style: italic;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mi" id="MathJax-Span-180" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="msub" id="MathJax-Span-181"><span style="display: inline-block; position: relative; width: 0.697em; height: 0px;"><span style="position: absolute; clip: rect(1.45em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-182" style="font-family: MathJax_Math; font-style: italic;">x</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mi" id="MathJax-Span-183" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-184" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-185" style="font-family: MathJax_Math; font-style: italic;">t</span><span class="mo" id="MathJax-Span-186" style="font-family: MathJax_Main;">)</span></span><span class="mo" id="MathJax-Span-187" style="font-family: MathJax_Main; padding-left: 0.234em;">−</span><span class="msub" id="MathJax-Span-188" style="padding-left: 0.234em;"><span style="display: inline-block; position: relative; width: 0.639em; height: 0px;"><span style="position: absolute; clip: rect(1.45em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-189" style="font-family: MathJax_Math; font-style: italic;">x</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mi" id="MathJax-Span-190" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-191" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-192" style="font-family: MathJax_Math; font-style: italic;">t</span><span class="mo" id="MathJax-Span-193" style="font-family: MathJax_Main;">)</span></span><span class="mo" id="MathJax-Span-194" style="vertical-align: 1.855em;"><span style="display: inline-block; position: relative; width: 0.871em; height: 0px;"><span style="position: absolute; font-family: MathJax_Size4; top: -2.833em; left: 0.003em;">⎞<span style="display: inline-block; width: 0px; height: 3.996em;"></span></span><span style="position: absolute; font-family: MathJax_Size4; top: -1.502em; left: 0.003em;">⎠<span style="display: inline-block; width: 0px; height: 3.996em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-195" style="font-family: MathJax_Main;">.</span></span><span style="display: inline-block; width: 0px; height: 3.302em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 4.754em; vertical-align: -1.996em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-6"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M6" display="block" overflow="scroll"><mfrac><mrow><mi mathvariant="normal">d</mi><msub><mi>x</mi><mi>i</mi></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow><mrow><mi mathvariant="normal">d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mi>β</mi><mspace width="0.16667em"></mspace><mrow><mo>(</mo><mrow><mfrac><mn>1</mn><msub><mi>V</mi><mi>i</mi></msub></mfrac><munder><mo>∑</mo><mi>j</mi></munder><mrow><msub><mi>c</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mfrac><mn>1</mn><msub><mi>δ</mi><mi>j</mi></msub></mfrac><msub><mi>V</mi><mi>j</mi></msub><msub><mi>x</mi><mi>j</mi></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow><mo>-</mo><msub><mi>x</mi><mi>i</mi></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow><mo>)</mo></mrow><mo>.</mo></math></script></div><div class="l">(3)</div></div><p id="P16">Since the regional parcellation in fMRI-based functional networks is somewhat arbitrary, the regional volumes <em>V<sub>i</sub></em> are not germane to the model, and must be replaced by graph quantities since our goal is a graph model for this dynamics. Clearly, the regional degree <em>δ<sub>i</sub></em> are closely related to the volumes <em>V<sub>i</sub></em>, and for cortical sheets, the relationship will be roughly linear. Unfortunately, limitations of connectivity and regional volume measurements, combined with the need to integrate both cortical and subcortical nuclei, precludes a straight-forward relationship. Thus, for the purpose of this paper we propose two simple alternatives: linear: <em>V<sub>i</sub></em> ∝ <em>δ<sub>i</sub></em>, and sub-linear: 
<span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-7-Frame" role="textbox" aria-readonly="true"><nobr><span class="math" id="M7" style="width: 3.302em; display: inline-block;"><span style="display: inline-block; position: relative; width: 2.26em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(1.16em 1000.003em 2.665em -0.46em); top: -2.196em; left: 0.003em;"><span class="mrow" id="MathJax-Span-197"><span class="msub" id="MathJax-Span-198"><span style="display: inline-block; position: relative; width: 0.639em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-199" style="font-family: MathJax_Math; font-style: italic;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mi" id="MathJax-Span-200" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-201" style="font-family: MathJax_Main; padding-left: 0.292em;">∝</span><span class="msqrt" id="MathJax-Span-202" style="padding-left: 0.292em;"><span style="display: inline-block; position: relative; width: 1.276em; height: 0px;"><span style="position: absolute; clip: rect(3.128em 1000.003em 4.343em -0.46em); top: -3.99em; left: 0.697em;"><span class="mrow" id="MathJax-Span-203"><span class="msub" id="MathJax-Span-204"><span style="display: inline-block; position: relative; width: 0.582em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-205" style="font-family: MathJax_Math; font-style: italic;">δ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.35em;"><span class="mi" id="MathJax-Span-206" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.996em;"></span></span><span style="position: absolute; clip: rect(0.929em 1000.003em 1.334em -0.518em); top: -1.965em; left: 0.697em;"><span style="border-left-width: 0.582em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.25px; vertical-align: -0.055em;"></span><span style="display: inline-block; width: 0px; height: 1.102em;"></span></span><span style="position: absolute; clip: rect(2.954em 1000.003em 4.517em -0.402em); top: -4.048em; left: 0.003em;"><span style="font-family: MathJax_Size1;">√</span><span style="display: inline-block; width: 0px; height: 3.996em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.921em; vertical-align: -0.496em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-7"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M7" overflow="scroll"><msub><mi>V</mi><mi>i</mi></msub><mo>∝</mo><msqrt><msub><mi>δ</mi><mi>i</mi></msub></msqrt></math></script>. The former is preferable if considering only the cortical sheet, the latter if considering subcortical nuclei as well (since their size is disproportionally lower than their inter-regional connectivity would suggest).</p><div id="P65"><p>After substituting regional volumes by their respective degree and concatenating over <em>i</em>, <a href="http://europepmc.org/articles/pmc3951650#FD4" rid="FD4" class=" disp-formula">Eq. 3</a> easily expands to the entire network with arbitrary topology:
</p><div class="disp-formula" id="FD5"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-8-Frame"><nobr><span class="math" id="M8" style="width: 7.7em; display: inline-block;"><span style="display: inline-block; position: relative; width: 5.327em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(0.582em 1000.003em 3.128em -0.402em); top: -2.254em; left: 0.003em;"><span class="mrow" id="MathJax-Span-208"><span class="mfrac" id="MathJax-Span-209" style="padding-left: 0.119em; padding-right: 0.119em;"><span style="display: inline-block; position: relative; width: 1.739em; height: 0px;"><span style="position: absolute; clip: rect(1.276em 1000.003em 2.607em -0.46em); top: -2.948em; left: 50%; margin-left: -0.807em;"><span class="mrow" id="MathJax-Span-210"><span class="mi" id="MathJax-Span-211" style="font-family: MathJax_Main;">d</span><span class="mi" id="MathJax-Span-212" style="font-family: MathJax_Main; font-weight: bold;">x</span><span class="mo" id="MathJax-Span-213" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-214" style="font-family: MathJax_Math; font-style: italic;">t</span><span class="mo" id="MathJax-Span-215" style="font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; clip: rect(1.334em 1000.003em 2.376em -0.46em); top: -1.502em; left: 50%; margin-left: -0.344em;"><span class="mrow" id="MathJax-Span-216"><span class="mi" id="MathJax-Span-217" style="font-family: MathJax_Main;">d</span><span class="mi" id="MathJax-Span-218" style="font-family: MathJax_Math; font-style: italic;">t</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; clip: rect(0.871em 1000.003em 1.276em -0.518em); top: -1.328em; left: 0.003em;"><span style="border-left-width: 1.739em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.25px; vertical-align: 0.003em;"></span><span style="display: inline-block; width: 0px; height: 1.102em;"></span></span></span></span><span class="mo" id="MathJax-Span-219" style="font-family: MathJax_Main; padding-left: 0.292em;">=</span><span class="mo" id="MathJax-Span-220" style="font-family: MathJax_Main; padding-left: 0.292em;">−</span><span class="mi" id="MathJax-Span-221" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mi" id="MathJax-Span-222" style="font-family: MathJax_Script;">L<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span class="mi" id="MathJax-Span-223" style="font-family: MathJax_Main; font-weight: bold;">x</span><span class="mo" id="MathJax-Span-224" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-225" style="font-family: MathJax_Math; font-style: italic;">t</span><span class="mo" id="MathJax-Span-226" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-227" style="font-family: MathJax_Main;">,</span></span><span style="display: inline-block; width: 0px; height: 2.26em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 3.337em; vertical-align: -1.079em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-8"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M8" display="block" overflow="scroll"><mfrac><mrow><mi mathvariant="normal">d</mi><mi mathvariant="bold">x</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow><mrow><mi mathvariant="normal">d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mo>-</mo><mi>β</mi><mi mathvariant="script">L</mi><mi mathvariant="bold">x</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>,</mo></math></script></div><div class="l">(4)</div></div><p> where the matrix <em>L</em> is the well-known network Laplacian, whose exact form will depend on which of the above two definitions of regional volume is used. For the former definition, we obtain 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig4.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig4.jpg"> = <em>I</em> − Δ<sup>−1</sup><strong>C</strong>, where Δ is the diagonal matrix with <em>δ<sub>i</sub></em> = Σ<em><sub>j</sub>c<sub>i,j</sub></em> as the <em>i</em>th diagonal element. For the latter definition, we have</p></div><div class="disp-formula" id="FD6"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-9-Frame"><nobr><span class="math" id="M9" style="width: 8.683em; display: inline-block;"><span style="display: inline-block; position: relative; width: 6.021em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(1.102em 1000.003em 2.433em -0.518em); top: -2.254em; left: 0.003em;"><span class="mrow" id="MathJax-Span-229"><span class="mi" id="MathJax-Span-230" style="font-family: MathJax_Script;">L<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span class="mo" id="MathJax-Span-231" style="font-family: MathJax_Main; padding-left: 0.292em;">=</span><span class="mi" id="MathJax-Span-232" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.292em;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.061em;"></span></span><span class="mo" id="MathJax-Span-233" style="font-family: MathJax_Main; padding-left: 0.234em;">−</span><span class="msup" id="MathJax-Span-234" style="padding-left: 0.234em;"><span style="display: inline-block; position: relative; width: 1.797em; height: 0px;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.376em -0.46em); top: -2.196em; left: 0.003em;"><span class="mi" id="MathJax-Span-235" style="font-family: MathJax_Main;">Δ</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; top: -2.37em; left: 0.582em;"><span class="mrow" id="MathJax-Span-236"><span class="mo" id="MathJax-Span-237" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mn" id="MathJax-Span-238" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-239" style="font-size: 70.7%; font-family: MathJax_Main;">/</span><span class="mn" id="MathJax-Span-240" style="font-size: 70.7%; font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mi" id="MathJax-Span-241" style="font-family: MathJax_Main; font-weight: bold;">C</span><span class="msup" id="MathJax-Span-242"><span style="display: inline-block; position: relative; width: 1.797em; height: 0px;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.376em -0.46em); top: -2.196em; left: 0.003em;"><span class="mi" id="MathJax-Span-243" style="font-family: MathJax_Main;">Δ</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; top: -2.37em; left: 0.582em;"><span class="mrow" id="MathJax-Span-244"><span class="mo" id="MathJax-Span-245" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mn" id="MathJax-Span-246" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-247" style="font-size: 70.7%; font-family: MathJax_Main;">/</span><span class="mn" id="MathJax-Span-248" style="font-size: 70.7%; font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-249" style="font-family: MathJax_Main;">.</span></span><span style="display: inline-block; width: 0px; height: 2.26em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.588em; vertical-align: -0.079em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-9"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M9" display="block" overflow="scroll"><mi mathvariant="script">L</mi><mo>=</mo><mi>I</mi><mo>-</mo><msup><mi mathvariant="normal">Δ</mi><mrow><mo>-</mo><mn>1</mn><mo>/</mo><mn>2</mn></mrow></msup><mi mathvariant="bold">C</mi><msup><mi mathvariant="normal">Δ</mi><mrow><mo>-</mo><mn>1</mn><mo>/</mo><mn>2</mn></mrow></msup><mo>.</mo></math></script></div><div class="l">(5)</div></div><p id="P17">In the remainder of this paper we use the second definition because it is preferable when including subcortical regions, and empirically gives networks with a closer match to resting state data. On numerical grounds too it is preferable, since it is symmetric and nonnegative definite, as described in previous graph studies [<a href="http://europepmc.org/articles/pmc3951650#R2" rid="R2" class=" bibr popnode">2</a>]. The corresponding eigenvalues are all between 0 and 2.</p><p id="P18">The network diffusion <a href="http://europepmc.org/articles/pmc3951650#FD5" rid="FD5" class=" disp-formula">equation 4</a> has an explicit solution</p><div id="P19"><p>
</p><div class="disp-formula" id="FD7"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-10-Frame"><nobr><span class="math" id="M10" style="width: 9.262em; display: inline-block;"><span style="display: inline-block; position: relative; width: 6.427em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.665em -0.518em); top: -2.254em; left: 0.003em;"><span class="mrow" id="MathJax-Span-251"><span class="mi" id="MathJax-Span-252" style="font-family: MathJax_Main; font-weight: bold;">x</span><span class="mo" id="MathJax-Span-253" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-254" style="font-family: MathJax_Math; font-style: italic;">t</span><span class="mo" id="MathJax-Span-255" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-256" style="font-family: MathJax_Main; padding-left: 0.292em;">=</span><span class="mo" id="MathJax-Span-257" style="font-family: MathJax_Main; padding-left: 0.292em;">exp</span><span class="mo" id="MathJax-Span-258" style="font-family: MathJax_Main;">(</span><span class="mo" id="MathJax-Span-259" style="font-family: MathJax_Main;">−</span><span class="mi" id="MathJax-Span-260" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mi" id="MathJax-Span-261" style="font-family: MathJax_Script;">L<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span class="mi" id="MathJax-Span-262" style="font-family: MathJax_Math; font-style: italic;">t</span><span class="mo" id="MathJax-Span-263" style="font-family: MathJax_Main;">)</span><span class="msub" id="MathJax-Span-264"><span style="display: inline-block; position: relative; width: 0.813em; height: 0px;"><span style="position: absolute; clip: rect(1.565em 1000.003em 2.376em -0.518em); top: -2.196em; left: 0.003em;"><span class="mi" id="MathJax-Span-265" style="font-family: MathJax_Main; font-weight: bold;">x</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.466em;"><span class="mn" id="MathJax-Span-266" style="font-size: 70.7%; font-family: MathJax_Main;">0</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-267" style="font-family: MathJax_Main;">,</span></span><span style="display: inline-block; width: 0px; height: 2.26em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.588em; vertical-align: -0.413em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-10"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M10" display="block" overflow="scroll"><mi mathvariant="bold">x</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>=</mo><mo>exp</mo><mo stretchy="false">(</mo><mo>-</mo><mi>β</mi><mi mathvariant="script">L</mi><mi>t</mi><mo stretchy="false">)</mo><msub><mi mathvariant="bold">x</mi><mn>0</mn></msub><mo>,</mo></math></script></div></div><p> which defines the evolution of the initial configuration <strong>x</strong><sub>0</sub> under subsequent graph diffusion process on 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig1.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig1.jpg">.</p></div></div><div id="S7" class="sec"><h3>2.5. A closed form solution of the graph diffusion model for functional brain networks</h3><p id="P20" class="p p-first">At time <em>t</em> the effect of an initial configuration, with only region <em>i</em> active, is given by <strong>x</strong>(<em>t</em>) = exp(−<em>β</em>
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig4.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig4.jpg"><em>t</em>)<strong>e</strong><em><sub>i</sub></em>, where <strong>e</strong><em><sub>i</sub></em> is the cardinal unit vector in the <em>i</em>th direction. Collecting the configurations due to all regions we obtain</p><div class="disp-formula" id="FD8"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-11-Frame"><nobr><span class="math" id="M11" style="width: 13.95em; display: inline-block;"><span style="display: inline-block; position: relative; width: 9.667em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.665em -0.518em); top: -2.254em; left: 0.003em;"><span class="mrow" id="MathJax-Span-269"><span class="mi" id="MathJax-Span-270" style="font-family: MathJax_Main; font-weight: bold;">x</span><span class="mo" id="MathJax-Span-271" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-272" style="font-family: MathJax_Math; font-style: italic;">t</span><span class="mo" id="MathJax-Span-273" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-274" style="font-family: MathJax_Main; padding-left: 0.292em;">=</span><span class="mo" id="MathJax-Span-275" style="font-family: MathJax_Main; padding-left: 0.292em;">exp</span><span class="mspace" id="MathJax-Span-276" style="height: 0.003em; vertical-align: 0.003em; width: 0.177em; display: inline-block; overflow: hidden;"></span><span class="mo" id="MathJax-Span-277" style="font-family: MathJax_Main;">(</span><span class="mo" id="MathJax-Span-278" style="font-family: MathJax_Main;">−</span><span class="mi" id="MathJax-Span-279" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mi" id="MathJax-Span-280" style="font-family: MathJax_Script;">L<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span class="mi" id="MathJax-Span-281" style="font-family: MathJax_Math; font-style: italic;">t</span><span class="mo" id="MathJax-Span-282" style="font-family: MathJax_Main;">)</span><span class="mspace" id="MathJax-Span-283" style="height: 0.003em; vertical-align: 0.003em; width: 0.177em; display: inline-block; overflow: hidden;"></span><span class="mo" id="MathJax-Span-284" style="font-family: MathJax_Main;">(</span><span class="msub" id="MathJax-Span-285"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px;"><span style="position: absolute; clip: rect(1.565em 1000.003em 2.376em -0.46em); top: -2.196em; left: 0.003em;"><span class="mi" id="MathJax-Span-286" style="font-family: MathJax_Main; font-weight: bold;">e</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mn" id="MathJax-Span-287" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-288" style="font-family: MathJax_Main; padding-left: 0.292em;">∣</span><span class="mo" id="MathJax-Span-289" style="font-family: MathJax_Main; padding-left: 0.292em;">⋯</span><span class="mo" id="MathJax-Span-290" style="font-family: MathJax_Main; padding-left: 0.292em;">∣</span><span class="msub" id="MathJax-Span-291" style="padding-left: 0.292em;"><span style="display: inline-block; position: relative; width: 0.929em; height: 0px;"><span style="position: absolute; clip: rect(1.565em 1000.003em 2.376em -0.46em); top: -2.196em; left: 0.003em;"><span class="mi" id="MathJax-Span-292" style="font-family: MathJax_Main; font-weight: bold;">e</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mi" id="MathJax-Span-293" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.061em;"></span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-294" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-295" style="font-family: MathJax_Main;">.</span></span><span style="display: inline-block; width: 0px; height: 2.26em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.588em; vertical-align: -0.413em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-11"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M11" display="block" overflow="scroll"><mi mathvariant="bold">x</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>=</mo><mo>exp</mo><mspace width="0.16667em"></mspace><mo stretchy="false">(</mo><mo>-</mo><mi>β</mi><mi mathvariant="script">L</mi><mi>t</mi><mo stretchy="false">)</mo><mspace width="0.16667em"></mspace><mo stretchy="false">(</mo><msub><mi mathvariant="bold">e</mi><mn>1</mn></msub><mo>∣</mo><mo>⋯</mo><mo>∣</mo><msub><mi mathvariant="bold">e</mi><mi>N</mi></msub><mo stretchy="false">)</mo><mo>.</mo></math></script></div></div><p id="P21">We hypothesize that the configuration at time <em>t</em> of an initial configuration involving only region <em>i</em> is simply the functional connectivity of <em>i</em> with all other regions. Therefore we obtain</p><div class="disp-formula" id="FD9"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-12-Frame"><nobr><span class="math" id="M12" style="width: 9.031em; display: inline-block;"><span style="display: inline-block; position: relative; width: 6.253em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.723em -0.518em); top: -2.254em; left: 0.003em;"><span class="mrow" id="MathJax-Span-297"><span class="msub" id="MathJax-Span-298"><span style="display: inline-block; position: relative; width: 0.871em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.318em -0.518em); top: -2.138em; left: 0.003em;"><span class="mi" id="MathJax-Span-299" style="font-family: MathJax_Script;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.144em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.466em;"><span class="mi" id="MathJax-Span-300" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.061em;"></span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mspace" id="MathJax-Span-301" style="height: 0.003em; vertical-align: 0.003em; width: 0.177em; display: inline-block; overflow: hidden;"></span><span class="mo" id="MathJax-Span-302" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-303" style="font-family: MathJax_Math; font-style: italic;">t</span><span class="mo" id="MathJax-Span-304" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-305" style="font-family: MathJax_Main; padding-left: 0.292em;">=</span><span class="mo" id="MathJax-Span-306" style="font-family: MathJax_Main; padding-left: 0.292em;">exp</span><span class="mspace" id="MathJax-Span-307" style="height: 0.003em; vertical-align: 0.003em; width: 0.177em; display: inline-block; overflow: hidden;"></span><span class="mo" id="MathJax-Span-308" style="font-family: MathJax_Main;">(</span><span class="mo" id="MathJax-Span-309" style="font-family: MathJax_Main;">−</span><span class="mi" id="MathJax-Span-310" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mi" id="MathJax-Span-311" style="font-family: MathJax_Script;">L<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span class="mi" id="MathJax-Span-312" style="font-family: MathJax_Math; font-style: italic;">t</span><span class="mo" id="MathJax-Span-313" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-314" style="font-family: MathJax_Main;">.</span></span><span style="display: inline-block; width: 0px; height: 2.26em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.671em; vertical-align: -0.496em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-12"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M12" display="block" overflow="scroll"><msub><mi mathvariant="script">C</mi><mi>f</mi></msub><mspace width="0.16667em"></mspace><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>=</mo><mo>exp</mo><mspace width="0.16667em"></mspace><mo stretchy="false">(</mo><mo>-</mo><mi>β</mi><mi mathvariant="script">L</mi><mi>t</mi><mo stretchy="false">)</mo><mo>.</mo></math></script></div></div><p id="P22">Here the functional connectivity matrix 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig5.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig5.jpg"> is shown as a function of network diffusion time. Note that due to the eigen-decomposition of 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig4.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig4.jpg"> described above, we have 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig5.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig5.jpg">(0) = <strong>I</strong>, and 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig5.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig5.jpg">(∞) = <strong>0</strong>. That is, regardless of the underlying structural network (as long as it is not disconnected), if no time is allowed for diffusion, there are simply no connections between brain regions; and in the steady state, all regions are connected to all other regions equally. Between these two extremes, a spectrum of functional networks exist. We hypothesize that the network diffusion time necessary to match the observed functional network will depend on the signal being interrogated (whether BOLD, EEG or MEG) and various details regarding action potentials and their speed of propagation through various neuronal populations. Rather than minutely modeling these parameters, we simply hypothesize that at a critical time constant <em>t<sub>crit</sub></em>, to be determined experimentally, the network 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig5.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig5.jpg">(<em>t<sub>crit</sub></em>) will match the observed functional network, or we have</p><div class="disp-formula" id="FD10"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-13-Frame"><nobr><span class="math" id="M13" style="width: 10.42em; display: inline-block;"><span style="display: inline-block; position: relative; width: 7.237em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.723em -0.518em); top: -2.254em; left: 0.003em;"><span class="mrow" id="MathJax-Span-316"><span class="msub" id="MathJax-Span-317"><span style="display: inline-block; position: relative; width: 0.871em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.318em -0.518em); top: -2.138em; left: 0.003em;"><span class="mi" id="MathJax-Span-318" style="font-family: MathJax_Script;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.144em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.466em;"><span class="mi" id="MathJax-Span-319" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.061em;"></span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mspace" id="MathJax-Span-320" style="height: 0.003em; vertical-align: 0.003em; width: 0.177em; display: inline-block; overflow: hidden;"></span><span class="mo" id="MathJax-Span-321" style="font-family: MathJax_Main;">(</span><span class="msub" id="MathJax-Span-322"><span style="display: inline-block; position: relative; width: 1.16em; height: 0px;"><span style="position: absolute; clip: rect(1.276em 1000.003em 2.26em -0.518em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-323" style="font-family: MathJax_Math; font-style: italic;">t</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.292em;"><span class="mi" id="MathJax-Span-324" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">crit</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-325" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-326" style="font-family: MathJax_Main; padding-left: 0.292em;">=</span><span class="mo" id="MathJax-Span-327" style="font-family: MathJax_Main; padding-left: 0.292em;">exp</span><span class="mo" id="MathJax-Span-328" style="font-family: MathJax_Main;">(</span><span class="mo" id="MathJax-Span-329" style="font-family: MathJax_Main;">−</span><span class="mi" id="MathJax-Span-330" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mi" id="MathJax-Span-331" style="font-family: MathJax_Script;">L<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span class="msub" id="MathJax-Span-332"><span style="display: inline-block; position: relative; width: 1.16em; height: 0px;"><span style="position: absolute; clip: rect(1.276em 1000.003em 2.26em -0.518em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-333" style="font-family: MathJax_Math; font-style: italic;">t</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.292em;"><span class="mi" id="MathJax-Span-334" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">crit</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-335" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-336" style="font-family: MathJax_Main;">.</span></span><span style="display: inline-block; width: 0px; height: 2.26em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.671em; vertical-align: -0.496em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-13"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M13" display="block" overflow="scroll"><msub><mi mathvariant="script">C</mi><mi>f</mi></msub><mspace width="0.16667em"></mspace><mo stretchy="false">(</mo><msub><mi>t</mi><mi mathvariant="italic">crit</mi></msub><mo stretchy="false">)</mo><mo>=</mo><mo>exp</mo><mo stretchy="false">(</mo><mo>-</mo><mi>β</mi><mi mathvariant="script">L</mi><msub><mi>t</mi><mi mathvariant="italic">crit</mi></msub><mo stretchy="false">)</mo><mo>.</mo></math></script></div><div class="l">(6)</div></div><p id="P23">Since the eigen decomposition 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig4.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig4.jpg"> = <em>U</em>∧<em>U<sup>t</sup></em> is dominated by a few very small eigenvalues, the hypothesized functional networks should also be similarly dominated by these eigenmodes. At small diffusion time, only the first eigenmode will be present, giving</p><div id="P24"><p>
</p><div class="disp-formula" id="FD11"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-14-Frame"><nobr><span class="math" id="M14" style="width: 8.278em; display: inline-block;"><span style="display: inline-block; position: relative; width: 5.732em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(1.102em 1000.003em 2.665em -0.518em); top: -2.196em; left: 0.003em;"><span class="mrow" id="MathJax-Span-338"><span class="msub" id="MathJax-Span-339"><span style="display: inline-block; position: relative; width: 0.871em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.318em -0.518em); top: -2.138em; left: 0.003em;"><span class="mi" id="MathJax-Span-340" style="font-family: MathJax_Script;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.144em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.466em;"><span class="mi" id="MathJax-Span-341" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.061em;"></span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mspace" id="MathJax-Span-342" style="height: 0.003em; vertical-align: 0.003em; width: 0.177em; display: inline-block; overflow: hidden;"></span><span class="mo" id="MathJax-Span-343" style="font-family: MathJax_Main;">(</span><span class="msub" id="MathJax-Span-344"><span style="display: inline-block; position: relative; width: 1.565em; height: 0px;"><span style="position: absolute; clip: rect(1.276em 1000.003em 2.26em -0.518em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-345" style="font-family: MathJax_Math; font-style: italic;">t</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.292em;"><span class="mi" id="MathJax-Span-346" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">small</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-347" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-348" style="font-family: MathJax_Main; padding-left: 0.292em;">≈</span><span class="msup" id="MathJax-Span-349" style="padding-left: 0.292em;"><span style="display: inline-block; position: relative; width: 1.623em; height: 0px;"><span style="position: absolute; clip: rect(1.45em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-350" style="font-family: MathJax_Math; font-style: italic;">e</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -2.312em; left: 0.35em;"><span class="mrow" id="MathJax-Span-351"><span class="mo" id="MathJax-Span-352" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mi" id="MathJax-Span-353" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="msub" id="MathJax-Span-354"><span style="display: inline-block; position: relative; width: 0.524em; height: 0px;"><span style="position: absolute; clip: rect(1.276em 1000.003em 2.086em -0.46em); top: -1.907em; left: 0.003em;"><span class="mi" id="MathJax-Span-355" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">λ</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.292em;"><span class="mn" id="MathJax-Span-356" style="font-size: 50%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 1.855em;"></span></span></span></span><span class="mi" id="MathJax-Span-357" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">t</span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="msub" id="MathJax-Span-358"><span style="display: inline-block; position: relative; width: 0.813em; height: 0px;"><span style="position: absolute; clip: rect(1.565em 1000.003em 2.376em -0.46em); top: -2.196em; left: 0.003em;"><span class="mi" id="MathJax-Span-359" style="font-family: MathJax_Main; font-weight: bold;">u</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.466em;"><span class="mn" id="MathJax-Span-360" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-361"><span style="display: inline-block; position: relative; width: 0.813em; height: 0px;"><span style="position: absolute; clip: rect(1.565em 1000.003em 2.376em -0.46em); top: -2.196em; left: 0.003em;"><span class="mi" id="MathJax-Span-362" style="font-family: MathJax_Main; font-weight: bold;">u</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; clip: rect(1.276em 1000.003em 2.086em -0.518em); top: -2.254em; left: 0.466em;"><span class="mi" id="MathJax-Span-363" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">t</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span><span style="position: absolute; clip: rect(1.276em 1000.003em 2.086em -0.46em); top: -1.617em; left: 0.466em;"><span class="mn" id="MathJax-Span-364" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-365" style="font-family: MathJax_Main;">,</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.921em; vertical-align: -0.496em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-14"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M14" display="block" overflow="scroll"><msub><mi mathvariant="script">C</mi><mi>f</mi></msub><mspace width="0.16667em"></mspace><mo stretchy="false">(</mo><msub><mi>t</mi><mi mathvariant="italic">small</mi></msub><mo stretchy="false">)</mo><mo>≈</mo><msup><mi>e</mi><mrow><mo>-</mo><mi>β</mi><msub><mi>λ</mi><mn>1</mn></msub><mi>t</mi></mrow></msup><msub><mi mathvariant="bold">u</mi><mn>1</mn></msub><msubsup><mi mathvariant="bold">u</mi><mn>1</mn><mi>t</mi></msubsup><mo>,</mo></math></script></div></div><p> where <em>λ</em><sub>1</sub> and <em>u</em><sub>1</sub> are known from 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig4.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig4.jpg"> = <em>U</em>∧<em>U<sup>t</sup></em>.</p></div></div><div id="S8" class="sec sec-last"><h3>2.6. Model inversion: inferring structural from functional connectivity</h3><p id="P25" class="p p-first">An intriguing possibility raised by our proposal is that due to linearity it may be inverted, i.e. to partially (due to the presence of noisy eigenvectors) obtain structural connectivity <strong>C</strong> from functional connectivity matrix 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig5.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig5.jpg">, as follows</p><div id="P26"><p>
</p><div class="disp-formula" id="FD12"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-15-Frame"><nobr><span class="math" id="M15" style="width: 6.195em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.285em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(0.466em 1000.003em 3.244em -0.518em); top: -2.196em; left: 0.003em;"><span class="mrow" id="MathJax-Span-367"><span class="mover" id="MathJax-Span-368"><span style="display: inline-block; position: relative; width: 1.102em; height: 0px;"><span style="position: absolute; clip: rect(1.276em 1000.003em 2.318em -0.518em); top: -2.138em; left: 0.003em;"><span class="mi" id="MathJax-Span-369" style="font-family: MathJax_Script;">L<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.144em;"></span></span><span style="position: absolute; clip: rect(1.276em 1000.003em 1.797em -0.518em); top: -2.486em; left: 0.524em;"><span class="mo" id="MathJax-Span-370"><span style="font-family: MathJax_Size1;">ˆ</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span></span></span><span class="mo" id="MathJax-Span-371" style="font-family: MathJax_Main; padding-left: 0.292em;">=</span><span class="mo" id="MathJax-Span-372" style="font-family: MathJax_Main; padding-left: 0.292em;">−</span><span class="mfrac" id="MathJax-Span-373" style="padding-left: 0.292em; padding-right: 0.119em;"><span style="display: inline-block; position: relative; width: 2.202em; height: 0px;"><span style="position: absolute; clip: rect(1.276em 1000.003em 2.665em -0.518em); top: -3.006em; left: 50%; margin-left: -1.039em;"><span class="mrow" id="MathJax-Span-374"><span class="mo" id="MathJax-Span-375" style="font-family: MathJax_Main;">log</span><span class="mo" id="MathJax-Span-376" style="font-family: MathJax_Main;">(</span><span class="msub" id="MathJax-Span-377"><span style="display: inline-block; position: relative; width: 0.871em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.318em -0.518em); top: -2.138em; left: 0.003em;"><span class="mi" id="MathJax-Span-378" style="font-family: MathJax_Script;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.144em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.466em;"><span class="mi" id="MathJax-Span-379" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.061em;"></span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-380" style="font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; clip: rect(1.218em 1000.003em 2.433em -0.518em); top: -1.386em; left: 50%; margin-left: -0.576em;"><span class="mrow" id="MathJax-Span-381"><span class="mi" id="MathJax-Span-382" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="msub" id="MathJax-Span-383"><span style="display: inline-block; position: relative; width: 1.16em; height: 0px;"><span style="position: absolute; clip: rect(1.276em 1000.003em 2.26em -0.518em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-384" style="font-family: MathJax_Math; font-style: italic;">t</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.292em;"><span class="mi" id="MathJax-Span-385" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">crit</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; clip: rect(0.871em 1000.003em 1.276em -0.518em); top: -1.328em; left: 0.003em;"><span style="border-left-width: 2.202em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.25px; vertical-align: 0.003em;"></span><span style="display: inline-block; width: 0px; height: 1.102em;"></span></span></span></span><span class="mo" id="MathJax-Span-386" style="font-family: MathJax_Main; padding-left: 0.177em;">,</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 3.671em; vertical-align: -1.329em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-15"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M15" display="block" overflow="scroll"><mover accent="true"><mi mathvariant="script">L</mi><mo>^</mo></mover><mo>=</mo><mo>-</mo><mfrac><mrow><mo>log</mo><mo stretchy="false">(</mo><msub><mi mathvariant="script">C</mi><mi>f</mi></msub><mo stretchy="false">)</mo></mrow><mrow><mi>β</mi><msub><mi>t</mi><mi mathvariant="italic">crit</mi></msub></mrow></mfrac><mo>,</mo></math></script></div><div class="l">(7)</div></div><p> where it is understood that log(
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig5.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig5.jpg">) is the principal logarithm of the matrix 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig5.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig5.jpg">, and where structural connectivity matrix <strong>C</strong> is obtained from</p></div><div id="P27"><p>
</p><div class="disp-formula" id="FD13"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-16-Frame"><nobr><span class="math" id="M16" style="width: 6.427em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.459em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(0.987em 1000.003em 2.549em -0.518em); top: -2.196em; left: 0.003em;"><span class="mrow" id="MathJax-Span-388"><span class="mover" id="MathJax-Span-389"><span style="display: inline-block; position: relative; width: 1.102em; height: 0px;"><span style="position: absolute; clip: rect(1.276em 1000.003em 2.318em -0.518em); top: -2.138em; left: 0.003em;"><span class="mi" id="MathJax-Span-390" style="font-family: MathJax_Script;">L<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.144em;"></span></span><span style="position: absolute; clip: rect(1.276em 1000.003em 1.797em -0.518em); top: -2.486em; left: 0.524em;"><span class="mo" id="MathJax-Span-391"><span style="font-family: MathJax_Size1;">ˆ</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span></span></span><span class="mo" id="MathJax-Span-392" style="font-family: MathJax_Main; padding-left: 0.292em;">=</span><span class="mi" id="MathJax-Span-393" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.292em;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.061em;"></span></span><span class="mo" id="MathJax-Span-394" style="font-family: MathJax_Main; padding-left: 0.234em;">−</span><span class="msup" id="MathJax-Span-395" style="padding-left: 0.234em;"><span style="display: inline-block; position: relative; width: 1.276em; height: 0px;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.376em -0.46em); top: -2.196em; left: 0.003em;"><span class="mi" id="MathJax-Span-396" style="font-family: MathJax_Main;">Δ</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; top: -2.37em; left: 0.582em;"><span class="mrow" id="MathJax-Span-397"><span class="mo" id="MathJax-Span-398" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mn" id="MathJax-Span-399" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mi" id="MathJax-Span-400" style="font-family: MathJax_Main; font-weight: bold;">C</span><span class="mo" id="MathJax-Span-401" style="font-family: MathJax_Main;">,</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 2.004em; vertical-align: -0.329em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-16"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M16" display="block" overflow="scroll"><mover accent="true"><mi mathvariant="script">L</mi><mo>^</mo></mover><mo>=</mo><mi>I</mi><mo>-</mo><msup><mi mathvariant="normal">Δ</mi><mrow><mo>-</mo><mn>1</mn></mrow></msup><mi mathvariant="bold">C</mi><mo>,</mo></math></script></div><div class="l">(8)</div></div><p> or</p></div><div id="P28"><p>
</p><div class="disp-formula" id="FD14"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-17-Frame"><nobr><span class="math" id="M17" style="width: 8.799em; display: inline-block;"><span style="display: inline-block; position: relative; width: 6.079em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(0.987em 1000.003em 2.549em -0.518em); top: -2.196em; left: 0.003em;"><span class="mrow" id="MathJax-Span-403"><span class="mover" id="MathJax-Span-404"><span style="display: inline-block; position: relative; width: 1.102em; height: 0px;"><span style="position: absolute; clip: rect(1.276em 1000.003em 2.318em -0.518em); top: -2.138em; left: 0.003em;"><span class="mi" id="MathJax-Span-405" style="font-family: MathJax_Script;">L<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.144em;"></span></span><span style="position: absolute; clip: rect(1.276em 1000.003em 1.797em -0.518em); top: -2.486em; left: 0.524em;"><span class="mo" id="MathJax-Span-406"><span style="font-family: MathJax_Size1;">ˆ</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span></span></span><span class="mo" id="MathJax-Span-407" style="font-family: MathJax_Main; padding-left: 0.292em;">=</span><span class="mi" id="MathJax-Span-408" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.292em;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.061em;"></span></span><span class="mo" id="MathJax-Span-409" style="font-family: MathJax_Main; padding-left: 0.234em;">−</span><span class="msup" id="MathJax-Span-410" style="padding-left: 0.234em;"><span style="display: inline-block; position: relative; width: 1.797em; height: 0px;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.376em -0.46em); top: -2.196em; left: 0.003em;"><span class="mi" id="MathJax-Span-411" style="font-family: MathJax_Main;">Δ</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; top: -2.37em; left: 0.582em;"><span class="mrow" id="MathJax-Span-412"><span class="mo" id="MathJax-Span-413" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mn" id="MathJax-Span-414" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-415" style="font-size: 70.7%; font-family: MathJax_Main;">/</span><span class="mn" id="MathJax-Span-416" style="font-size: 70.7%; font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mi" id="MathJax-Span-417" style="font-family: MathJax_Main; font-weight: bold;">C</span><span class="msup" id="MathJax-Span-418"><span style="display: inline-block; position: relative; width: 1.797em; height: 0px;"><span style="position: absolute; clip: rect(1.334em 1000.003em 2.376em -0.46em); top: -2.196em; left: 0.003em;"><span class="mi" id="MathJax-Span-419" style="font-family: MathJax_Main;">Δ</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; top: -2.37em; left: 0.582em;"><span class="mrow" id="MathJax-Span-420"><span class="mo" id="MathJax-Span-421" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mn" id="MathJax-Span-422" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-423" style="font-size: 70.7%; font-family: MathJax_Main;">/</span><span class="mn" id="MathJax-Span-424" style="font-size: 70.7%; font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-425" style="font-family: MathJax_Main;">,</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 2.004em; vertical-align: -0.329em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-17"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M17" display="block" overflow="scroll"><mover accent="true"><mi mathvariant="script">L</mi><mo>^</mo></mover><mo>=</mo><mi>I</mi><mo>-</mo><msup><mi mathvariant="normal">Δ</mi><mrow><mo>-</mo><mn>1</mn><mo>/</mo><mn>2</mn></mrow></msup><mi mathvariant="bold">C</mi><msup><mi mathvariant="normal">Δ</mi><mrow><mo>-</mo><mn>1</mn><mo>/</mo><mn>2</mn></mrow></msup><mo>,</mo></math></script></div><div class="l">(9)</div></div><p> whichever is appropriate. The inversion is specified only up to the normalized Laplacian, and going from there to connectivity matrix would require knowledge of node degree. However, this can be obtained a priori, either from the given functional matrix or prior structural matrices. Although a detailed investigation of this application will warrant a separate study, here we provide a sketch of preliminary results and show two estimated structural connectivity matrices obtained from the functional connectivity. First the functional connectivity matrix 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig5.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig5.jpg"> is regularized by factoring it into its SVD components, or</p></div><div class="disp-formula" id="FD15"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-18-Frame"><nobr><span class="math" id="M18" style="width: 6.079em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.227em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(0.292em 1000.003em 3.591em -0.518em); top: -2.196em; left: 0.003em;"><span class="mrow" id="MathJax-Span-427"><span class="msub" id="MathJax-Span-428"><span style="display: inline-block; position: relative; width: 0.871em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.318em -0.518em); top: -2.138em; left: 0.003em;"><span class="mi" id="MathJax-Span-429" style="font-family: MathJax_Script;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.144em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.466em;"><span class="mi" id="MathJax-Span-430" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.061em;"></span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-431" style="font-family: MathJax_Main; padding-left: 0.292em;">=</span><span class="munderover" id="MathJax-Span-432" style="padding-left: 0.292em;"><span style="display: inline-block; position: relative; width: 1.045em; height: 0px;"><span style="position: absolute; clip: rect(1.739em 1000.003em 3.475em -0.46em); top: -2.833em; left: 0.003em;"><span class="mo" id="MathJax-Span-433" style="font-family: MathJax_Size2; vertical-align: 0.003em;">∑</span><span style="display: inline-block; width: 0px; height: 2.839em;"></span></span><span style="position: absolute; clip: rect(1.276em 1000.003em 2.202em -0.46em); top: -0.807em; left: 0.061em;"><span class="mrow" id="MathJax-Span-434"><span class="mi" id="MathJax-Span-435" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span class="mo" id="MathJax-Span-436" style="font-size: 70.7%; font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-437" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span><span style="position: absolute; clip: rect(1.16em 1000.003em 2.086em -0.518em); top: -3.064em; left: 0.292em;"><span class="mi" id="MathJax-Span-438" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">K<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mrow" id="MathJax-Span-439" style="padding-left: 0.177em;"><span class="msub" id="MathJax-Span-440"><span style="display: inline-block; position: relative; width: 0.813em; height: 0px;"><span style="position: absolute; clip: rect(1.565em 1000.003em 2.376em -0.46em); top: -2.196em; left: 0.003em;"><span class="mi" id="MathJax-Span-441" style="font-family: MathJax_Main; font-weight: bold;">u</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.466em;"><span class="mi" id="MathJax-Span-442" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-443"><span style="display: inline-block; position: relative; width: 0.813em; height: 0px;"><span style="position: absolute; clip: rect(1.565em 1000.003em 2.376em -0.518em); top: -2.196em; left: 0.003em;"><span class="mi" id="MathJax-Span-444" style="font-family: MathJax_Main; font-weight: bold;">v</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; clip: rect(1.276em 1000.003em 2.086em -0.518em); top: -2.254em; left: 0.466em;"><span class="mi" id="MathJax-Span-445" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">t</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span><span style="position: absolute; clip: rect(1.276em 1000.003em 2.086em -0.46em); top: -1.56em; left: 0.466em;"><span class="mi" id="MathJax-Span-446" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="msub" id="MathJax-Span-447"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-448" style="font-family: MathJax_Math; font-style: italic;">λ</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mi" id="MathJax-Span-449" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-450" style="font-family: MathJax_Main;">.</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 4.421em; vertical-align: -1.829em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-18"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M18" display="block" overflow="scroll"><msub><mi mathvariant="script">C</mi><mi>f</mi></msub><mo>=</mo><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mi>K</mi></munderover><mrow><msub><mi mathvariant="bold">u</mi><mi>k</mi></msub><msubsup><mi mathvariant="bold">v</mi><mi>k</mi><mi>t</mi></msubsup><msub><mi>λ</mi><mi>k</mi></msub></mrow><mo>.</mo></math></script></div></div><p id="P29">We next apply a threshold to the singular values <em>λ<sub>k</sub></em>, keeping the values above the threshold, obtaining</p><div id="P30"><p>
</p><div class="disp-formula" id="FD16"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" role="textbox" aria-readonly="true" style="text-align: center;"><span class="MathJax" id="MathJax-Element-19-Frame"><nobr><span class="math" id="M19" style="width: 6.427em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.459em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(1.102em 1000.003em 3.649em -0.518em); top: -2.196em; left: 0.003em;"><span class="mrow" id="MathJax-Span-452"><span class="msubsup" id="MathJax-Span-453"><span style="display: inline-block; position: relative; width: 1.045em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.318em -0.518em); top: -2.138em; left: 0.003em;"><span class="mi" id="MathJax-Span-454" style="font-family: MathJax_Script;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.144em;"></span></span><span style="position: absolute; clip: rect(1.45em 1000.003em 2.086em -0.518em); top: -2.37em; left: 0.755em;"><span class="mi" id="MathJax-Span-455" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">r</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -1.675em; left: 0.466em;"><span class="mi" id="MathJax-Span-456" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.061em;"></span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="mo" id="MathJax-Span-457" style="font-family: MathJax_Main; padding-left: 0.292em;">=</span><span class="munder" id="MathJax-Span-458" style="padding-left: 0.292em;"><span style="display: inline-block; position: relative; width: 1.218em; height: 0px;"><span style="position: absolute; clip: rect(1.739em 1000.003em 3.475em -0.46em); top: -2.833em; left: 0.119em;"><span class="mo" id="MathJax-Span-459" style="font-family: MathJax_Size2; vertical-align: 0.003em;">∑</span><span style="display: inline-block; width: 0px; height: 2.839em;"></span></span><span style="position: absolute; clip: rect(1.276em 1000.003em 2.26em -0.46em); top: -0.865em; left: 0.003em;"><span class="mrow" id="MathJax-Span-460"><span class="mi" id="MathJax-Span-461" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span class="mo" id="MathJax-Span-462" style="font-size: 70.7%; font-family: MathJax_Main;">∈</span><span class="mi" id="MathJax-Span-463" style="font-size: 70.7%; font-family: MathJax_Script;">K<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.234em;"></span></span></span><span style="display: inline-block; width: 0px; height: 1.97em;"></span></span></span></span><span class="mrow" id="MathJax-Span-464" style="padding-left: 0.177em;"><span class="msub" id="MathJax-Span-465"><span style="display: inline-block; position: relative; width: 0.813em; height: 0px;"><span style="position: absolute; clip: rect(1.565em 1000.003em 2.376em -0.46em); top: -2.196em; left: 0.003em;"><span class="mi" id="MathJax-Span-466" style="font-family: MathJax_Main; font-weight: bold;">u</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.466em;"><span class="mi" id="MathJax-Span-467" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-468"><span style="display: inline-block; position: relative; width: 0.813em; height: 0px;"><span style="position: absolute; clip: rect(1.565em 1000.003em 2.376em -0.518em); top: -2.196em; left: 0.003em;"><span class="mi" id="MathJax-Span-469" style="font-family: MathJax_Main; font-weight: bold;">v</span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span><span style="position: absolute; clip: rect(1.276em 1000.003em 2.086em -0.518em); top: -2.254em; left: 0.466em;"><span class="mi" id="MathJax-Span-470" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">t</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span><span style="position: absolute; clip: rect(1.276em 1000.003em 2.086em -0.46em); top: -1.56em; left: 0.466em;"><span class="mi" id="MathJax-Span-471" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span><span class="msub" id="MathJax-Span-472"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -2.08em; left: 0.003em;"><span class="mi" id="MathJax-Span-473" style="font-family: MathJax_Math; font-style: italic;">λ</span><span style="display: inline-block; width: 0px; height: 2.086em;"></span></span><span style="position: absolute; top: -1.733em; left: 0.408em;"><span class="mi" id="MathJax-Span-474" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-475" style="font-family: MathJax_Main;">,</span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 3.337em; vertical-align: -1.912em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-19"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M19" display="block" overflow="scroll"><msubsup><mi mathvariant="script">C</mi><mi>f</mi><mi>r</mi></msubsup><mo>=</mo><munder><mo>∑</mo><mrow><mi>k</mi><mo>∈</mo><mi mathvariant="script">K</mi></mrow></munder><mrow><msub><mi mathvariant="bold">u</mi><mi>k</mi></msub><msubsup><mi mathvariant="bold">v</mi><mi>k</mi><mi>t</mi></msubsup><msub><mi>λ</mi><mi>k</mi></msub></mrow><mo>,</mo></math></script></div></div><p> where 
<span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-20-Frame" role="textbox" aria-readonly="true"><nobr><span class="math" id="M20" style="width: 1.102em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(0.871em 1000.003em 2.376em -0.518em); top: -1.791em; left: 0.003em;"><span class="mrow" id="MathJax-Span-477"><span class="msubsup" id="MathJax-Span-478"><span style="display: inline-block; position: relative; width: 1.045em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.318em -0.518em); top: -2.138em; left: 0.003em;"><span class="mi" id="MathJax-Span-479" style="font-family: MathJax_Script;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.144em;"></span></span><span style="position: absolute; clip: rect(1.45em 1000.003em 2.086em -0.518em); top: -2.37em; left: 0.755em;"><span class="mi" id="MathJax-Span-480" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">r</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -1.675em; left: 0.466em;"><span class="mi" id="MathJax-Span-481" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.061em;"></span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 1.797em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.838em; vertical-align: -0.662em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-20"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M20" overflow="scroll"><msubsup><mi mathvariant="script">C</mi><mi>f</mi><mi>r</mi></msubsup></math></script> is the regularized functional connectivity matrix and 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig6.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig6.jpg"> is the set of all singular values <em>λ<sub>k</sub></em> exceeding a given threshold. Matrix 
<span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-21-Frame" role="textbox" aria-readonly="true"><nobr><span class="math" id="M21" style="width: 1.102em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.755em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(0.871em 1000.003em 2.376em -0.518em); top: -1.791em; left: 0.003em;"><span class="mrow" id="MathJax-Span-483"><span class="msubsup" id="MathJax-Span-484"><span style="display: inline-block; position: relative; width: 1.045em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.318em -0.518em); top: -2.138em; left: 0.003em;"><span class="mi" id="MathJax-Span-485" style="font-family: MathJax_Script;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.144em;"></span></span><span style="position: absolute; clip: rect(1.45em 1000.003em 2.086em -0.518em); top: -2.37em; left: 0.755em;"><span class="mi" id="MathJax-Span-486" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">r</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -1.675em; left: 0.466em;"><span class="mi" id="MathJax-Span-487" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.061em;"></span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 1.797em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.838em; vertical-align: -0.662em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-21"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M21" overflow="scroll"><msubsup><mi mathvariant="script">C</mi><mi>f</mi><mi>r</mi></msubsup></math></script> is then used in <a href="http://europepmc.org/articles/pmc3951650#FD12" rid="FD12" class=" disp-formula">equations (7</a>–<a href="http://europepmc.org/articles/pmc3951650#FD14" rid="FD14" class=" disp-formula">9)</a> to obtain an estimate of the structural connectivity. Threshold is defined as <em>αλ</em><sub>max</sub>, where <em>λ</em><sub>max</sub> is the largest singular value in 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig5.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig5.jpg">, and <em>α</em> (0 <em>&lt; α</em> ≤ 1) is chosen by the user.</p></div></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://europepmc.org/articles/pmc3951650#ui-ncbiinpagenav-2" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#ui-ncbiinpagenav3" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">3. Methods</h2><div id="S10" class="sec sec-first"><h3>3.1. Subjects and MR imaging</h3><p id="P31" class="p p-first">T1-weighted structural MR and High Angular Resolution Diffusion Imaging (HARDI) data were collected on 8 healthy adults on a 3 Tesla GE Signa EXCITE scanner (GE Healthcare, Waukesha, WI, USA). HARDI data were acquired using 55 isotropically distributed diffusion-encoding directions at <em>b</em> = 1000 s/mm<sup>2</sup> and one at <em>b</em> = 0 s/mm<sup>2</sup>, acquired at 72 1.8 – mm thick interleaved slices with no gap between slices and 128 × 128 matrix size that was zero-filled during reconstruction to 256×256 with a field of view (FOV) of 230 mm. The structural scan was an axial 3D inversion recovery fast spoiled gradient recalled echo (FSPGR) T1 weighted protocol (<em>TE</em> = 1.5 ms, <em>TR</em> = 6.3 ms, <em>TI</em> = 400ms, flip angle of 15) with 230 mm FOV and 156 1.0 – mm contiguous partitions at a 256 × 256 matrix.</p><p id="P32" class="p p-last">Resting state fMRI was performed, together with anatomical MRI, with an eight-channel head coil using echo-planar imaging based functional MRI pulse sequences (repetition time <em>TR</em> = 2 s, echo time <em>TE</em> = 30 ms, flip angle 70, matrix size 64 × 64 × 28, axial field of view 24 cm, 5 mm slice thickness; rs-fMRI was acquired with 180 samples. 10 samples at the beginning were discarded. Before rs-fMRI, the subject was instructed to think of nothing in particular. The subjects’ ages range from 23 to 60, with three females and five males. Subjects were scanned under normal subject protocol approved by the institutional review board (IRB).</p></div><div id="S11" class="sec sec-last"><h3>3.2. Extraction of structural and functional brain networks</h3><p id="P33" class="p p-first">Diffusion tractography processing closely followed established pipelines [<a href="http://europepmc.org/articles/pmc3951650#R40" rid="R40" class=" bibr popnode">40</a>, <a href="http://europepmc.org/articles/pmc3951650#R53" rid="R53" class=" bibr popnode">53</a>]. Briefly, structural and diffusion MR volumes were co-registered using SPM tools in MATLAB [<a href="http://europepmc.org/articles/pmc3951650#R4" rid="R4" class=" bibr popnode">4</a>, <a href="http://europepmc.org/articles/pmc3951650#R28" rid="R28" class=" bibr popnode">28</a>], then parcellated into 90 cerebral cortical structures as per [<a href="http://europepmc.org/articles/pmc3951650#R60" rid="R60" class=" bibr popnode">60</a>]. Parcellated regions were used to seed probabilistic tractography in co-registered diffusion MRI volumes. Connectivity weight between any two regions was given by a weighted sum of tracts going between them, as per [<a href="http://europepmc.org/articles/pmc3951650#R38" rid="R38" class=" bibr popnode">38</a>]. Simple statistical thresholding was performed to remove spurious weak connections, defined as those below the <em>p</em> = 0.001 level of significance.</p><p id="P34">The SPM-based package DPARSF [<a href="http://europepmc.org/articles/pmc3951650#R58" rid="R58" class=" bibr popnode">58</a>] was used to perform the standard resting-state preprocessing steps. For all subjects the first 10 out of 180 time points each were discarded. Spatial resolution was set at 2×2×2mm<sup>3</sup>. The images were realigned, then normalized using DARTEL [<a href="http://europepmc.org/articles/pmc3951650#R7" rid="R7" class=" bibr popnode">7</a>]. The images were next smoothened with Gaussian smoothing kernel with full width half max <em>FWHM</em> = [4 4 4]mm. This was followed by detrending the images over the range 0.01–0.08Hz. Finally, the noise covariates were regressed out. The functional connectivity networks and resulting matrices were obtained from resting state fMRI scans using the CONN functional connectivity toolbox [<a href="http://europepmc.org/articles/pmc3951650#R66" rid="R66" class=" bibr popnode">66</a>]. Weak functional connectivity, defined as smaller than 
<span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-22-Frame" role="textbox" aria-readonly="true"><nobr><span class="math" id="M22" style="width: 3.591em; display: inline-block;"><span style="display: inline-block; position: relative; width: 2.491em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(1.276em 1000.003em 2.781em -0.46em); top: -2.196em; left: 0.003em;"><span class="mrow" id="MathJax-Span-489"><span class="mn" id="MathJax-Span-490" style="font-family: MathJax_Main;">0.05</span><span class="msubsup" id="MathJax-Span-491"><span style="display: inline-block; position: relative; width: 1.797em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.318em -0.518em); top: -2.138em; left: 0.003em;"><span class="mi" id="MathJax-Span-492" style="font-family: MathJax_Script;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.144em;"></span></span><span style="position: absolute; clip: rect(1.45em 1000.003em 2.086em -0.518em); top: -2.37em; left: 0.755em;"><span class="mi" id="MathJax-Span-493" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">max</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -1.675em; left: 0.466em;"><span class="mi" id="MathJax-Span-494" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.061em;"></span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.202em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.838em; vertical-align: -0.662em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-22"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M22" overflow="scroll"><mn>0.05</mn><msubsup><mi mathvariant="script">C</mi><mi>f</mi><mi mathvariant="italic">max</mi></msubsup></math></script>, is set to zero, with 
<span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-23-Frame" role="textbox" aria-readonly="true"><nobr><span class="math" id="M23" style="width: 1.855em; display: inline-block;"><span style="display: inline-block; position: relative; width: 1.276em; height: 0px; font-size: 144%;"><span style="position: absolute; clip: rect(0.871em 1000.003em 2.376em -0.518em); top: -1.791em; left: 0.003em;"><span class="mrow" id="MathJax-Span-496"><span class="msubsup" id="MathJax-Span-497"><span style="display: inline-block; position: relative; width: 1.797em; height: 0px;"><span style="position: absolute; clip: rect(1.218em 1000.003em 2.318em -0.518em); top: -2.138em; left: 0.003em;"><span class="mi" id="MathJax-Span-498" style="font-family: MathJax_Script;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.177em;"></span></span><span style="display: inline-block; width: 0px; height: 2.144em;"></span></span><span style="position: absolute; clip: rect(1.45em 1000.003em 2.086em -0.518em); top: -2.37em; left: 0.755em;"><span class="mi" id="MathJax-Span-499" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">max</span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span><span style="position: absolute; clip: rect(1.218em 1000.003em 2.26em -0.46em); top: -1.675em; left: 0.466em;"><span class="mi" id="MathJax-Span-500" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.061em;"></span></span><span style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 1.797em;"></span></span></span><span style="border-left-width: 0.004em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.838em; vertical-align: -0.662em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-23"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M23" overflow="scroll"><msubsup><mi mathvariant="script">C</mi><mi>f</mi><mi mathvariant="italic">max</mi></msubsup></math></script> the largest absolute value of interregional functional connectivity matrix 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig5.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig5.jpg">.</p><p id="P66" class="p p-last">Both the linear and non-linear generative models were evaluated by comparing the similarity between the functional connectivity predicted by the model and the empirical functional connectivity measured from resting state fMRI data. The measure of similarity we chose is the simple Pearson correlation coefficient, evaluated only over thresholded node-pairs, in a manner similar to [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>]. This avoids the noise introduced by the spurious connectivity between weakly connected or non-connected regions.</p></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://europepmc.org/articles/pmc3951650#ui-ncbiinpagenav-2" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#ui-ncbiinpagenav3" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">4. Results</h2><div id="S13" class="sec sec-first"><h3>4.1. Performance of model depends on global connectivity coupling parameter</h3><p id="P35" class="p p-first-last"><a href="http://europepmc.org/articles/PMC3951650/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a> (left) depicts the Pearson correlation between true FC and the nonlinear model prediction for all subjects over the values of <em>c</em> in the nonlinear model. The resulting curve takes on a bell shape. A strikingly similar behavior is seen in the linear network diffusion model (<a href="http://europepmc.org/articles/PMC3951650/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, right), whose agreement with true FC also varies with the global “connectivity coupling” parameter <em>βt</em>, follows a bell shape, similar to the non-linear model. In Discussion we explain how this behavior is observed in almost all computational models of brain dynamics, including coupled oscillator models [<a href="http://europepmc.org/articles/pmc3951650#R22" rid="R22" class=" bibr popnode">22</a>].</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a class="icnblk_img figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><img src="./Network_Diffusion_Brain_files/nihms552872f1.gif" class="small-thumb" alt="Figure 1" title="Figure 1" src-large="http://europepmc.org/articles/PMC3951650/bin/nihms552872f1.jpg"></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a4--><div><span>Eight subjects’ Pearson correlation over two models. The left figure depicts the correlation for all subjects over values of <em>c</em> in the non-linear model, and the right figure shows the correlation behaviour of the proposed graph model as a function</span> <strong>...</strong></div></div></div></div><div id="S14" class="sec"><h3>4.2. Correlation of rsFC with FC predicted by each model</h3><p id="P36" class="p p-first">We compare the performance of the linear Galán and non-linear Honey <em>et al.</em> [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>] models with the proposed functional connectivity estimates by evaluating the Pearson coefficient of the correlation between each model prediction and the true functional connectivity matrix rsFC. <a href="http://europepmc.org/articles/PMC3951650/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig 2</span></a> reflects the scatter plot of each of the mean functional connectivity estimates over all eight subjects relative to the empirical matrix. The figure suggests that in the case of the linear model estimate of functional connectivity the estimate is poor, while the alternate methods nonlinear and Laplacian models offer a tighter scattering of the estimates relative to the true functional connectivity. For the nonlinear model, the estimated FC is evaluated at the value of the parameter <em>c</em> yielding the highest correlation with true FC. <a href="http://europepmc.org/articles/PMC3951650/table/T1/" target="true" class="fig-table-link table figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> summarizes the performance of each approach over all eight subjects. Clearly, FC predicted by the network diffusion model is more closely correlated to the true FC matrix than possible with the other models. In order to statistically validate this, we performed Fisher’s <em>R</em> to <em>z</em> transformation, and obtained its <em>p</em>-value. The Fishers R to z transformation is a tool that measures whether two given values of Pearsons R are statistically different or not (quite apart from whether each R is by itself significant). Here we used it to report the significance of the difference in R between two correlations: empirical FC versus FC predicted by the nonlinear model, and empirical FC versus FC predicted by the network-diffusion model. The reported Fisher statistic is therefore a measure of the incremental value of the network diffusion model over the non-linear model. This allows us to test the statistical significance of the correlation between estimated FC and the empirical FC. These are tabulated in <a href="http://europepmc.org/articles/PMC3951650/table/T1/" target="true" class="fig-table-link table figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. All improvements are highly significant statistically compared with the non-linear model. From the table it can be seen that in general the non-linear model provides higher correlations with measured data than the Galán linear model, but lower than the network diffusion model. <a href="http://europepmc.org/articles/PMC3951650/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a> depicts group means of structural and functional networks, both empirical and model predictors. Galán linear model appears to miss the frontal lobe connectivity.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a class="icnblk_img figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><img src="./Network_Diffusion_Brain_files/nihms552872f2.gif" class="small-thumb" alt="Figure 2" title="Figure 2" src-large="http://europepmc.org/articles/PMC3951650/bin/nihms552872f2.jpg"></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a4--><div><span>Scatter plots. Clockwise from top left: structure connectivity, linear functional model, network diffusion functional model, and non-linear functional model. Blue: frontal, cyan: parietal, green: occipital, orange: temporal, red: subcortical.</span></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a class="icnblk_img figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><img src="./Network_Diffusion_Brain_files/nihms552872f3.gif" class="small-thumb" alt="Figure 3" title="Figure 3" src-large="http://europepmc.org/articles/PMC3951650/bin/nihms552872f3.jpg"></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a4--><div><span>Eight subjects’ mean networks. Clockwise from top left: structural, empirical functional, empirical functional with intraconnected hemispheres, proposed network diffusion functional connectivities, non-linear (Honey <em>et al.</em>) model [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>], and the</span> <strong>...</strong></div></div></div><!--table ft1--><!--table-wrap mode=article t1--><div class="table-wrap iconblock ten_col whole_rhythm clearfix" id="T1"><a href="http://europepmc.org/articles/PMC3951650/table/T1/" class="table img_link icnblk_img" target="table" rid-ob="ob-T1"><img src="./Network_Diffusion_Brain_files/table-icon.gif" class="small-thumb" alt="Table 1" title="Table 1"></a><div class="icnblk_cntnt"><div><a href="http://europepmc.org/articles/PMC3951650/table/T1/" target="table" rid-figpopup="T1" rid-ob="ob-T1">Table 1</a></div><!--caption a4--><div><span>Models comparison</span></div></div></div><p id="P37">Inter-hemispheric connections are clearly underestimated by all models considered, which we attribute to the well known distance bias in tractography [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>]. When inter-hemispheres are omitted (<a href="http://europepmc.org/articles/PMC3951650/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>, ‘Intrahemi FC’), the estimated networks begin to match empirical data quite well.</p><p id="P38">The network diffusion model has a global a priori unknown parameter <em>βt</em>. The above results correspond to maxima over a range of <em>βt</em> (<a href="http://europepmc.org/articles/PMC3951650/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, right). Here we show that we can replace the per-subject optimal parameter with a global “learned” parameter given by the median of <em>βt<sub>crit</sub></em> over eight subjects, since the maximum correlation occurs approximately at the same point for all cases (<a href="http://europepmc.org/articles/PMC3951650/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, right). <a href="http://europepmc.org/articles/PMC3951650/table/T2/" target="true" class="fig-table-link table figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> contains those results. The loss of performance is minimal, as suggested by the high (hence non-significant) <em>p-</em>values associated with Fisher’s <em>R</em> to <em>z</em> transformation listed in the same table. This emphasizes the value of the proposed model for predictive purposes, with parameter value learned from group analysis.</p><!--table ft1--><!--table-wrap mode=article t1--><div class="table-wrap iconblock ten_col whole_rhythm clearfix" id="T2"><a href="http://europepmc.org/articles/PMC3951650/table/T2/" class="table img_link icnblk_img" target="table" rid-ob="ob-T2"><img src="./Network_Diffusion_Brain_files/table-icon.gif" class="small-thumb" alt="Table 2" title="Table 2"></a><div class="icnblk_cntnt"><div><a href="http://europepmc.org/articles/PMC3951650/table/T2/" target="table" rid-figpopup="T2" rid-ob="ob-T2">Table 2</a></div><!--caption a4--><div><span>Correlations with global value <em>βt<sub>crit</sub></em> = 2</span></div></div></div><p id="P39">To verify that the estimated functional connectivities were not obtained by chance, the mean structural matrix as well as the means of the estimated functional matrices from the three methods were each randomly scrambled. The Pearson correlation of each scrambled matrix with the true functional connectivity shown in <a href="http://europepmc.org/articles/PMC3951650/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a> indicates that our correlation results are extremely unlikely to be due to chance.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a class="icnblk_img figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><img src="./Network_Diffusion_Brain_files/nihms552872f4.gif" class="small-thumb" alt="Figure 4" title="Figure 4" src-large="http://europepmc.org/articles/PMC3951650/bin/nihms552872f4.jpg"></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a4--><div><span>Histograms resulting from randomizing structural and estimated functional networks. The resulting Pearson correlations for the randomized matrices are negligible while those obtained from the estimated network matrices are significant.</span></div></div></div><p id="P40" class="p p-last">In order to further characterize the performance and robustness of each model, they were recomputed for progressively sparse (pruned) mean structural connectivity, and their correlation with measured functional network was determined at each sparsity level (<a href="http://europepmc.org/articles/PMC3951650/figure/F5/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5</span></a>). The graph diffusion network shows a consistent Pearson correlation up to a thresholding level of about 0.3 of the maximum edge weight while remaining superior to the linear and non-linear models. This result also demonstrates the superior stability of the proposed approach with respect to connectivity measures.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a class="icnblk_img figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><img src="./Network_Diffusion_Brain_files/nihms552872f5.gif" class="small-thumb" alt="Figure 5" title="Figure 5" src-large="http://europepmc.org/articles/PMC3951650/bin/nihms552872f5.jpg"></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a4--><div><span>Pearson correlation resulting from sparsified structural connectivity matrix. Level of thresholding denoted as a fraction of maximum edge weight. The percentage of surviving edges varies from 39% (no thresholding) to about 1%. The proposed model maintains</span> <strong>...</strong></div></div></div></div><div id="S15" class="sec"><h3>4.3. Summary network measures</h3><p id="P41" class="p p-first">Next we demonstrate that the proposed model recapitulates the main summary network statistics of functional brain networks. For this experiment the same level of sparsity (15%) was maintained in all predicted networks obtained from the pruned structural network whose sparsity was 10%. Using the Matlab toolbox Brain Connectivity Toolbox [<a href="http://europepmc.org/articles/pmc3951650#R54" rid="R54" class=" bibr popnode">54</a>], and following [<a href="http://europepmc.org/articles/pmc3951650#R54" rid="R54" class=" bibr popnode">54</a>], the diagonal and negative elements of the matrices are excluded. We note that the negative elements of the mean connectivity matrices discussed in this work have a small mean when compared with the positive elements. Additionally, the statistics are computed only for the largest 15% elements of each matrix, since this is where Pearson correlation achieves a maximum (<a href="http://europepmc.org/articles/PMC3951650/figure/F5/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5</span></a>, black vertical line). Very few negative elements survive, if any.</p><p id="P42" class="p p-last">The metrics used were mean path length, global efficiency, maximized modularity, and optimal community structure. For all networks the statistics were obtained at the same level of sparsity. For mean path length and efficiency, the distance between nodes was modeled as a Gaussian with respect to connectivity strength. The non-existing edges were assigned a distance of infinity. As shown in <a href="http://europepmc.org/articles/PMC3951650/figure/F6/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig 6</span></a>, the summary metrics from the proposed model closely resemble those from the measured functional network, to a somewhat higher extent than other models considered. The results provide additional means of validating the graph diffusion model; and while the mapping from a network to these summary metrics is not one to one, these results demonstrate that the model gives networks with the same kind of statistical behavior as expected from real functional networks.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F6" co-legend-rid="lgnd_F6"><a class="icnblk_img figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><img src="./Network_Diffusion_Brain_files/nihms552872f6.gif" class="small-thumb" alt="Figure 6" title="Figure 6" src-large="http://europepmc.org/articles/PMC3951650/bin/nihms552872f6.jpg"></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6</a></div><!--caption a4--><div><span>Various network measures of functional and structural networks evaluated at threshold level 15% (see <a href="http://europepmc.org/articles/PMC3951650/figure/F5/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig 5</span></a>). Clockwise from top left: Mean path length, global efficiency, optimal community structure, and maximum modularity. The matrices are identified</span> <strong>...</strong></div></div></div></div><div id="S16" class="sec"><h3>4.4. Seed-based connectivity analysis</h3><p id="P43" class="p p-first">We now investigate seed-based connectivity of some specific regions with interesting functional roles (<a href="http://europepmc.org/articles/PMC3951650/figure/F7/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Figs 7</span></a>–<a href="http://europepmc.org/articles/PMC3951650/figure/F9/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>9</span></a>). <a href="http://europepmc.org/articles/PMC3951650/figure/F7/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Fig 7</span></a> shows functional connectivity out of the right posterior cingulate and suggests a strong connectivity with the frontal lobe and the precuneus, regions with known strong resting state functional connectivity to the posterior cingulate cortex [<a href="http://europepmc.org/articles/pmc3951650#R62" rid="R62" class=" bibr popnode">62</a>, <a href="http://europepmc.org/articles/pmc3951650#R63" rid="R63" class=" bibr popnode">63</a>].</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F7" co-legend-rid="lgnd_F7"><a class="icnblk_img figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"><img src="./Network_Diffusion_Brain_files/nihms552872f7.gif" class="small-thumb" alt="Figure 7" title="Figure 7" src-large="http://europepmc.org/articles/PMC3951650/bin/nihms552872f7.jpg"></a><div class="icnblk_cntnt" id="lgnd_F7"><div><a class="figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7">Figure 7</a></div><!--caption a4--><div><span>Eight subjects’ mean color maps resulting from a seed placed at right posterior cingulum. Clockwise from top left: empirical functional, linear modeled functional, the proposed network diffusion functional connectivities, and non-linear (Honey</span> <strong>...</strong></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F9" co-legend-rid="lgnd_F9"><a class="icnblk_img figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9"><img src="./Network_Diffusion_Brain_files/nihms552872f9.gif" class="small-thumb" alt="Figure 9" title="Figure 9" src-large="http://europepmc.org/articles/PMC3951650/bin/nihms552872f9.jpg"></a><div class="icnblk_cntnt" id="lgnd_F9"><div><a class="figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9">Figure 9</a></div><!--caption a4--><div><span>Left: Estimate of the right superior motor area and right rectus connectivity from structure compared with fMRI-obtained connectivity over all eight subjects. The two regions are only functionally connected. The proposed model closely estimates the functional</span> <strong>...</strong></div></div></div><p id="P44"><a href="http://europepmc.org/articles/PMC3951650/figure/F8/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Fig. 8</span></a> shows the functional connectivity out of the right frontal superior gyrus. We observe correlation with the right precentral, the right middle frontal, and the middle temporal lobe. Mantini <em>et al.</em> report correlation between superior frontal sulcus and ventral precentral as well as middle frontal sulcus [<a href="http://europepmc.org/articles/pmc3951650#R45" rid="R45" class=" bibr popnode">45</a>].</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F8" co-legend-rid="lgnd_F8"><a class="icnblk_img figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8"><img src="./Network_Diffusion_Brain_files/nihms552872f8.gif" class="small-thumb" alt="Figure 8" title="Figure 8" src-large="http://europepmc.org/articles/PMC3951650/bin/nihms552872f8.jpg"></a><div class="icnblk_cntnt" id="lgnd_F8"><div><a class="figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8">Figure 8</a></div><!--caption a4--><div><span><!--caption a8--><strong>Eight subjects’ mean color maps resulting from a seed placed at right frontal superior</strong></span></div></div></div><p id="P45" class="p p-last">We now show that the proposed model matches true FC for region pairs with and without known SC. We give examples of the proposed model’s estimation of function from structure by considering two functional nodes with non-existing structural connectivity, and two nodes with both functional and structural connectivities. In the first case we consider the nodes of right superior motor area and the right rectus over all eight subjects. The goal is to highlight the model’s ability to estimate function in absence of structural connectivity edge. The resulting connectivity coefficients are given in <a href="http://europepmc.org/articles/PMC3951650/figure/F9/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Fig. 9</span></a> left. Since the two regions are only functionally connected, the structure vs. function plot is all zeros. On the same figure, the estimated FC remains close to the empirically obtained FC for all eight subjects. The model was able to capture the functional connectivity even in absence of corresponding structural connectivity. In the second case we consider two regions that are both structurally as well as functionally connected. <a href="http://europepmc.org/articles/PMC3951650/figure/F9/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Fig. 9</span></a> right shows the connectivity of the superior medial and the right rectus. The red plot gives the correlation of SC and the empirical FC over all eight subjects. With the exception of subjects 5 and 6, the estimated functional connectivity is nearly consistent with the empirical FC.</p></div><div id="S17" class="sec sec-last"><h3>4.5. Model inversion: examples</h3><p id="P46" class="p p-first">We evaluate the estimated structural connectivity matrix 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms552872ig7.jpg" src="./Network_Diffusion_Brain_files/nihms552872ig7.jpg"> for <em>α</em> = 0.1 (where <em>α</em> is as defined in Section 2.6) and <em>α</em> = 0.3 given the mean functional and structural connectivity matrices. The smaller value of <em>α</em> leads to a Pearson correlation with the actual structural connectivity of 0.4901 (<a href="http://europepmc.org/articles/PMC3951650/figure/F10/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Fig. 10c</span></a>), while for <em>α</em> = 0.3 the corresponding Pearson correlation is 0.4199 (<a href="http://europepmc.org/articles/PMC3951650/figure/F10/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Fig. 10d</span></a>). For reference, <a href="http://europepmc.org/articles/PMC3951650/figure/F10/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Fig. 10a</span></a> shows the mean functional connectivity of all eight subjects, while <a href="http://europepmc.org/articles/PMC3951650/figure/F10/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Fig. 10b</span></a> depicts the corresponding structural connectivity of the eight subjects. The matrices’ elements have been arranged in such a way that they are ordered by lobes (frontal, parietal, temporal, occipital, cingulate, and subcortical) and the various regions are listed on the figures. Additionally, The nodes are arranged such that the left and right hemispheres’ nodes alternate in an odd-even fashion.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F10" co-legend-rid="lgnd_F10"><a class="icnblk_img figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10"><img src="./Network_Diffusion_Brain_files/nihms552872f10.gif" class="small-thumb" alt="Figure 10" title="Figure 10" src-large="http://europepmc.org/articles/PMC3951650/bin/nihms552872f10.jpg"></a><div class="icnblk_cntnt" id="lgnd_F10"><div><a class="figpopup" href="http://europepmc.org/articles/PMC3951650/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10">Figure 10</a></div><!--caption a4--><div><span>Estimating structural connectivity from empirical functional connectivity (Fig 10(a)). Empirical structural connectivity is depicted in 10(b). Figures 10(c) and 10(d) show the estimated structural connectivity at SVD thresholds of 0.1 and 0.3, respectively.</span> <strong>...</strong></div></div></div><p id="P47" class="p p-last">Although these results are preliminary, they conclusively show that plausible structural connectivities can be inferred from functional connectivity using the inverse inference approach proposed here. Clearly, there is room for improvement in this process, and we are currently investigating sparsity-inducing inversion algorithms to overcome the ill-conditioning and non-uniqueness of the network diffusion operator which will be reported in a future paper.</p></div></div><div id="S18" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://europepmc.org/articles/pmc3951650#ui-ncbiinpagenav-2" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#ui-ncbiinpagenav3" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S18title">5. Discussion</h2><div id="S19" class="sec sec-first"><h3>5.1. Summary and significance</h3><p id="P48" class="p p-first">We have shown that the correlation structure of BOLD functional brain networks is well represented by a simple network diffusion process on the underlying structural brain network. Such a linear graph model admits a closed form deterministic and testable solution of macroscopic interactions of brain activity without requiring any details of neural coding or their biophysical substrates. The presented correlation numbers (<a href="http://europepmc.org/articles/PMC3951650/table/T1/" target="true" class="fig-table-link table figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) demonstrate improved performance over existing linear and non-linear models. However, the gap disappears after group averaging of networks, as shown by the nonsignificant <em>p</em>-value of Fisher’s transform in the last row of the table. This indicates that while non-linear models are good for canonical brain networks, they may be equally or less accurate for individual subjects than the presented linear model. We demonstrated preliminary evidence for invertibility of our model, which can enable reverse inference from functional to structural connectivity - an intriguing possibility with new potential applications. Philosophically, our work could motivate the field to look for simple models in explaining brain-related phenomena. Although validation was performed using a limited number of subjects, our results are unlikely to be statistical artifacts because no learning of model parameters is necessary - there is only one global diffusivity parameter, whose values are readily estimated from each dataset directly, and which was found to be highly consistent between subjects.</p><p id="P49" class="p p-last">Hence one of the most attractive features of our model is its low dimensionality, which compares favorably with many existing non-linear generative models that can have tens of unknown parameters whose specific values can greatly influence behavior. For instance, in oscillator models completely different quasi-stable states can be realized by different samples of parameter space. We do not claim that linear modeling will address detailed questions regarding the nature and cause of oscillations in the brain - this is still best explored via previously described large, non-linear simulation models. However, our linear model is perfectly adequate for the limited purpose of deriving the correlation structure of low frequency BOLD data, and hence for estimating the long-range functional connectivity of the brain. It provides a simple closed form relating structural and functional networks - a tremendously useful practical feature which could, in the future, obviate the need to obtain resting-state functional networks or allow the merging of structural and functional networks.</p></div><div id="S20" class="sec"><h3>5.2. Relationship to other linear and statistical graph approaches</h3><p id="P50" class="p p-first">The proposed network diffusion model is not a unique theoretical result since it arises naturally from the graph Laplacian, a rather well-studied entity in graph theory [<a href="http://europepmc.org/articles/pmc3951650#R18" rid="R18" class=" bibr popnode">18</a>]. Thus, various flavors of known graph metrics are intimately related to our model, including communicability [<a href="http://europepmc.org/articles/pmc3951650#R27" rid="R27" class=" bibr popnode">27</a>], synchronizability [<a href="http://europepmc.org/articles/pmc3951650#R17" rid="R17" class=" bibr popnode">17</a>], commute distance in random walks [<a href="http://europepmc.org/articles/pmc3951650#R10" rid="R10" class=" bibr popnode">10</a>], and effective resistance in resistive networks [<a href="http://europepmc.org/articles/pmc3951650#R26" rid="R26" class=" bibr popnode">26</a>], to give only a few examples. <a href="http://europepmc.org/articles/pmc3951650#FD5" rid="FD5" class=" disp-formula">Equations 4</a> and <a href="http://europepmc.org/articles/pmc3951650#FD6" rid="FD6" class=" disp-formula">5</a> define the classic network heat diffusion equation, which has been extensively investigated in many network problems, and is the graph-analog of the classical heat equation whose solution is given by the Laplace-Beltrami operator [<a href="http://europepmc.org/articles/pmc3951650#R67" rid="R67" class=" bibr popnode">67</a>]. It is known that for random graphs embedded in Euclidean space, the graph Laplacian approaches the Laplace-Beltrami operator as the density of graph nodes approaches infinity [<a href="http://europepmc.org/articles/pmc3951650#R67" rid="R67" class=" bibr popnode">67</a>]. Zhang and Hancock [<a href="http://europepmc.org/articles/pmc3951650#R67" rid="R67" class=" bibr popnode">67</a>] have used the Laplacian matrix exponentiation for graph-based image smoothing. A network communicability measure on unweighted graphs was proposed [<a href="http://europepmc.org/articles/pmc3951650#R27" rid="R27" class=" bibr popnode">27</a>] recently. Indeed, the exponentiation of adjacency or Laplacian matrix is quite a common tool in graph theory, and has found multiple applications in various contexts [<a href="http://europepmc.org/articles/pmc3951650#R2" rid="R2" class=" bibr popnode">2</a>, <a href="http://europepmc.org/articles/pmc3951650#R57" rid="R57" class=" bibr popnode">57</a>, <a href="http://europepmc.org/articles/pmc3951650#R67" rid="R67" class=" bibr popnode">67</a>]. However, to our knowledge, the use of graph diffusion for modeling brain-specific phenomena is unique to our work, with the exception of our previous publication on network diffusion models of neurodegeneration [<a href="http://europepmc.org/articles/pmc3951650#R53" rid="R53" class=" bibr popnode">53</a>].</p><p id="P51">We do not consider here prior linear modeling of isolated cortical regions [<a href="http://europepmc.org/articles/pmc3951650#R48" rid="R48" class=" bibr popnode">48</a>, <a href="http://europepmc.org/articles/pmc3951650#R49" rid="R49" class=" bibr popnode">49</a>], instead focusing on large connected networks for which the best known linear model was implemented by [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>] following Galán [<a href="http://europepmc.org/articles/pmc3951650#R29" rid="R29" class=" bibr popnode">29</a>], both being examples of multivariate autoregressive (MAR) models [<a href="http://europepmc.org/articles/pmc3951650#R61" rid="R61" class=" bibr popnode">61</a>]. However, general MAR models involve hundreds of unknown parameters whose estimation without prior knowledge is practically unfeasible for the current task [<a href="http://europepmc.org/articles/pmc3951650#R61" rid="R61" class=" bibr popnode">61</a>]. In some respects, our method could be thought of as a MAR model with prior information supplied in the form of anatomic connectivity. In a certain sense, all linear models involving the same graph, including the proposed and Galán models, are going to share many common features, including the eigenspectrum and steady state behavior given by the dominant eigenvector. However, in the context of the structure-function relationship, which linear model one chooses matters, and steady state characterization is insufficient: we need a model whose dynamics change with time, and at some time scale reproduce the correct functional structure. Although previous authors have concluded that linear models are not as good as non-linear ones, our results show that the problem is not with linear models <em>per se</em> but which linear model is being used. The utility of linear graph models is revealed after careful first-principles modeling. Plausible neuroscientific constraints we have introduced appear necessary to access the appropriate regime in the space of all linear graph models. Other differences between the Galán and proposed models may be enumerated. Although the former obviously admits a closed-form solution, this has not historically been explored for long-range brain networks. Our model explicitly employs the Laplacian, whereas the Galán model uses the adjacency matrix. The two models become directly comparable only for uniform node degree, which is unrealistic for the brain. The Galán model must be explicitly normalized in order to guarantee stability, whereas the diffusion model is always stable.</p><p id="P52" class="p p-last">Model-free approaches should also be mentioned in the context of structure-function relationship. Purely statistical tools like Pearson correlation are good examples, as is more recent work on partial correlation or partial least squares. Partial correlation has been suggested as a way of disambiguating direct versus indirect functional connections between brain regions. A detailed study of the ability of various types of correlation and other statistical measures to reproduce known anatomic connections in the visual cortex was conducted in [<a href="http://europepmc.org/articles/pmc3951650#R20" rid="R20" class=" bibr popnode">20</a>]. Partial correlation is a good candidate for inferring structure from function. How these methods perform in comparison to the purely deterministic approach using graph eigen-analysis suggested in this paper is an interesting question which will be investigated in the future. However, we believe a model-based approach has a better chance of constraining this inherently ill-posed problem compared to purely statistical methods.</p></div><div id="S21" class="sec"><h3>5.3. Relationship to non-linear neural mass, field and oscillator models</h3><p id="P53" class="p p-first">The classic NMM did not consider large networked regions, but subsequent extension to small networks connecting 3–4 distinct functional domains like memory, execution, vision, etc., called Dynamic Causal Models (DCMs), were proposed. NMMs were extended to spatially distributed neural fields using mean field steady state approximations, yielding a traveling wave involving spatiotemporal convolution with spatially invariant connectivity kernels modeled Green function [<a href="http://europepmc.org/articles/pmc3951650#R6" rid="R6" class=" bibr popnode">6</a>, <a href="http://europepmc.org/articles/pmc3951650#R19" rid="R19" class=" bibr popnode">19</a>, <a href="http://europepmc.org/articles/pmc3951650#R24" rid="R24" class=" bibr popnode">24</a>, <a href="http://europepmc.org/articles/pmc3951650#R50" rid="R50" class=" bibr popnode">50</a>] - a kind of spatial Laplacian analogous to our network Laplacian. Traditional field models, however, have generally employed spatially invariant connectivity kernels, which are difficult to apply to long-range, inhomogeneous cortico-cortical connections. An example where this was successful is in modeling thalamo-cortical connectivity [<a href="http://europepmc.org/articles/pmc3951650#R41" rid="R41" class=" bibr popnode">41</a>, <a href="http://europepmc.org/articles/pmc3951650#R52" rid="R52" class=" bibr popnode">52</a>], but we are aware of no report on whole brain resting state networks.</p><p id="P54">Neural mass models are much more successfully applied to the current task, whereby NMMs are deployed at the nodes of long-range networks, whose dynamics are then coupled via anatomic connectivity [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>, <a href="http://europepmc.org/articles/pmc3951650#R51" rid="R51" class=" bibr popnode">51</a>]. Our coupled NMM implementation based on [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>] matched our low-frequency BOLD correlations better than previous reports [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>]. These coupled NMMs behave chaotically or stably, depending on coupling strength and propagation delays between regions.</p><p id="P55">For the practical task of inferring the correlation of the structure-function relationship, however, they present some limitations vis-á-vis the proposed approach. First, these generative models require large simulation runs, making them impractical for the purpose of routine analysis. Second, while both the actuators (chaotic oscillators) and output metric (phase synchronization) in these simulations have a long tradition [<a href="http://europepmc.org/articles/pmc3951650#R65" rid="R65" class=" bibr popnode">65</a>], their abstract nature makes them difficult to apply and interpret in practical connectome analysis. Although this elegant theory is now finding its way to practical applications [<a href="http://europepmc.org/articles/pmc3951650#R22" rid="R22" class=" bibr popnode">22</a>], its agreement with actual measurements remains difficult to assess. The empirical data in [<a href="http://europepmc.org/articles/pmc3951650#R22" rid="R22" class=" bibr popnode">22</a>] do not stand out compared to [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>] or the current results, although this could change with new analyses. Another limitation is the need to fit a large number of model parameters, an area where a linear model with a single global parameter has distinct advantages. Finally, much of the oscillator modeling has been performed using simulated [<a href="http://europepmc.org/articles/pmc3951650#R44" rid="R44" class=" bibr popnode">44</a>] or macaque [<a href="http://europepmc.org/articles/pmc3951650#R21" rid="R21" class=" bibr popnode">21</a>, <a href="http://europepmc.org/articles/pmc3951650#R31" rid="R31" class=" bibr popnode">31</a>] networks; hence further applications in human health and disease will be keenly awaited.</p><p id="P56" class="p p-last">Our correlation numbers are consistently lower than those in [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>]. Group averaged structural and functional connectivity was used in [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>], whereas we used individual subjects SC and FC, which are typically more variable and challenging than group averages. Our SC networks show lesser degree of inter-hemispheric connectivity than [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>], partly due to the limitations of tractography algorithms this could lower the SC-FC relationship. There are other methodological differences which might account for the discrepancy, for instance their use of 998 regions compared to 90 in our case. Although their results after reducing the data to 66 regions shows a very high correlation (<em>R</em> = 0.70) compared to ours, this could simply be a result of averaging, an impression reinforced by the fact that the authors’ original 998-region correlation (<em>R</em> = 0.46) is much weaker. Finally, we do not perform the somewhat controversial statistical rescaling of SC edge weights to fit a Gaussian distribution, as reported in [<a href="http://europepmc.org/articles/pmc3951650#R36" rid="R36" class=" bibr popnode">36</a>].</p></div><div id="S22" class="sec"><h3>5.4. Why does graph diffusion work at all - Emergent linearity in brain dynamics</h3><p id="P57" class="p p-first">It may be puzzling that a linear model with no neuroscientific detail apart from a macroscopic view of population dynamics is able to equal or outperform richly detailed non-linear dynamic models described above. In motivating our work we recruit two commonalities gleaned from them: a) practically any method of coupling local dynamics via realistic anatomic connectivity is able to sustain the kinds of steady-state spatiotemporal dynamic behavior observed in the brain; and b) the network effect given by anatomic connectivity appears to dominate over local dynamics of the brain, especially at low (BOLD) frequencies. Thus steady state correlation patterns at BOLD frequencies should be insensitive to higher order non-linearities like multistable transient states driven by stochastic noise. This impression is reinforced by comparing <a href="http://europepmc.org/articles/PMC3951650/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, which shows how long-range correlations change with the global connectivity coupling parameter <em>βt</em>, with a similar figure from the coupled oscillator paper [<a href="http://europepmc.org/articles/pmc3951650#R22" rid="R22" class=" bibr popnode">22</a>] and indeed with computational models of various vintages and frequency ranges [<a href="http://europepmc.org/articles/pmc3951650#R15" rid="R15" class=" bibr popnode">15</a>, <a href="http://europepmc.org/articles/pmc3951650#R31" rid="R31" class=" bibr popnode">31</a>, <a href="http://europepmc.org/articles/pmc3951650#R44" rid="R44" class=" bibr popnode">44</a>]. Most of these models exhibit the characteristic bell-shaped behavior with respect to the global coupling parameter. A consistent finding of these investigations is a “quiescent” ground state characterized by low frequency correlations - we speculate that it is this regime that allows our linear model to recapitulate second-order functional correlations from quasi-chaotic signals. This does not imply that the previous non-linear oscillator models are approximately linear (they are not, since they display multi-stable quasi-chaotic behavior), but that the gross behavior of networked sites has a surprisingly consistent pattern over a range of models. Ultimately, the best argument for our model is perhaps simply that it seems to work.</p><p id="P58" class="p p-last">The literature on complex systems suggests that just as simple local interactions can give rise to highly complex global effects; conversely, simple macroscopic behavior can emerge from complex local processes. The emergent behavior of large scale complex systems can be surprisingly linear within large phase domains bounded by (non-linear) phase transitions. Indeed, the emergence of predictable and regular behavior from chaotic ensembles is considered a hallmark of complexity [<a href="http://europepmc.org/articles/pmc3951650#R55" rid="R55" class=" bibr popnode">55</a>]. For example, the admittance of large electrical networks of capacitative and resistive elements is known to be chaotic, yet its frequency response is essentially linear in large frequency ranges [<a href="http://europepmc.org/articles/pmc3951650#R5" rid="R5" class=" bibr popnode">5</a>]. This kind of predictable, regular emergent behavior is seen in complex systems as varied as the flocking of geese [<a href="http://europepmc.org/articles/pmc3951650#R47" rid="R47" class=" bibr popnode">47</a>] and complex biological signaling networks [<a href="http://europepmc.org/articles/pmc3951650#R9" rid="R9" class=" bibr popnode">9</a>]. Our results similarly suggest that the macroscopic consequence of the interaction of complex but coupled neuronal processes in the brain are surprisingly amenable to coarse mechanistic modeling.</p></div><div id="S23" class="sec sec-last"><h3>5.5. Limitations and future work</h3><p id="P59" class="p p-first">The second order correlation of the linear Galán model is computable in closed form via the network’s eigen decomposition; however here it was investigated via stochastic simulations in keeping with prior work to ensure effective comparison with earlier results. Network extraction requires a lengthy, involved process [<a href="http://europepmc.org/articles/pmc3951650#R38" rid="R38" class=" bibr popnode">38</a>] whose deficiencies are inherited by our results. Our choice of the AAL atlas was motivated by its proven anatomic specificity and widespread use in related network analysis work. With only 90 nodes, our network might be considered “lumpy”; yet it is unclear if a finer scale network with more nodes will yield improvements. While a coarse atlas precludes finer scales of functional patterning, it improves the reliability of network connectivity. Variations in size and shape of cortical regions was not accounted for in our model. We did not consider regionally varying model parameters.</p><p id="P60">We note that the results reported in this work are limited by distance bias. Distance bias is inherent to tractography from which the structural connectivity matrix is obtained, and is a constrain of current technology. Another potential issue is the thresholding operation we used to remove connections considered to be zero. <a href="http://europepmc.org/articles/PMC3951650/figure/F5/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a> suggests that the proposed model is not very sensitive to edge thresholding - a strength of the linear approach. We further employed a statistical significance threshold based on variance over all subjects, but due to their small number a bootstrap resampling technique might be more effective, and will be considered in future work. Our model does not possess oscillatory behavior, which would require accurate knowledge of path delays - in systems terms the Laplace transform has a real-valued pole - because the rich frequency content of neural populations is difficult to access and interpret from low-frequency BOLD signals. Our results support the view that frequency characterization is not necessary in capturing the whole brain correlation structure of BOLD activation. However, a fuller characterization would require path delays, which we will pursue in future work. This would be especially critical for modeling richer frequency signals, for example from magnetoencephalograhy (MEG). MEG can facilitate an investigation of the frequency response of the diffusion model and its eigenmodes - an aspect we are unable to explore due to BOLD’s poor temporal resolution. Finally, we are cognizant of inherent limitations of linear approximations of complex dynamic phenomena, and the risks associated with overinterpretation.</p><p id="P61" class="p p-last">We emphasize that this work is only aimed at capturing the stationary correlation structure of functional activation, i.e. its functional connectivity, and it has no ability to generate actual time series of functional activity. The precise time series is neither germane nor accessible by any other method either, since an infinite number of time-series can have the same functional connectivity. A more serious limitation of our model is that it currently does not provide frequency information of the kind a generative simulation model can deliver. However, this is not a fundamental feature of linear modeling, and in future work we will develop frequency-resolved network diffusion models. Low frequency BOLD data are not adequate for that purpose, and this effort will necessitate EEG and MEG acquisitions.</p></div></div><div id="unreferenced-ft" class="tsec sec headless whole_rhythm"><div class="unreferenced">​
                    <div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="caption"><h3>Highlights</h3></div><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
--><li><div>Resting state functional connectivity estimate obtained from structural connectivity</div></li><li><div>Proposed model is novel and linear, using the network Laplacian</div></li><li><div>Comparison with published models suggest comparable or superior performance</div></li></ul></div></div></div><div id="S25" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://europepmc.org/articles/pmc3951650#ui-ncbiinpagenav-2" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#ui-ncbiinpagenav3" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S25title">Acknowledgments</h2><div class="sec"><p id="__pid711888">Authors would like to thank Olaf Sporns for supplying computer scripts that allowed the simulation of the non-linear model. AR and FA were supported by NIH grant R01 NS075425. HUV is supported by NSF grant 0956306.</p></div></div><div id="__fn-groupid711892" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://europepmc.org/articles/pmc3951650#ui-ncbiinpagenav-2" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#ui-ncbiinpagenav3" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__fn-groupid711892title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="__pid711897" class="p p-first-last"><strong>Publisher's Disclaimer: </strong>This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p><p></p></div></div><div id="__ref-listid711905" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://europepmc.org/articles/pmc3951650#ui-ncbiinpagenav-2" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#ui-ncbiinpagenav3" title="Go to other sections in this page" role="button" aria-expanded="false">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__ref-listid711905title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Achard S, Ed B. Efficiency and cost of economical brain functional networks. <span><span class="ref-journal">PLoS Comput Biol. </span>2007;<span class="ref-vol">3</span>:e17.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/17274684" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Agaskar A, Lu YM.  <span class="ref-journal">An uncertainty principle for functions defined on graphs.</span> Vol. 8138. SPIE; 2011. p. 81380T.</span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Albright TD. Direction and orientation selectivity of neurons in visual area MT of the macaque. <span><span class="ref-journal">Journal of Neurophysiology. </span>1984;<span class="ref-vol">52</span>:1106–1130.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/6520628" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Alemán-Gómez Y, Melie-García L, Valdés-Hernandez P. BASPM: Toolbox for automatic parcellation of brain structures. <span><span class="ref-journal">Human Brain Mapping </span>2006</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Almond D, Budd C, McCullen N.  Emergent behaviour in large electrical networks. In: Georgoulis EH, Iske A, Levesley J, editors. <span class="ref-journal">Approximation Algorithms for Complex Systems, Vol. 3 of Springer Proceedings in Mathematics.</span> Springer; Berlin Heidelberg: 2011. pp. 3–26.</span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Amari S. Dynamics of pattern formation in lateral-inhibition type neural fields. <span><span class="ref-journal">Biological Cybernetics. </span>1977;<span class="ref-vol">27</span>:77–87.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/911931" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Ashburner J. A fast diffeomorphic image registration algorithm. <span><span class="ref-journal">NeuroImage. </span>2007;<span class="ref-vol">38</span>:95–113.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/17761438" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Bassett DS, Greenfield DL, Meyer-Lindenberg A, Weinberger DR, Moore SW, Bullmore ET. Efficient physical embedding of topologically complex information processing networks in brains and computer circuits. <span><span class="ref-journal">PLoS Comput Biol. </span>2010;<span class="ref-vol">6</span>:e1000748.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC2858671/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/20421990" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Bhalla US. Understanding complex signaling networks through models and metaphors. <span><span class="ref-journal">Progress in Biophysics and Molecular Biology </span>2003</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/12475569" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Boley D, Ranjan G, Zhang ZL. Commute times for a directed graph using an asymmetric laplacian. <span><span class="ref-journal">Linear Algebra and its Applications. </span>2011;<span class="ref-vol">435</span>:224–242.</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Breakspear M, Terry J, Friston K. Modulation of excitatory synaptic coupling facilitates synchronization and complex dynamics in a biophysical model of neuronal dynamics. <span><span class="ref-journal">Neurocomputing. </span>2003;<span class="ref-vol">52–54</span>:151–158.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/14653499" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Bullmore E, Sporns O. Complex brain networks: graph theoretical analysis of structural and functional systems. <span><span class="ref-journal">Nature Reviews Neuroscience. </span>2009;<span class="ref-vol">10</span>:186–198.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/19190637" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Bullmore ET, Bassett DS. Brain graphs: Graphical models of the human brain connectome. <span><span class="ref-journal">Annual Review of Clinical Psychology. </span>2011;<span class="ref-vol">7</span>:113–140.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/21128784" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Cabeza R, Kingstone A.  <span class="ref-journal">Handbook of functional neuroimaging of cognition.</span> 2 MIT Press; 2006. </span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Cabral J, Hugues E, Sporns O, Deco G. Role of local network oscillations in resting-state functional connectivity. <span><span class="ref-journal">NeuroImage. </span>2011;<span class="ref-vol">57</span>:130–139.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/21511044" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Calhoun VD, Liu J, Adal T. A review of group ICA for fMRI data and ICA for joint inference of imaging, genetic, and ERP data. <span><span class="ref-journal">NeuroImage. </span>2009;<span class="ref-vol">45</span>:S163–S172.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC2651152/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/19059344" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Chen G, Duan Z. Network synchronizability analysis: A graph-theoretic approach. <span><span class="ref-journal">Chaos: An Interdisciplinary Journal of Nonlinear Science. </span>2008;<span class="ref-vol">18</span>:037102.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/19045476" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Chung FRK.  <span class="ref-journal">Number 92 in Regional Conference Series in Mathematics.</span> AMS; 1997. Spectral Graph Theory.</span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Daunizeau J, Kiebel SJ, Friston KJ. Dynamic causal modelling of distributed electromagnetic responses. <span><span class="ref-journal">NeuroImage. </span>2009;<span class="ref-vol">47</span>:590–601.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC2722904/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/19398015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Dawson DA, Cha K, Lewis LB, Mendola JD, Shmuel A. Evaluation and calibration of functional network modeling methods based on known anatomical connections. <span><span class="ref-journal">NeuroImage. </span>2013;<span class="ref-vol">67</span>:331–343.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/23153969" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Deco G, Jirsa V, McIntosh A, Sporns O, Kötter R. Key role of coupling, delay, and noise in resting brain fluctuations. <span><span class="ref-journal">Proceedings of the National Academy of Sciences </span>2009</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC2690605/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/19497858" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Deco G, Jirsa VK. Ongoing cortical activity at rest: Criticality, multistability, and ghost attractors. <span><span class="ref-journal">The Journal of Neuroscience. </span>2012;<span class="ref-vol">32</span>:3366–3375.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/22399758" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Deco G, Jirsa VK, Robinson PA, Breakspear M, Friston K. The dynamic brain: From spiking neurons to neural masses and cortical fields. <span><span class="ref-journal">PLoS Comput Biol. </span>2008a;<span class="ref-vol">4</span>:e1000092.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC2519166/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/18769680" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Deco G, Jirsa VK, Robinson PA, Breakspear M, Friston K. The dynamic brain: From spiking neurons to neural masses and cortical fields. <span><span class="ref-journal">PLoS Comput Biol. </span>2008b;<span class="ref-vol">4</span>:e1000092.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC2519166/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/18769680" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Deco G, Senden M, Jirsa V. How anatomy shapes dynamics: A semi-analytical study of the brain at rest by a simple spin model. <span><span class="ref-journal">Frontiers in Computational Neuroscience. </span>2012:6.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC3447303/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/23024632" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Ellens W, Spieksma F, van Mieghem P, Jamakovic A, Kooij R. Effective graph resistance. <span><span class="ref-journal">Linear Algebra and its Applications. </span>2011;<span class="ref-vol">435</span>:2491–2506.</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Estrada E, Hatano N. Communicability in complex networks. <span><span class="ref-journal">Phys Rev E. </span>2008;<span class="ref-vol">77</span>:036111.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/18517465" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Friston K, Holmes A, Worsley K, Poline JP, Frith CD, Frackowiak RSJ. Statistical parametric maps in functional imaging: A general linear approach. <span><span class="ref-journal">Hum Brain Mapp. </span>1994;<span class="ref-vol">2</span>:189–210.</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Galán RF. On how network architecture determines the dominant patterns of spontaneous neural activity. <span><span class="ref-journal">PLoS ONE. </span>2008;<span class="ref-vol">3</span>:e2148.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC2374893/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/18478091" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Ghosh A, Rho Y, McIntosh A, Kötter R, Jisra V. Cortical network dynamics with time delays reveals functional connectivity in the resting brain. <span><span class="ref-journal">Cogn Neurodyn. </span>2008a;<span class="ref-vol">2</span>:115–120.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC2427063/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/19003478" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Ghosh A, Rho Y, McIntosh AR, Kötter R, Jirsa VK. Noise during rest enables the exploration of the brain’s dynamic repertoire. <span><span class="ref-journal">PLoS Comput Biol. </span>2008b;<span class="ref-vol">4</span>:e1000196.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC2551736/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/18846206" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Gong G, He Y, Concha L, Lebel C, Gross D. Mapping anatomical connectivity patterns of human cerebral cortex using in vivo diffusion tensor imaging tractography. <span><span class="ref-journal">Cereb Cortex. </span>2009;<span class="ref-vol">19</span>:524–536.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC2722790/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/18567609" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Greicius MD, Supekar K, Menon V, Dougherty RF. Resting-state functional connectivity reflects structural connectivity in the default mode network. <span><span class="ref-journal">Cerebral Cortex. </span>2009;<span class="ref-vol">19</span>:72–78.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC2605172/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/18403396" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Honey CJ, Kötter R, Breakspear M, Sporns O. Network structure of cerebral cortex shapes functional connectivity on multiple time scales. <span><span class="ref-journal">Proceedings of the National Academy of Sciences. </span>2007;<span class="ref-vol">104</span>:10240–10245.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC1891224/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/17548818" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Honey CJ, Thivierge JP, Sporns O. Can structure predict function in the human brain? <span><span class="ref-journal">NeuroImage. </span>2010;<span class="ref-vol">52</span>:766–776.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/20116438" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Honey C, Sporns O, Cammoun L, Gigandet X, Thiran J, Meuli R, Hagmann P. Predicting human resting-state functional connectivity from structural connectivity. <span><span class="ref-journal">Proceedings of the National Academy of Sciences. </span>2009;<span class="ref-vol">106</span>:2035–2040.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC2634800/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/19188601" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Hutchison RM, Womelsdorf T, Gati JS, Everling S, Menon RS. Resting-state networks show dynamic functional connectivity in awake humans and anesthetized macaques. <span><span class="ref-journal">Human Brain Mapping. </span>2013;<span class="ref-vol">34</span>:2154–2177.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/22438275" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Iturria-Medina Y, Canales-Rodríguez E, Melie-García L, Valdés-Hernández P, Martínez-Montes E, Alemán-Gómez Y, Sánchez-Bornot J. Characterizing brain anatomical connections using diffusion weighted MRI and graph theory. <span><span class="ref-journal">NeuroImage. </span>2007;<span class="ref-vol">36</span>:645–660.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/17466539" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Iturria-Medina Y, Sotero RC, Canales-Rodríguez EJ, Alemán-Gómez Y, Melie-García L. Studying the human brain anatomical network via diffusion-weighted MRI and graph theory. <span><span class="ref-journal">NeuroImage. </span>2008;<span class="ref-vol">40</span>:1064–1076.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/18272400" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Ivković M, Kuceyeski A, Raj A. Statistics of weighted brain networks reveal hierarchical organization and gaussian degree distribution. <span><span class="ref-journal">PLoS ONE. </span>2012;<span class="ref-vol">7</span>:e35029.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC3382201/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/22761649" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Jirsa VK, Kelso JAS. Spatiotemporal pattern formation in neural systems with heterogeneous connection topologies. <span><span class="ref-journal">Phys Rev E. </span>2000;<span class="ref-vol">62</span>:8462–8465.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/11138148" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Jones DT, Vemuri P, Murphy MC, Gunter JL, Senjem ML, Machulda MM, Przybelski SA, Gregg BE, Kantarci K, Knopman DS, Boeve BF, Petersen RC, Jack JCR. Non-stationarity in the resting brains modular architecture. <span><span class="ref-journal">PLoS ONE. </span>2012;<span class="ref-vol">7</span>:e39731.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC3386248/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/22761880" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Joyce KE, Satoru H, JLP The human functional brain network demonstrates structural and dynamical resilience to targeted attack. <span><span class="ref-journal">PLoS Comput Biol. </span>2013;<span class="ref-vol">9</span>:e1002885.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC3554573/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/23358557" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Knock S, McIntosh A, Sporns O, Kötter R, Hagmann P, Jirsa V. The effects of physiologically plausible connectivity structure on local and global dynamics in large scale brain models. <span><span class="ref-journal">Journal of Neuroscience Methods. </span>2009;<span class="ref-vol">183</span>:86–94.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/19607860" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Mantini D, Perrucci MG, Del Gratta C, Romani GL, Corbetta M. Electrophysiological signatures of resting state networks in the human brain. <span><span class="ref-journal">Proceedings of the National Academy of Sciences. </span>2007;<span class="ref-vol">104</span>:13170–13175.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC1941820/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/17670949" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Mars RB, Jbabdi S, Sallet J, O’Reilly JX, Croxson PL, Olivier E, Noonan MP, Bergmann C, Mitchell AS, Baxter MG, Behrens TEJ, Johansen-Berg H, Tomassini V, Miller KL, Rushworth MFS. Diffusion-weighted imaging tractography-based parcellation of the human parietal cortex and comparison with human and macaque resting-state functional connectivity. <span><span class="ref-journal">The Journal of Neuroscience. </span>2011;<span class="ref-vol">31</span>:4087–4100.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC3091022/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/21411650" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Martinez S, Cortes J, Bullo F. Motion coordination with distributed information. <span><span class="ref-journal">IEEE Control Systems. </span>2007;<span class="ref-vol">27</span>:75–88.</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Moran R, Kiebel S, Stephan K, Reilly R, Daunizeau J, Friston K. A neural mass model of spectral responses in electrophysiology. <span><span class="ref-journal">NeuroImage. </span>2007;<span class="ref-vol">37</span>:706–720.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC2644418/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/17632015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Moran RJ, Stephan KE, Dolan RJ, Friston KJ. Consistent spectral predictors for dynamic causal models of steady-state responses. <span><span class="ref-journal">NeuroImage. </span>2011;<span class="ref-vol">55</span>:1694–1708.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC3093618/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/21238593" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Nunez PL. The brain wave equation: a model for the EEG. <span><span class="ref-journal">Mathematical Biosciences. </span>1974;<span class="ref-vol">21</span>:279–297.</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Ponten S, Daffertshofer A, Hillebrand A, Stam C. The relationship between structural and functional connectivity: Graph theoretical analysis of an EEG neural mass model. <span><span class="ref-journal">NeuroImage. </span>2010;<span class="ref-vol">52</span>:985–994.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/19853665" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Qubbaj MR, Jirsa VK. Neural field dynamics with heterogeneous connection topology. <span><span class="ref-journal">Phys Rev Lett. </span>2007;<span class="ref-vol">98</span>:238102.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/17677938" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Raj A, Kuceyeski A, Weiner M. A network diffusion model of disease progression in dementia. <span><span class="ref-journal">Neuron. </span>2012;<span class="ref-vol">73</span>:1204–1215.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC3623298/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/22445347" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Rubinov M, Sporns O. Complex network measures of brain connectivity: Uses and interpretations. <span><span class="ref-journal">NeuroImage. </span>2010;<span class="ref-vol">52</span>:1059–1069.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/19819337" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Shalizi C.  <span class="ref-journal">PhD diss.</span> 2001. Causal Architecture, Complexity and Self-Organization in Time Series and Cellular Automata.</span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="element-citation">Shi J, Malik J. Normalized cuts and image segmentation. <span><span class="ref-journal">Pattern Analysis and Machine Intelligence, IEEE Transactions on. </span>2000;<span class="ref-vol">22</span>:888–905.</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="element-citation">Smolka B, Wojciechowski K. Random walk approach to image enhancement. <span><span class="ref-journal">Signal Process. </span>2001;<span class="ref-vol">81</span>:465–482.</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">58. <span class="element-citation">Song X, Dong Z, Long X, LIS, Zuo X, et al.  DPARSF: aMATLAB toolbox for “pipeline” data analysis of resting-state fMRI. <span><span class="ref-journal">Front Syst Neurosci. </span>2010:4.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC2889691/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/20577591" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">59. <span class="element-citation">Stephan KE, Kasper L, Harrison LM, Daunizeau J, den Ouden HE, Breakspear M, Friston KJ. Nonlinear dynamic causal models for fMRI. <span><span class="ref-journal">NeuroImage. </span>2008;<span class="ref-vol">42</span>:649–662.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://europepmc.org/articles/PMC2636907/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/18565765" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R60">60. <span class="element-citation">Tzourio-Mazoyer N, Landeau B, Papathanassiou D, Crivello F, Etard O, Delcroix N, Mazoyer B, Joliot M. Automated anatomical labeling of activations in SPM using a macroscopic anatomical parcellation of the MNI MRI single-subject brain. <span><span class="ref-journal">NeuroImage. </span>2002;<span class="ref-vol">15</span>:273–289.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/11771995" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R61">61. <span class="element-citation">Valdes-Sosa P. Spatio-temporal autoregressive models defined over brain manifolds. <span><span class="ref-journal">Neuroinformatics. </span>2004;<span class="ref-vol">2</span>:239–250.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/15319519" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R62">62. <span class="element-citation">van den Heuvel M, Mandl R, Luigjes J, Pol HH. Microstructural organization of the cingulum tract and the level of default mode functional connectivity. <span><span class="ref-journal">The Journal of Neuroscience. </span>2008;<span class="ref-vol">28</span>:10844–10851.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/18945892" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R63">63. <span class="element-citation">van den Heuvel MP, Mandl RC, Kahn RS, Pol HEH. Functionally linked resting-state networks reflect the underlying structural connectivity architecture of the human brain. <span><span class="ref-journal">Human Brain Mapping. </span>2009;<span class="ref-vol">30</span>:3127–3141.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/19235882" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R64">64. <span class="element-citation">van den Heuvel MP, Pol HEH. Exploring the brain network: A review on resting-state fmri functional connectivity. <span><span class="ref-journal">European Neuropsychopharmacology. </span>2010;<span class="ref-vol">20</span>:519–534.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/20471808" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R65">65. <span class="element-citation">Varela F, Rodriguez E, Martinerie J. The brainweb: Phase synchronization and large-scale integration. <span><span class="ref-journal">Nat Rev Neurosci. </span>2001;<span class="ref-vol">2</span>:229–239.</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/11283746" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R66">66. <span class="element-citation">Whitfield-Gabrieli S, Nieto-Castanon A. Conn: A functional connectivity toolbox for correlated and anticorrelated brain networks. <span><span class="ref-journal">Brain Connectivity </span>2012</span>  <span class="nowrap ref pubmed">[<a href="http://europepmc.org/abstract/MED/22642651" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3358892&amp;issue-id=127779&amp;journal-id=1130&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R67">67. <span class="element-citation">Zhang F, Hancock ER. Graph spectral image smoothing using the heat kernel. <span><span class="ref-journal">Pattern Recognition. </span>2008;<span class="ref-vol">41</span>:3328–3342.</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li role="presentation"><a href="http://europepmc.org/articles/pmc3951650#__abstractid760818title" role="menuitem" tabindex="-1">Abstract</a></li><li role="presentation"><a href="http://europepmc.org/articles/pmc3951650#S1title" role="menuitem" tabindex="-1">1. Introduction</a></li><li role="presentation"><a href="http://europepmc.org/articles/pmc3951650#S2title" role="menuitem" tabindex="-1">2. Theory</a></li><li role="presentation"><a href="http://europepmc.org/articles/pmc3951650#S9title" role="menuitem" tabindex="-1">3. Methods</a></li><li role="presentation"><a href="http://europepmc.org/articles/pmc3951650#S12title" role="menuitem" tabindex="-1">4. Results</a></li><li role="presentation"><a href="http://europepmc.org/articles/pmc3951650#S18title" role="menuitem" tabindex="-1">5. Discussion</a></li><li role="presentation"><a href="http://europepmc.org/articles/pmc3951650#S25title" role="menuitem" tabindex="-1">Acknowledgments</a></li><li role="presentation"><a href="http://europepmc.org/articles/pmc3951650#__fn-groupid711892title" role="menuitem" tabindex="-1">Footnotes</a></li><li role="presentation"><a href="http://europepmc.org/articles/pmc3951650#__ref-listid711905title" role="menuitem" tabindex="-1">References</a></li></ul></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav3"><li role="presentation"><a href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#__abstractid760818title" role="menuitem" tabindex="-1">Abstract</a></li><li role="presentation"><a href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#S1title" role="menuitem" tabindex="-1">1. Introduction</a></li><li role="presentation"><a href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#S2title" role="menuitem" tabindex="-1">2. Theory</a></li><li role="presentation"><a href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#S9title" role="menuitem" tabindex="-1">3. Methods</a></li><li role="presentation"><a href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#S12title" role="menuitem" tabindex="-1">4. Results</a></li><li role="presentation"><a href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#S18title" role="menuitem" tabindex="-1">5. Discussion</a></li><li role="presentation"><a href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#S25title" role="menuitem" tabindex="-1">Acknowledgments</a></li><li role="presentation"><a href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#__fn-groupid711892title" role="menuitem" tabindex="-1">Footnotes</a></li><li role="presentation"><a href="file:///media/abbhinav/Data%20Drive/College%20Data/Semester%205/Honors%20Project/Network%20diffusion%20accurately%20models%20the%20relationship%20between%20structural%20and%20functional%20brain...%20-%20Europe%20PMC%20Article%20-%20Europe%20PubMed%20Central.html#__ref-listid711905title" role="menuitem" tabindex="-1">References</a></li></ul></div></div><!--post-content--></div>

<script type="text/javascript">
$(document).ready(function(){
   $('A.cite-reflink[rid]').mouseover(function(){ TagToTipAllowed($(this).attr('rid'), WIDTH, 300, FIX, [this, 0, 0], ABOVE, true); });
   $('A.cite-reflink[rid]').mouseout(function() { UnTipAllowed(); });
});
</script>


</div>
<div align="center">

</div>
</td>
</tr>
</tbody></table>
</div>
<div class="ui-tabs-panel ui-tabs-hide ui-widget-content ui-corner-bottom" id="fragment-related-citations">
<div id="id1d">
<span id="references_loading_label"><img src="./Network_Diffusion_Brain_files/ajax-loader.gif" height="25 px" width="25 px"></span>
<div align="right" style="display:none"><small><a href="http://europepmc.org/Help#citationsnetwork">&nbsp;<img border="0" src="./Network_Diffusion_Brain_files/qmark.png">How does Europe PubMed Central derive its citations network?</a></small></div>
<div class="citations">
<h3>

</h3>

<div class="citedByHolder collapsed" style="display:none">
<div class="citations_listing">

</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</div>
<br>
<br>
<div class="citations">
<h3></h3>

<div class="citesFollowingHolder collapsed" style="display:none">
<div class="citations_listing">

</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</div>
</div>
</div>
<div class="ui-tabs-panel ui-tabs-hide ui-widget-content ui-corner-bottom" id="fragment-related-bioentities">
<div id="id1e">
<span id="loading_bioentities_label"><img src="./Network_Diffusion_Brain_files/ajax-loader.gif" height="25 px" width="25 px"></span>

<div id="protiensHolder" class="collapsed">
<div id="proteins" style="display:none;">
<table cellpadding="0">
<tbody><tr>
<td colspan="2">
<b><span>Genes &amp; Proteins</span></b>
</td>
</tr>
<tr>
<td valign="top" width="500" align="left">

<div class="bxref1 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>


<div class="bxref2 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>

<div class="btmsmr collapsed" style="display:none;">
<div class="relartsdiv">
<h1> </h1>

<div class="tmsummary">
<span class="text1">
<div width="100%">

</div>
</span>
</div>

</div>
<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</td>
</tr>
</tbody></table>
</div>
</div>


<div id="protiensintHolder" class="collapsed">
<div id="protiensint" style="display:none;">
<table cellpadding="0">
<tbody><tr>
<td colspan="2">
<b><span>Protein Interactions</span></b>
</td>
</tr>
<tr>
<td valign="top" width="500" align="left">

<div class="bxref1 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>


<div class="bxref2 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>

<div class="btmsmr collapsed" style="display:none;">
<div class="relartsdiv">
<h1> </h1>

<div class="tmsummary">
<span class="text1">
<div width="100%">

</div>
</span>
</div>

</div>
<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</td>
</tr>
</tbody></table>
</div>
</div>


<div id="protienssqnHolder" class="collapsed">
<div id="protienssqn" style="display:none;">
<table cellpadding="0">
<tbody><tr>
<td colspan="2">
<b><span>Protein Families</span></b>
</td>
</tr>
<tr>
<td valign="top" width="500" align="left">

<div class="bxref1 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>


<div class="bxref2 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>

<div class="btmsmr collapsed" style="display:none;">
<div class="relartsdiv">
<h1> </h1>

<div class="tmsummary">
<span class="text1">
<div width="100%">

</div>
</span>
</div>

</div>
<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</td>
</tr>
</tbody></table>
</div>
</div>


<div id="nuclesqnHolder" class="collapsed">
<div id="nuclesqn" style="display:none;">
<table cellpadding="0">
<tbody><tr>
<td colspan="2">
<b><span>Nucleotide Sequences</span></b>
</td>
</tr>
<tr>
<td valign="top" width="500" align="left">

<div class="bxref1 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>


<div class="bxref2 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>

<div class="btmsmr collapsed" style="display:none;">
<div class="relartsdiv">
<h1> </h1>

<div class="tmsummary">
<span class="text1">
<div width="100%">

</div>
</span>
</div>

</div>
<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</td>
</tr>
</tbody></table>
</div>
</div>


<div id="array_expressHolder" class="collapsed">
<div id="array_express" style="display:none;">
<table cellpadding="0">
<tbody><tr>
<td colspan="2">
<b><span>Functional Genomics Experiments</span></b>
</td>
</tr>
<tr>
<td valign="top" width="500" align="left">

<div class="bxref1 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>


<div class="bxref2 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>

<div class="btmsmr collapsed" style="display:none;">
<div class="relartsdiv">
<h1> </h1>

<div class="tmsummary">
<span class="text1">
<div width="100%">

</div>
</span>
</div>

</div>
<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</td>
</tr>
</tbody></table>
</div>
</div>


<div id="protiensstructHolder" class="collapsed">
<div id="protiensstruct" style="display:none;">
<table cellpadding="0">
<tbody><tr>
<td colspan="2">
<b><span>Protein Structures</span></b>
</td>
</tr>
<tr>
<td valign="top" width="500" align="left">

<div class="bxref1 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>


<div class="bxref2 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>

<div class="btmsmr collapsed" style="display:none;">
<div class="relartsdiv">
<h1> </h1>

<div class="tmsummary">
<span class="text1">
<div width="100%">

</div>
</span>
</div>

</div>
<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</td>
</tr>
</tbody></table>
</div>
</div>


<div id="gotermsHolder" class="collapsed">
<div id="goterms" style="display:none;">
<table cellpadding="0">
<tbody><tr>
<td colspan="2">
<b><span>Gene Ontology (GO) Terms</span></b>
</td>
</tr>
<tr>
<td valign="top" width="500" align="left">

<div class="bxref1 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>


<div class="bxref2 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>

<div class="btmsmr collapsed" style="display:none;">
<div class="relartsdiv">
<h1> </h1>

<div class="tmsummary">
<span class="text1">
<div width="100%">

</div>
</span>
</div>

</div>
<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</td>
</tr>
</tbody></table>
</div>
</div>


<div id="organismsHolder" class="collapsed">
<div id="organisms" style="display:none;">
<table cellpadding="0">
<tbody><tr>
<td colspan="2">
<b><span>Species</span></b>
</td>
</tr>
<tr>
<td valign="top" width="500" align="left">

<div class="bxref1 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>


<div class="bxref2 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>

<div class="btmsmr collapsed" style="display:none;">
<div class="relartsdiv">
<h1> </h1>

<div class="tmsummary">
<span class="text1">
<div width="100%">

</div>
</span>
</div>

</div>
<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</td>
</tr>
</tbody></table>
</div>
</div>


<div id="diseasesHolder" class="collapsed">
<div id="diseases" style="display:none;">
<table cellpadding="0">
<tbody><tr>
<td colspan="2">
<b><span>Diseases</span></b>
</td>
</tr>
<tr>
<td valign="top" width="500" align="left">

<div class="bxref1 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>


<div class="bxref2 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>

<div class="btmsmr collapsed" style="display:none;">
<div class="relartsdiv">
<h1> </h1>

<div class="tmsummary">
<span class="text1">
<div width="100%">

</div>
</span>
</div>

</div>
<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</td>
</tr>
</tbody></table>
</div>
</div>


<div id="accessionHolder" class="collapsed">
<div id="accession" style="display:none;">
<table cellpadding="0">
<tbody><tr>
<td colspan="2">
<b><span>Data Citations</span></b>
</td>
</tr>
<tr>
<td valign="top" width="500" align="left">

<div class="bxref1 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>


<div class="bxref2 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>

<div class="btmsmr collapsed" style="display:none;">
<div class="relartsdiv">
<h1> </h1>

<div class="tmsummary">
<span class="text1">
<div width="100%">

</div>
</span>
</div>

</div>
<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</td>
</tr>
</tbody></table>
</div>
</div>


<div id="chemicalHolder" class="collapsed">
<div id="chemical" style="display:none;">
<table cellpadding="0">
<tbody><tr>
<td colspan="2">
<b><span>Chemicals</span></b>
</td>
</tr>
<tr>
<td valign="top" width="500" align="left">

<div class="bxref1 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>


<div class="bxref2 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>

<div class="btmsmr collapsed" style="display:none;">
<div class="relartsdiv">
<h1> </h1>

<div class="tmsummary">
<span class="text1">
<div width="100%">

</div>
</span>
</div>

</div>
<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</td>
</tr>
</tbody></table>
</div>
</div>


<div id="efoHolder" class="collapsed">
<div id="efo" style="display:none;">
<table cellpadding="0">
<tbody><tr>
<td colspan="2">
<b><span>Experimental Factor Ontology (EFO) Terms</span></b>
</td>
</tr>
<tr>
<td valign="top" width="500" align="left">

<div class="bxref1 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>


<div class="bxref2 collapsed" style="display:none;">

<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>

<div class="btmsmr collapsed" style="display:none;">
<div class="relartsdiv">
<h1> </h1>

<div class="tmsummary">
<span class="text1">
<div width="100%">

</div>
</span>
</div>

</div>
<div>&nbsp;</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</td>
</tr>
</tbody></table>
</div>
</div>

</div>
</div>
<div class="ui-tabs-panel ui-tabs-hide ui-widget-content ui-corner-bottom" id="fragment-related-pubmedarts">
<div width="100%" id="id1f">
<span id="rel_articles_loading_label"><img src="./Network_Diffusion_Brain_files/ajax-loader.gif" height="25 px" width="25 px"></span>
<div id="related_reviews_id">
<div class="citations">
<h3></h3>
<div class="citationsHolder collapsed" style="display:none">
<div class="citations_listing">

</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</div>
</div>
<br>
<div id="other_related_articles_id">
<div class="citations">
<h3></h3>
<div class="citationsHolder collapsed" style="display:none">
<div class="citations_listing">

</div>
<div class="showHideRelated">
<a>Show all items</a>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="ui-tabs-panel ui-tabs-hide ui-widget-content ui-corner-bottom" id="fragment-related-labLinks">
<div width="100%" id="id20" style="display:block;">
<span id="loading_links_label"><img src="./Network_Diffusion_Brain_files/ajax-loader.gif" height="25 px" width="25 px"></span>



</div>
</div>
</div>
</div>
</div>
<div class="page_info_margin">

<div>
<div style="display:none">

</div>
<div>
<script type="text/javascript">$(document).ready(function(){
 document.getElementById("fulltextnavpanel").className='citation_navigation_selected';
 });</script>



<div id="vertmenu">
<h5>Formats</h5>
<ul>
<li>
<div class="citation_navigation_unselected">
<a href="http://europepmc.org/abstract/MED/24384152" tabindex="1">Abstract</a>
</div>
</li>
<li>
<div id="fulltextnavpanel" class="citation_navigation_unselected">
Full Text <img border="0" src="./Network_Diffusion_Brain_files/small_logo.gif" height="14" width="14">
</div>
</li>

<li>
<div class="citation_navigation_unselected">
<a href="http://europepmc.org/articles/pmc3951650?pdf=render" tabindex="2" target="_blank">PDF</a>
</div>
</li>
<li>
<div class="citation_navigation_unselected">
<div style="border-top: 1px dashed #cccccc;">
<a style="display:none;" href="http://europepmc.org/articles/pmc3951650?1-1.ILinkListener-fulltextmarginpanel-export-exportlink">
<small>
<span><b>Export</b> citation (RIS)</span>
</small>
</a>
<a style="display:none; background-color:#fff;" href="http://europepmc.org/FAQs#RIS_Format" target="_top"><img border="0" src="./Network_Diffusion_Brain_files/qmark.png" alt="Export Citation in Research Information Systems (RIS) Format" onmouseover="TipAllowed(&#39;The Research Information Systems (RIS) Format is compatible with&lt;br/&gt;EndNote, Mendeley, Reference Manager, ProCite and RefWorks.&#39;)" onmouseout="UnTipAllowed()"></a>
</div>
</div>
<div class="citation_navigation_unselected">
<div>
<a style="display:none;" href="http://europepmc.org/articles/pmc3951650?1-1.ILinkListener-fulltextmarginpanel-email-emaillink" onclick="var w = window.open(href, &#39;europe_pmc_email_clipboard&#39;, &#39;scrollbars=no,location=no,menuBar=no,resizable=no,status=no,toolbar=no,width=550,height=600&#39;); try {if (w.blur) w.focus();}catch(ignore){}; return false;">
<small>
<span><b>Email</b> citation</span>
</small>
</a>
</div>
</div>
<div class="citation_navigation_unselected">
<div class="social_network_buttons">
<span id="twitter_article_button" class="social_network_button"><iframe id="twitter-widget-0" scrolling="no" frameborder="0" allowtransparency="true" src="./Network_Diffusion_Brain_files/tweet_button.b4a367eaaf8727457b2c58a20f015188.en.html" class="twitter-share-button twitter-tweet-button twitter-share-button twitter-count-none" title="Twitter Tweet Button" data-twttr-rendered="true" style="position: static; visibility: visible; width: 56px; height: 20px;"></iframe>

  
		</span>
</div>
</div>
</li>
</ul>

<br>
<h5>Journal and Issue</h5>
<div id="vertsubmenu" class="journalIssue">
<ul>
<li>
<a href="http://europepmc.org/journals/">Journal List</a><a href="http://www.ncbi.nlm.nih.gov/pmc/?term=hhs%20author%20manuscript[filter]&sort=SortDate&cmd=search&EntrezSystem2.PEntrez.Pmc.Pmc_LimitsTab.LimitsOff=true">HHS Author Manuscripts</a>
</li>
</ul>
</div>

</div>
<br>
</div>
</div>


<div>
<div id="vertmenu" style="overflow: auto;">
<h5>BioEntities in Full Text</h5>
<div id="vertsubmenu">
<div class="mesh_category"><span>Genes/Proteins</span></div>
<div>
<div class="bioent_item">
<a href="http://europepmc.org/articles/pmc3951650?europe_pmc_bio_tm_extredirect_gene_protein=http://www.uniprot.org/uniprot/?query=name:%22Cfr%22" onclick="var w = window.open(href, &#39;&#39;, &#39;scrollbars=yes,location=no,menuBar=no,resizable=yes,status=no,toolbar=no,width=900,height=900&#39;); try {if (w.blur) w.focus();}catch(ignore){}; return false;">
<span class="bioent_name">Cfr (3)</span>
</a>
</div>
<div class="search">
<a href="http://europepmc.org/search?page=1&query=%22Cfr%22">

<img src="./Network_Diffusion_Brain_files/search_magnifier-ver-BC510423D42C2D3A48ECB3E8E919E145.gif" border="0" height="14" width="14" align="top" onmouseover="TipAllowed(&#39;Perform a new search&#39;)" onmouseout="UnTipAllowed()">

</a>
</div>
</div><div>
<div class="bioent_item">
<a href="http://europepmc.org/articles/pmc3951650?europe_pmc_bio_tm_extredirect_gene_protein=http://www.uniprot.org/uniprot/?query=name:%22vis%22" onclick="var w = window.open(href, &#39;&#39;, &#39;scrollbars=yes,location=no,menuBar=no,resizable=yes,status=no,toolbar=no,width=900,height=900&#39;); try {if (w.blur) w.focus();}catch(ignore){}; return false;">
<span class="bioent_name">vis (2)</span>
</a>
</div>
<div class="search">
<a href="http://europepmc.org/search?page=1&query=%22vis%22">

<img src="./Network_Diffusion_Brain_files/search_magnifier-ver-BC510423D42C2D3A48ECB3E8E919E145.gif" border="0" height="14" width="14" align="top" onmouseover="TipAllowed(&#39;Perform a new search&#39;)" onmouseout="UnTipAllowed()">

</a>
</div>
</div><div>
<div class="bioent_item">
<a href="http://europepmc.org/articles/pmc3951650?europe_pmc_bio_tm_extredirect_gene_protein=http://www.uniprot.org/uniprot/?query=name:%22SPM%22" onclick="var w = window.open(href, &#39;&#39;, &#39;scrollbars=yes,location=no,menuBar=no,resizable=yes,status=no,toolbar=no,width=900,height=900&#39;); try {if (w.blur) w.focus();}catch(ignore){}; return false;">
<span class="bioent_name">SPM (2)</span>
</a>
</div>
<div class="search">
<a href="http://europepmc.org/search?page=1&query=%22SPM%22">

<img src="./Network_Diffusion_Brain_files/search_magnifier-ver-BC510423D42C2D3A48ECB3E8E919E145.gif" border="0" height="14" width="14" align="top" onmouseover="TipAllowed(&#39;Perform a new search&#39;)" onmouseout="UnTipAllowed()">

</a>
</div>
</div>
<div class="more">
<small><span><a href="http://europepmc.org/articles/pmc3951650#" id="gene_protein" style="color: #666666;"><b>4 more...</b></a></span></small>
</div>
<span><script type="text/javascript">
var $tabs = jQuery('#tabs_abstract').tabs();
 jQuery('#gene_protein').click(function() {
$tabs.tabs('option','selected', 2); 
 return true;
});
 </script>
</span>
</div>
<div id="vertsubmenu">
<div class="mesh_category"><span>Gene Ontology (GO)</span></div>
<div>
<div class="bioent_item">
<a href="http://europepmc.org/articles/pmc3951650?europe_pmc_bio_tm_extredirect_go_term=http://www.uniprot.org/uniprot/?query=go:0007610&sort=score" onclick="var w = window.open(href, &#39;&#39;, &#39;scrollbars=yes,location=no,menuBar=no,resizable=yes,status=no,toolbar=no,width=900,height=900&#39;); try {if (w.blur) w.focus();}catch(ignore){}; return false;">
<span class="bioent_name">behavior (23)</span>
</a>
</div>
<div class="search">
<a href="http://europepmc.org/search?page=1&query=%22behavior%22">

<img src="./Network_Diffusion_Brain_files/search_magnifier-ver-BC510423D42C2D3A48ECB3E8E919E145.gif" border="0" height="14" width="14" align="top" onmouseover="TipAllowed(&#39;Perform a new search&#39;)" onmouseout="UnTipAllowed()">

</a>
</div>
</div><div>
<div class="bioent_item">
<a href="http://europepmc.org/articles/pmc3951650?europe_pmc_bio_tm_extredirect_go_term=http://www.uniprot.org/uniprot/?query=go:0001508&sort=score" onclick="var w = window.open(href, &#39;&#39;, &#39;scrollbars=yes,location=no,menuBar=no,resizable=yes,status=no,toolbar=no,width=900,height=900&#39;); try {if (w.blur) w.focus();}catch(ignore){}; return false;">
<span class="bioent_name">action potentials (2)</span>
</a>
</div>
<div class="search">
<a href="http://europepmc.org/search?page=1&query=%22action+potentials%22">

<img src="./Network_Diffusion_Brain_files/search_magnifier-ver-BC510423D42C2D3A48ECB3E8E919E145(1).gif" border="0" height="14" width="14" align="top" onmouseover="TipAllowed(&#39;Perform a new search&#39;)" onmouseout="UnTipAllowed()">

</a>
</div>
</div><div>
<div class="bioent_item">
<a href="http://europepmc.org/articles/pmc3951650?europe_pmc_bio_tm_extredirect_go_term=http://www.uniprot.org/uniprot/?query=go:0023052&sort=score" onclick="var w = window.open(href, &#39;&#39;, &#39;scrollbars=yes,location=no,menuBar=no,resizable=yes,status=no,toolbar=no,width=900,height=900&#39;); try {if (w.blur) w.focus();}catch(ignore){}; return false;">
<span class="bioent_name">biological signaling (1)</span>
</a>
</div>
<div class="search">
<a href="http://europepmc.org/search?page=1&query=%22biological+signaling%22">

<img src="./Network_Diffusion_Brain_files/search_magnifier-ver-BC510423D42C2D3A48ECB3E8E919E145(1).gif" border="0" height="14" width="14" align="top" onmouseover="TipAllowed(&#39;Perform a new search&#39;)" onmouseout="UnTipAllowed()">

</a>
</div>
</div>
<div class="more">
<small><span><a href="http://europepmc.org/articles/pmc3951650#" id="go_term" style="color: #666666;"><b>3 more...</b></a></span></small>
</div>
<span><script type="text/javascript">
var $tabs = jQuery('#tabs_abstract').tabs();
 jQuery('#go_term').click(function() {
$tabs.tabs('option','selected', 2); 
 return true;
});
 </script>
</span>
</div>
<div id="vertsubmenu">
<div class="mesh_category"><span>Species</span></div>
<div>
<div class="bioent_item">
<a href="http://europepmc.org/articles/pmc3951650?europe_pmc_bio_tm_extredirect_Species=http://www.uniprot.org/uniprot/?query=organism:557721&sort=score" onclick="var w = window.open(href, &#39;&#39;, &#39;scrollbars=yes,location=no,menuBar=no,resizable=yes,status=no,toolbar=no,width=900,height=900&#39;); try {if (w.blur) w.focus();}catch(ignore){}; return false;">
<span class="bioent_name">atlas (2)</span>
</a>
</div>
<div class="search">
<a href="http://europepmc.org/search?page=1&query=%22atlas%22">

<img src="./Network_Diffusion_Brain_files/search_magnifier-ver-BC510423D42C2D3A48ECB3E8E919E145(1).gif" border="0" height="14" width="14" align="top" onmouseover="TipAllowed(&#39;Perform a new search&#39;)" onmouseout="UnTipAllowed()">

</a>
</div>
</div><div>
<div class="bioent_item">
<a href="http://europepmc.org/articles/pmc3951650?europe_pmc_bio_tm_extredirect_Species=http://www.uniprot.org/uniprot/?query=organism:8842&sort=score" onclick="var w = window.open(href, &#39;&#39;, &#39;scrollbars=yes,location=no,menuBar=no,resizable=yes,status=no,toolbar=no,width=900,height=900&#39;); try {if (w.blur) w.focus();}catch(ignore){}; return false;">
<span class="bioent_name">geese (1)</span>
</a>
</div>
<div class="search">
<a href="http://europepmc.org/search?page=1&query=%22geese%22">

<img src="./Network_Diffusion_Brain_files/search_magnifier-ver-BC510423D42C2D3A48ECB3E8E919E145(1).gif" border="0" height="14" width="14" align="top" onmouseover="TipAllowed(&#39;Perform a new search&#39;)" onmouseout="UnTipAllowed()">

</a>
</div>
</div><div>
<div class="bioent_item">
<a href="http://europepmc.org/articles/pmc3951650?europe_pmc_bio_tm_extredirect_Species=http://www.uniprot.org/uniprot/?query=organism:9606&sort=score" onclick="var w = window.open(href, &#39;&#39;, &#39;scrollbars=yes,location=no,menuBar=no,resizable=yes,status=no,toolbar=no,width=900,height=900&#39;); try {if (w.blur) w.focus();}catch(ignore){}; return false;">
<span class="bioent_name">human (1)</span>
</a>
</div>
<div class="search">
<a href="http://europepmc.org/search?page=1&query=%22human%22">

<img src="./Network_Diffusion_Brain_files/search_magnifier-ver-BC510423D42C2D3A48ECB3E8E919E145(1).gif" border="0" height="14" width="14" align="top" onmouseover="TipAllowed(&#39;Perform a new search&#39;)" onmouseout="UnTipAllowed()">

</a>
</div>
</div>
<div class="more">
<small><span><a href="http://europepmc.org/articles/pmc3951650#" id="Species" style="color: #666666;"><b>3 more...</b></a></span></small>
</div>
<span><script type="text/javascript">
var $tabs = jQuery('#tabs_abstract').tabs();
 jQuery('#Species').click(function() {
$tabs.tabs('option','selected', 2); 
 return true;
});
 </script>
</span>
</div>
<div id="vertsubmenu">
<div class="mesh_category"><span>Diseases</span></div>
<div>
<div class="bioent_item">
<a href="http://europepmc.org/articles/pmc3951650?europe_pmc_bio_tm_extredirect_Diseases=/search/?page=1&query=dementias">
<span class="bioent_name">dementias (1)</span>
</a>
</div>
<div class="search">
<a href="http://europepmc.org/search?page=1&query=%22dementias%22">

<img src="./Network_Diffusion_Brain_files/search_magnifier-ver-BC510423D42C2D3A48ECB3E8E919E145(1).gif" border="0" height="14" width="14" align="top" onmouseover="TipAllowed(&#39;Perform a new search&#39;)" onmouseout="UnTipAllowed()">

</a>
</div>
</div>
<div class="more">
<small></small>
</div>
<span><script type="text/javascript">
var $tabs = jQuery('#tabs_abstract').tabs();
 jQuery('#Diseases').click(function() {
$tabs.tabs('option','selected', 2); 
 return true;
});
 </script>
</span>
</div>
<div id="vertsubmenu">
<div class="mesh_category"><span>Chemicals</span></div>
<div>
<div class="bioent_item">
<a href="http://europepmc.org/articles/pmc3951650?europe_pmc_bio_tm_extredirect_Chemicals=http://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:6993" onclick="var w = window.open(href, &#39;&#39;, &#39;scrollbars=yes,location=no,menuBar=no,resizable=yes,status=no,toolbar=no,width=900,height=900&#39;); try {if (w.blur) w.focus();}catch(ignore){}; return false;">
<span class="bioent_name">singular (3)</span>
</a>
</div>
<div class="search">
<a href="http://europepmc.org/search?page=1&query=%22singular%22">

<img src="./Network_Diffusion_Brain_files/search_magnifier-ver-BC510423D42C2D3A48ECB3E8E919E145(1).gif" border="0" height="14" width="14" align="top" onmouseover="TipAllowed(&#39;Perform a new search&#39;)" onmouseout="UnTipAllowed()">

</a>
</div>
</div><div>
<div class="bioent_item">
<a href="http://europepmc.org/articles/pmc3951650?europe_pmc_bio_tm_extredirect_Chemicals=http://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:30443" onclick="var w = window.open(href, &#39;&#39;, &#39;scrollbars=yes,location=no,menuBar=no,resizable=yes,status=no,toolbar=no,width=900,height=900&#39;); try {if (w.blur) w.focus();}catch(ignore){}; return false;">
<span class="bioent_name">germane (2)</span>
</a>
</div>
<div class="search">
<a href="http://europepmc.org/search?page=1&query=%22germane%22">

<img src="./Network_Diffusion_Brain_files/search_magnifier-ver-BC510423D42C2D3A48ECB3E8E919E145(1).gif" border="0" height="14" width="14" align="top" onmouseover="TipAllowed(&#39;Perform a new search&#39;)" onmouseout="UnTipAllowed()">

</a>
</div>
</div><div>
<div class="bioent_item">
<a href="http://europepmc.org/articles/pmc3951650?europe_pmc_bio_tm_extredirect_Chemicals=http://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:20141" onclick="var w = window.open(href, &#39;&#39;, &#39;scrollbars=yes,location=no,menuBar=no,resizable=yes,status=no,toolbar=no,width=900,height=900&#39;); try {if (w.blur) w.focus();}catch(ignore){}; return false;">
<span class="bioent_name">3 ms (1)</span>
</a>
</div>
<div class="search">
<a href="http://europepmc.org/search?page=1&query=%223+ms%22">

<img src="./Network_Diffusion_Brain_files/search_magnifier-ver-BC510423D42C2D3A48ECB3E8E919E145(1).gif" border="0" height="14" width="14" align="top" onmouseover="TipAllowed(&#39;Perform a new search&#39;)" onmouseout="UnTipAllowed()">

</a>
</div>
</div>
<div class="more">
<small><span><a href="http://europepmc.org/articles/pmc3951650#" id="Chemicals" style="color: #666666;"><b>2 more...</b></a></span></small>
</div>
<span><script type="text/javascript">
var $tabs = jQuery('#tabs_abstract').tabs();
 jQuery('#Chemicals').click(function() {
$tabs.tabs('option','selected', 2); 
 return true;
});
 </script>
</span>
</div>
</div>
</div>

<div id="annotation_legend_container"></div>
</div>

<span>
<script type="text/javascript">
/*<![CDATA[*/

function getXMLHttpRequestObject()
{
var xmlhttp;
if (!xmlhttp && typeof XMLHttpRequest != 'undefined') {
try {
xmlhttp = new XMLHttpRequest();
} catch (e) {
xmlhttp = false;
}
}
return xmlhttp;
}
function getArticleXMLHttpRequestToUrl(url)
{
var http = new getXMLHttpRequestObject();
var abs = "/articles/"
http.open("GET", abs+url, true);
http.onreadystatechange = function() {
}
http.send(null);
}
function getAbstractXMLHttpRequestToUrl(url)
{
var http = new getXMLHttpRequestObject();
var abs = "/abstract/"
http.open("GET", abs+url, true);
http.onreadystatechange = function() {
}
http.send(null);
}
function getXMLHttpRequestForVisualisation(url)
{
var http = new getXMLHttpRequestObject();
http.open("GET",url, true);
http.onreadystatechange = function() {
}
http.send(null);
}

/*]]>*/
</script>
</span>
<script type="text/javascript">$(document).ready(function(){
 jQuery('#tabs_abstract').tabs("disable", 4); 
 });

jQuery('#labLinksarts').removeClass('ui-tabs-enabled');

jQuery('#labLinksarts').addClass('ui-tabs-disabled');

</script>
</div>
<script type="text/javascript">
/*<![CDATA[*/

jQuery('#tabs_abstract').tabs();

/*]]>*/
</script>
<script type="text/javascript"></script>

</div>
<div id="epmc_pageFooter" class="not4Frame">
<div>
<p class="epmc_footerNav">
<a href="http://www.ebi.ac.uk/Information/termsofuse.html" target="_blank" class="results2">Terms of Use</a>
&nbsp;&nbsp;|&nbsp;&nbsp;
<a href="http://europepmc.org/Copyright" target="_top" class="results2">Copyright</a>
&nbsp;&nbsp;|&nbsp;&nbsp;
<a href="http://europepmc.org/Accessibility" target="_top" class="results2">Accessibility</a>
&nbsp;&nbsp;|&nbsp;&nbsp;
<a href="http://www.ebi.ac.uk/Information/e-directive.html#cookies" target="_blank" class="results2">Cookies</a>
<span id="twitter_footer">&nbsp;&nbsp;|&nbsp;&nbsp;
<iframe id="twitter-widget-1" scrolling="no" frameborder="0" allowtransparency="true" src="./Network_Diffusion_Brain_files/follow_button.ae36744a06482b14f4c1ab8ab6ea765e.en.html" class="twitter-follow-button twitter-follow-button" title="Twitter Follow Button" data-twttr-rendered="true" style="position: static; visibility: visible; width: 170px; height: 20px;"></iframe>
</span>
</p>
<div class="epmc_footerText">Europe PubMed Central is a service of the <a href="http://europepmc.org/Funders/" target="_top" class="link">Europe PMC
Funders' Group</a> working in partnership with the <a href="http://www.ebi.ac.uk/" target="_blank" class="link">European
Bioinformatics Institute</a>, <a href="http://www.jisc.ac.uk/" target="_blank" class="link">JISC</a>, <a href="http://www.manchester.ac.uk/" target="_blank" class="link">University of
Manchester</a> and the <a href="http://www.bl.uk/science" target="_blank" class="link">British Library</a> in cooperation with the <a href="http://www.ncbi.nlm.nih.gov/" target="_blank" class="link">National Center
for Biotechnology Information</a> at the <a href="http://www.nlm.nih.gov/" target="_blank" class="link">U.S. National Library of Medicine (NCBI/NLM)</a>. It
includes content provided to the <a href="http://www.pubmedcentral.nih.gov/" target="_blank" class="link">PubMed
Central International archive</a> by participating publishers.
</div>
<script src="./Network_Diffusion_Brain_files/jquery.cookie.js" type="text/javascript"></script>
<script src="./Network_Diffusion_Brain_files/jquery.gchart.min.js" type="text/javascript"></script>
<script src="./Network_Diffusion_Brain_files/css_browser_selector.js" type="text/javascript"></script>
<script type="text/javascript">
/*<![CDATA[*/

jQuery(function(){ jQuery.ajax({
url: '/twitter.jsp',
success: function(textResponse){
//alert('Twitter Loaded');
jQuery('#twitter_footer').html(textResponse);
}
});});
// alert('Try lo load twitter');

/*]]>*/
</script>
</div>
</div>
</div>
<script type="text/javascript" src="./Network_Diffusion_Brain_files/wz_tooltip.js"></script>
<script type="text/javascript" src="./Network_Diffusion_Brain_files/jquery.corner.js"></script>
<script type="text/javascript">
/*<![CDATA[*/

/**$(document).ready(function(){
$('#container-1 > ul').tabs();
});*/

/*]]>*/
</script>
<script type="text/javascript">
/*<![CDATA[*/

jQuery.cachedScript = function( url, options ) {
// Allow user to set any option except for dataType, cache, and url
options = jQuery.extend( options || {}, {
dataType: "script",
cache: true,
url: url
});
// Use $.ajax() since it is more flexible than $.getScript
// Return the jqXHR object so we can chain callbacks
return jQuery.ajax( options );
};
jQuery.cachedCss = function( url, options ) {
// Allow user to set any option except for dataType, cache, and url
options = jQuery.extend( options || {}, {
dataType: "text",
cache: true,
url: url
});
// Use $.ajax() since it is more flexible than $.getScript
// Return the jqXHR object so we can chain callbacks
return jQuery.ajax( options );
};

/*]]>*/
</script>
<script type="text/javascript">
/*<![CDATA[*/

var piwikIdSite = document.getElementById('piwikIdSite').innerHTML;
var _paq = _paq || [];
_paq.push(['trackPageView']);
_paq.push(['enableLinkTracking']);
(function() {
var u="/piwik/";
_paq.push(['setTrackerUrl', u+'piwik.php']);
_paq.push(['setSiteId', piwikIdSite]);
var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
})();

/*]]>*/
</script>
<noscript>&amp;lt;p&amp;gt;&amp;lt;img src="/piwik/piwik.php?idsite=1" style="border:0;" alt="" /&amp;gt;&amp;lt;/p&amp;gt;</noscript>


<div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-family: MathJax_Script, sans-serif;"></div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></body></html>